<!DOCTYPE html><html lang="en" class="no-js"><head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="A site for documenting MCW Research Computing systems and services.">
      
      
        <meta name="author" content="MCW RCC">
      
      
        <link rel="canonical" href="https://docs.rcc.mcw.edu/software/r-pkg-list/">
      
      
        <link rel="prev" href="../cluster-pkg-list/">
      
      
        <link rel="next" href="../alphafold/">
      
      
        <link rel="alternate" type="application/rss+xml" title="RSS feed" href="../../feed_rss_created.xml">
        <link rel="alternate" type="application/rss+xml" title="RSS feed of updated content" href="../../feed_rss_updated.xml">
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.23">
    
    
      
        <title>R Packages - MCW Research Computing</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.84d31ad4.min.css">
      
      
  
  
    
      
      
    
  
  
  <style>:root{--md-admonition-icon--tip:url('data:image/svg+xml;charset=utf-8,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20viewBox%3D%220%200%2024%2024%22%3E%3Cpath%20d%3D%22M12%202a7%207%200%200%201%207%207c0%202.38-1.19%204.47-3%205.74V17a1%201%200%200%201-1%201H9a1%201%200%200%201-1-1v-2.26C6.19%2013.47%205%2011.38%205%209a7%207%200%200%201%207-7M9%2021v-1h6v1a1%201%200%200%201-1%201h-4a1%201%200%200%201-1-1m3-17a5%205%200%200%200-5%205c0%202.05%201.23%203.81%203%204.58V16h4v-2.42c1.77-.77%203-2.53%203-4.58a5%205%200%200%200-5-5%22/%3E%3C/svg%3E');}</style>



    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../_static/css/extra.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  

<script id="__analytics">function __md_analytics(){function e(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],e("js",new Date),e("config","G-50ZTKYWP10"),document.addEventListener("DOMContentLoaded",(function(){document.forms.search&&document.forms.search.query.addEventListener("blur",(function(){this.value&&e("event","search",{search_term:this.value})}));document$.subscribe((function(){var t=document.forms.feedback;if(void 0!==t)for(var a of t.querySelectorAll("[type=submit]"))a.addEventListener("click",(function(a){a.preventDefault();var n=document.location.pathname,d=this.getAttribute("data-md-value");e("event","feedback",{page:n,data:d}),t.firstElementChild.disabled=!0;var r=t.querySelector(".md-feedback__note [data-md-value='"+d+"']");r&&(r.hidden=!1)})),t.hidden=!1})),location$.subscribe((function(t){e("config","G-50ZTKYWP10",{page_path:t.pathname})}))}));var t=document.createElement("script");t.async=!0,t.src="https://www.googletagmanager.com/gtag/js?id=G-50ZTKYWP10",document.getElementById("__analytics").insertAdjacentElement("afterEnd",t)}</script>
  
    <script>if("undefined"!=typeof __md_analytics){var consent=__md_get("__consent");consent&&consent.analytics&&__md_analytics()}</script>
  

    
    
    
  <link href="../../assets/stylesheets/glightbox.min.css" rel="stylesheet"><script src="../../assets/javascripts/glightbox.min.js"></script><style id="glightbox-style">
            html.glightbox-open { overflow: initial; height: 100%; }
            .gslide-title { margin-top: 0px; user-select: text; }
            .gslide-desc { color: #666; user-select: text; }
            .gslide-image img { background: white; }
            .gscrollbar-fixer { padding-right: 15px; }
            .gdesc-inner { font-size: 0.75rem; }
            body[data-md-color-scheme="slate"] .gdesc-inner { background: var(--md-default-bg-color); }
            body[data-md-color-scheme="slate"] .gslide-title { color: var(--md-default-fg-color); }
            body[data-md-color-scheme="slate"] .gslide-desc { color: var(--md-default-fg-color); }
        </style></head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#installed-r-packages" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
        <aside class="md-banner">
          <div class="md-banner__inner md-grid md-typeset">
            
            
  <a style="color: white; " href="https://docs.rcc.mcw.edu/pubs/">
    Do you rely on Research Computing resources? Please acknowledge that support in your publications. It really helps!
  </a>

          </div>
          
        </aside>
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow md-header--lifted" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="MCW Research Computing" class="md-header__button md-logo" aria-label="MCW Research Computing" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9 3v15h3V3zm3 2 4 13 3-1-4-13zM5 5v13h3V5zM3 19v2h18v-2z"></path></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"></path></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MCW Research Computing
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              R Packages
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"></path></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"></path></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"></path></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"></path></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"></path></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/mcw-rcc/mcw-rcc.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"></path></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
    
      
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../.." class="md-tabs__link">
          
  
  
    
  
  Home

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../cluster/quickstart/" class="md-tabs__link">
          
  
  
    
  
  Cluster

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../storage/rcc-storage/" class="md-tabs__link">
          
  
  
    
  
  Data Storage

        </a>
      </li>
    
  

      
        
  
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../modules/" class="md-tabs__link">
          
  
  
    
  
  Software

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../secure-computing/reshpc/" class="md-tabs__link">
          
  
  
    
  
  ResHPC

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../learning/learning/" class="md-tabs__link">
          
  
  
    
  
  Learning

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
    
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="MCW Research Computing" class="md-nav__button md-logo" aria-label="MCW Research Computing" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9 3v15h3V3zm3 2 4 13 3-1-4-13zM5 5v13h3V5zM3 19v2h18v-2z"></path></svg>

    </a>
    MCW Research Computing
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/mcw-rcc/mcw-rcc.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"></path></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_1">
        
          
          <div class="md-nav__link md-nav__container">
            <a href="../.." class="md-nav__link ">
              
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_1" id="__nav_1_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_1_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_1">
            <span class="md-nav__icon md-icon"></span>
            Home
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
    
  
  
    <li class="md-nav__item">
      <a href="../../news/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Recent News
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../grants/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Grant Boilerplate
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../pubs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Publications
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../maintenance/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Maintenance
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../faq/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    FAQ
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2">
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Cluster
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Cluster
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/quickstart/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Quick Start
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/accounts/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Getting an Account
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/etiquette/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    User Etiquette
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/hardware/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Hardware Specs
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_5">
        
          
          <label class="md-nav__link" for="__nav_2_5" id="__nav_2_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Connecting
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_5">
            <span class="md-nav__icon md-icon"></span>
            Connecting
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/login/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Logging in
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/ondemand/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Open OnDemand
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/remote-access/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Remote Access
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_5_4">
        
          
          <label class="md-nav__link" for="__nav_2_5_4" id="__nav_2_5_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    SSH Clients
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_5_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_5_4">
            <span class="md-nav__icon md-icon"></span>
            SSH Clients
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/mobaxterm/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MobaXterm
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/putty/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    PuTTY
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/ssh-keys/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    SSH Keys
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_6">
        
          
          <label class="md-nav__link" for="__nav_2_6" id="__nav_2_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Running Jobs
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_6">
            <span class="md-nav__icon md-icon"></span>
            Running Jobs
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/running-jobs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Submitting SLURM Jobs
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/storage-job/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Using Data in a Job
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/software-job/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Software in a Job
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/troubleshoot/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Troubleshoot Jobs
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/xdmod/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Job Metrics
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3">
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Data Storage
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Data Storage
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/rcc-storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Storage Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/paid-storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Paid Additional Storage
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/ref-data/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Reference Data
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/mcw-storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Additional Storage Options
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_5">
        
          
          <label class="md-nav__link" for="__nav_3_5" id="__nav_3_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Data Transfer
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_5">
            <span class="md-nav__icon md-icon"></span>
            Data Transfer
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/file-transfer/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Transfer to Cluster
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/globus/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Globus
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/rclone/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Rclone
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_6">
        
          
          <label class="md-nav__link" for="__nav_3_6" id="__nav_3_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Manage &amp; Share
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_6">
            <span class="md-nav__icon md-icon"></span>
            Manage &amp; Share
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/data-access/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Mounting Drives
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/data-sharing/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Sharing Data
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/file-cleanup/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Cleanup &amp; Archiving
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/file-recovery/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Recovery
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/file-permissions/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Permission Changes
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
      
        
        
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" checked>
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Software
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Software
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../modules/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Using Modules
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../module-request/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Requesting Software
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    
    
      
        
      
        
      
    
    
    
      
      
        
          
          
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_3" checked>
        
          
          <label class="md-nav__link" for="__nav_4_3" id="__nav_4_3_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Installed Software
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4_3">
            <span class="md-nav__icon md-icon"></span>
            Installed Software
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../cluster-pkg-list/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Cluster Packages
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    R Packages
    
  </span>
  

      </a>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
          
          
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4">
        
          
          <label class="md-nav__link" for="__nav_4_4" id="__nav_4_4_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Guides
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4">
            <span class="md-nav__icon md-icon"></span>
            Guides
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../alphafold/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    AlphaFold
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ansys/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    ANSYS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../conda/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Conda
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../containers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Containers
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../git/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Git and GitHub
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../advanced_git/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Git and GitHub (advanced)
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../jupyter/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Jupyter
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../lsdyna/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    LS-DYNA
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../matlab/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MATLAB Parallel Server
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../namd/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    NAMD
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../nextflow/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Nextflow
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../python/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Python
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../pytorch/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    PyTorch
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../R/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    R
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../schrodinger/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Schr√∂dinger
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../tensorflow/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    TensorFlow
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5">
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    ResHPC
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            ResHPC
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/reshpc/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/project-account/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Accounts &amp; Projects
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/project-storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Storage
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/mfa-login/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Logging in
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/mfa-data-transfer/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Transfer
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/workflow/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Run a Job
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/software/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Software
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6">
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Learning
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Learning
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../learning/learning/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../learning/hpc_clusters/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    HPC Clusters Tutorials
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../learning/slurm/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    SLURM Guides
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../learning/programming/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Programming Guides
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="installed-r-packages">Installed R packages<a class="headerlink" href="#installed-r-packages" title="Permanent link">#</a></h1>
<!-- markdownlint-disable MD011 -->
<p>The following 1115 packages are installed with R 4.5.0.</p>
<!-- markdownlint-enable MD011 -->

<p>Last updated: <em><span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date" title="February 17, 2026 18:58:19 UTC">February 17, 2026</span></em></p>
<!-- 
// Copyright 2014-2022 Stanford Research Computing Center
//
// The following code is a derivative work of https://raw.githubusercontent.com/stanford-rc/www.sherlock.stanford.edu/main/src/docs/software/list.md, which is licensed GPLv3. This code therefore is also licensed under the terms GPLv3.
-->

<!-- markdownlint-disable MD056 -->
<table>
<thead>
<tr>
<th style="text-align: left;">Package</th>
<th style="text-align: left;">Version</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;">abc</td>
<td style="text-align: left;">2.2.2</td>
<td style="text-align: left;">Tools for Approximate Bayesian Computation (ABC)</td>
</tr>
<tr>
<td style="text-align: left;">abc.data</td>
<td style="text-align: left;">1.1</td>
<td style="text-align: left;">Data Only: Tools for Approximate Bayesian Computation (ABC)</td>
</tr>
<tr>
<td style="text-align: left;">abind</td>
<td style="text-align: left;">1.4-8</td>
<td style="text-align: left;">Combine Multidimensional Arrays</td>
</tr>
<tr>
<td style="text-align: left;">ade4</td>
<td style="text-align: left;">1.7-23</td>
<td style="text-align: left;">Analysis of Ecological Data: Exploratory and Euclidean Methods in Environmental Sciences</td>
</tr>
<tr>
<td style="text-align: left;">admisc</td>
<td style="text-align: left;">0.38</td>
<td style="text-align: left;">Adrian Dusa's Miscellaneous</td>
</tr>
<tr>
<td style="text-align: left;">afex</td>
<td style="text-align: left;">1.4-1</td>
<td style="text-align: left;">Analysis of Factorial Experiments</td>
</tr>
<tr>
<td style="text-align: left;">affxparser</td>
<td style="text-align: left;">1.80.0</td>
<td style="text-align: left;">Affymetrix File Parsing SDK</td>
</tr>
<tr>
<td style="text-align: left;">affy</td>
<td style="text-align: left;">1.86.0</td>
<td style="text-align: left;">Methods for Affymetrix Oligonucleotide Arrays</td>
</tr>
<tr>
<td style="text-align: left;">affyio</td>
<td style="text-align: left;">1.78.0</td>
<td style="text-align: left;">Tools for parsing Affymetrix data files</td>
</tr>
<tr>
<td style="text-align: left;">airr</td>
<td style="text-align: left;">1.5.0</td>
<td style="text-align: left;">AIRR Data Representation Reference Library</td>
</tr>
<tr>
<td style="text-align: left;">alabaster.base</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Save Bioconductor Objects to File</td>
</tr>
<tr>
<td style="text-align: left;">alabaster.matrix</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Load and Save Artifacts from File</td>
</tr>
<tr>
<td style="text-align: left;">alabaster.ranges</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Load and Save Ranges-related Artifacts from File</td>
</tr>
<tr>
<td style="text-align: left;">alabaster.schemas</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Schemas for the Alabaster Framework</td>
</tr>
<tr>
<td style="text-align: left;">alabaster.se</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Load and Save SummarizedExperiments from File</td>
</tr>
<tr>
<td style="text-align: left;">alakazam</td>
<td style="text-align: left;">1.3.0</td>
<td style="text-align: left;">Immunoglobulin Clonal Lineage and Diversity Analysis</td>
</tr>
<tr>
<td style="text-align: left;">amap</td>
<td style="text-align: left;">0.8-20</td>
<td style="text-align: left;">Another Multidimensional Analysis Package</td>
</tr>
<tr>
<td style="text-align: left;">anndata</td>
<td style="text-align: left;">0.7.5.6</td>
<td style="text-align: left;">'anndata' for R</td>
</tr>
<tr>
<td style="text-align: left;">annotate</td>
<td style="text-align: left;">1.86.0</td>
<td style="text-align: left;">Annotation for microarrays</td>
</tr>
<tr>
<td style="text-align: left;">AnnotationDbi</td>
<td style="text-align: left;">1.70.0</td>
<td style="text-align: left;">Manipulation of SQLite-based annotations in Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">AnnotationFilter</td>
<td style="text-align: left;">1.32.0</td>
<td style="text-align: left;">Facilities for Filtering Bioconductor Annotation Resources</td>
</tr>
<tr>
<td style="text-align: left;">AnnotationHub</td>
<td style="text-align: left;">3.16.0</td>
<td style="text-align: left;">Client to access AnnotationHub resources</td>
</tr>
<tr>
<td style="text-align: left;">anytime</td>
<td style="text-align: left;">0.3.11</td>
<td style="text-align: left;">Anything to 'POSIXct' or 'Date' Converter</td>
</tr>
<tr>
<td style="text-align: left;">aod</td>
<td style="text-align: left;">1.3.3</td>
<td style="text-align: left;">Analysis of Overdispersed Data</td>
</tr>
<tr>
<td style="text-align: left;">ape</td>
<td style="text-align: left;">5.8-1</td>
<td style="text-align: left;">Analyses of Phylogenetics and Evolution</td>
</tr>
<tr>
<td style="text-align: left;">apeglm</td>
<td style="text-align: left;">1.30.0</td>
<td style="text-align: left;">Approximate posterior estimation for GLM coefficients</td>
</tr>
<tr>
<td style="text-align: left;">aplot</td>
<td style="text-align: left;">0.2.5</td>
<td style="text-align: left;">Decorate a 'ggplot' with Associated Information</td>
</tr>
<tr>
<td style="text-align: left;">argparse</td>
<td style="text-align: left;">2.2.5</td>
<td style="text-align: left;">Command Line Optional and Positional Argument Parser</td>
</tr>
<tr>
<td style="text-align: left;">argparser</td>
<td style="text-align: left;">0.7.2</td>
<td style="text-align: left;">Command-Line Argument Parser</td>
</tr>
<tr>
<td style="text-align: left;">arm</td>
<td style="text-align: left;">1.14-4</td>
<td style="text-align: left;">Data Analysis Using Regression and Multilevel/Hierarchical Models</td>
</tr>
<tr>
<td style="text-align: left;">arrayhelpers</td>
<td style="text-align: left;">1.1-0</td>
<td style="text-align: left;">Convenience Functions for Arrays</td>
</tr>
<tr>
<td style="text-align: left;">arrow</td>
<td style="text-align: left;">19.0.1.1</td>
<td style="text-align: left;">Integration to 'Apache' 'Arrow'</td>
</tr>
<tr>
<td style="text-align: left;">ashr</td>
<td style="text-align: left;">2.2-63</td>
<td style="text-align: left;">Methods for Adaptive Shrinkage, using Empirical Bayes</td>
</tr>
<tr>
<td style="text-align: left;">askpass</td>
<td style="text-align: left;">1.2.1</td>
<td style="text-align: left;">Password Entry Utilities for R, Git, and SSH</td>
</tr>
<tr>
<td style="text-align: left;">assertthat</td>
<td style="text-align: left;">0.2.1</td>
<td style="text-align: left;">Easy Pre and Post Assertions</td>
</tr>
<tr>
<td style="text-align: left;">assorthead</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">Assorted Header-Only C++ Libraries</td>
</tr>
<tr>
<td style="text-align: left;">AUCell</td>
<td style="text-align: left;">1.30.0</td>
<td style="text-align: left;">AUCell: Analysis of 'gene set' activity in single-cell RNA-seq data (e.g. identify cells with specific gene signatures)</td>
</tr>
<tr>
<td style="text-align: left;">babelgene</td>
<td style="text-align: left;">22.9</td>
<td style="text-align: left;">Gene Orthologs for Model Organisms in a Tidy Data Format</td>
</tr>
<tr>
<td style="text-align: left;">backbone</td>
<td style="text-align: left;">2.1.4</td>
<td style="text-align: left;">Extracts the Backbone from Graphs</td>
</tr>
<tr>
<td style="text-align: left;">backports</td>
<td style="text-align: left;">1.5.0</td>
<td style="text-align: left;">Reimplementations of Functions Introduced Since R-3.0.0</td>
</tr>
<tr>
<td style="text-align: left;">base</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">The R Base Package</td>
</tr>
<tr>
<td style="text-align: left;">base64enc</td>
<td style="text-align: left;">0.1-3</td>
<td style="text-align: left;">Tools for base64 encoding</td>
</tr>
<tr>
<td style="text-align: left;">base64url</td>
<td style="text-align: left;">1.4</td>
<td style="text-align: left;">Fast and URL-Safe Base64 Encoder and Decoder</td>
</tr>
<tr>
<td style="text-align: left;">basefun</td>
<td style="text-align: left;">1.2-2</td>
<td style="text-align: left;">Infrastructure for Computing with Basis Functions</td>
</tr>
<tr>
<td style="text-align: left;">basilisk</td>
<td style="text-align: left;">1.20.0</td>
<td style="text-align: left;">Freezing Python Dependencies Inside Bioconductor Packages</td>
</tr>
<tr>
<td style="text-align: left;">basilisk.utils</td>
<td style="text-align: left;">1.20.0</td>
<td style="text-align: left;">Basilisk Installation Utilities</td>
</tr>
<tr>
<td style="text-align: left;">batchelor</td>
<td style="text-align: left;">1.24.0</td>
<td style="text-align: left;">Single-Cell Batch Correction Methods</td>
</tr>
<tr>
<td style="text-align: left;">bayesplot</td>
<td style="text-align: left;">1.12.0</td>
<td style="text-align: left;">Plotting for Bayesian Models</td>
</tr>
<tr>
<td style="text-align: left;">bayestestR</td>
<td style="text-align: left;">0.15.3</td>
<td style="text-align: left;">Understand and Describe Bayesian Models and Posterior Distributions</td>
</tr>
<tr>
<td style="text-align: left;">BBmisc</td>
<td style="text-align: left;">1.13</td>
<td style="text-align: left;">Miscellaneous Helper Functions for B. Bischl</td>
</tr>
<tr>
<td style="text-align: left;">bbmle</td>
<td style="text-align: left;">1.0.25.1</td>
<td style="text-align: left;">Tools for General Maximum Likelihood Estimation</td>
</tr>
<tr>
<td style="text-align: left;">bdsmatrix</td>
<td style="text-align: left;">1.3-7</td>
<td style="text-align: left;">Routines for Block Diagonal Symmetric Matrices</td>
</tr>
<tr>
<td style="text-align: left;">beachmat</td>
<td style="text-align: left;">2.24.0</td>
<td style="text-align: left;">Compiling Bioconductor to Handle Each Matrix Type</td>
</tr>
<tr>
<td style="text-align: left;">beeswarm</td>
<td style="text-align: left;">0.4.0</td>
<td style="text-align: left;">The Bee Swarm Plot, an Alternative to Stripchart</td>
</tr>
<tr>
<td style="text-align: left;">bestglm</td>
<td style="text-align: left;">0.37.3</td>
<td style="text-align: left;">Best Subset GLM and Regression Utilities</td>
</tr>
<tr>
<td style="text-align: left;">BgeeDB</td>
<td style="text-align: left;">2.34.0</td>
<td style="text-align: left;">Annotation and gene expression data retrieval from Bgee database. TopAnat, an anatomical entities Enrichment Analysis tool for UBERON ontology</td>
</tr>
<tr>
<td style="text-align: left;">BH</td>
<td style="text-align: left;">1.87.0-1</td>
<td style="text-align: left;">Boost C++ Header Files</td>
</tr>
<tr>
<td style="text-align: left;">bigD</td>
<td style="text-align: left;">0.3.1</td>
<td style="text-align: left;">Flexibly Format Dates and Times to a Given Locale</td>
</tr>
<tr>
<td style="text-align: left;">biglm</td>
<td style="text-align: left;">0.9-3</td>
<td style="text-align: left;">Bounded Memory Linear and Generalized Linear Models</td>
</tr>
<tr>
<td style="text-align: left;">bigmemory</td>
<td style="text-align: left;">4.6.4</td>
<td style="text-align: left;">Manage Massive Matrices with Shared Memory and Memory-Mapped Files</td>
</tr>
<tr>
<td style="text-align: left;">bigmemory.sri</td>
<td style="text-align: left;">0.1.8</td>
<td style="text-align: left;">A Shared Resource Interface for Bigmemory Project Packages</td>
</tr>
<tr>
<td style="text-align: left;">bio3d</td>
<td style="text-align: left;">2.4-5</td>
<td style="text-align: left;">Biological Structure Analysis</td>
</tr>
<tr>
<td style="text-align: left;">Biobase</td>
<td style="text-align: left;">2.68.0</td>
<td style="text-align: left;">Biobase: Base functions for Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">BiocBaseUtils</td>
<td style="text-align: left;">1.10.0</td>
<td style="text-align: left;">General utility functions for developing Bioconductor packages</td>
</tr>
<tr>
<td style="text-align: left;">BiocFileCache</td>
<td style="text-align: left;">2.16.0</td>
<td style="text-align: left;">Manage Files Across Sessions</td>
</tr>
<tr>
<td style="text-align: left;">BiocGenerics</td>
<td style="text-align: left;">0.54.0</td>
<td style="text-align: left;">S4 generic functions used in Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">BiocIO</td>
<td style="text-align: left;">1.18.0</td>
<td style="text-align: left;">Standard Input and Output for Bioconductor Packages</td>
</tr>
<tr>
<td style="text-align: left;">BiocManager</td>
<td style="text-align: left;">1.30.25</td>
<td style="text-align: left;">Access the Bioconductor Project Package Repository</td>
</tr>
<tr>
<td style="text-align: left;">BiocNeighbors</td>
<td style="text-align: left;">2.2.0</td>
<td style="text-align: left;">Nearest Neighbor Detection for Bioconductor Packages</td>
</tr>
<tr>
<td style="text-align: left;">BiocParallel</td>
<td style="text-align: left;">1.42.2</td>
<td style="text-align: left;">Bioconductor facilities for parallel evaluation</td>
</tr>
<tr>
<td style="text-align: left;">BiocSingular</td>
<td style="text-align: left;">1.24.0</td>
<td style="text-align: left;">Singular Value Decomposition for Bioconductor Packages</td>
</tr>
<tr>
<td style="text-align: left;">BiocVersion</td>
<td style="text-align: left;">3.21.1</td>
<td style="text-align: left;">Set the appropriate version of Bioconductor packages</td>
</tr>
<tr>
<td style="text-align: left;">biocViews</td>
<td style="text-align: left;">1.76.0</td>
<td style="text-align: left;">Categorized views of R package repositories</td>
</tr>
<tr>
<td style="text-align: left;">biomaRt</td>
<td style="text-align: left;">2.64.0</td>
<td style="text-align: left;">Interface to BioMart databases (i.e. Ensembl)</td>
</tr>
<tr>
<td style="text-align: left;">Biostrings</td>
<td style="text-align: left;">2.76.0</td>
<td style="text-align: left;">Efficient manipulation of biological strings</td>
</tr>
<tr>
<td style="text-align: left;">biovizBase</td>
<td style="text-align: left;">1.56.0</td>
<td style="text-align: left;">Basic graphic utilities for visualization of genomic data.</td>
</tr>
<tr>
<td style="text-align: left;">bit</td>
<td style="text-align: left;">4.6.0</td>
<td style="text-align: left;">Classes and Methods for Fast Memory-Efficient Boolean Selections</td>
</tr>
<tr>
<td style="text-align: left;">bit64</td>
<td style="text-align: left;">4.6.0-1</td>
<td style="text-align: left;">A S3 Class for Vectors of 64bit Integers</td>
</tr>
<tr>
<td style="text-align: left;">bitops</td>
<td style="text-align: left;">1.0-9</td>
<td style="text-align: left;">Bitwise Operations</td>
</tr>
<tr>
<td style="text-align: left;">blme</td>
<td style="text-align: left;">1.0-6</td>
<td style="text-align: left;">Bayesian Linear Mixed-Effects Models</td>
</tr>
<tr>
<td style="text-align: left;">blob</td>
<td style="text-align: left;">1.2.4</td>
<td style="text-align: left;">A Simple S3 Class for Representing Vectors of Binary Data ('BLOBS')</td>
</tr>
<tr>
<td style="text-align: left;">bmp</td>
<td style="text-align: left;">0.3</td>
<td style="text-align: left;">Read Windows Bitmap (BMP) Images</td>
</tr>
<tr>
<td style="text-align: left;">boot</td>
<td style="text-align: left;">1.3-31</td>
<td style="text-align: left;">Bootstrap Functions (Originally by Angelo Canty for S)</td>
</tr>
<tr>
<td style="text-align: left;">bootnet</td>
<td style="text-align: left;">1.6</td>
<td style="text-align: left;">Bootstrap Methods for Various Network Estimation Routines</td>
</tr>
<tr>
<td style="text-align: left;">bread</td>
<td style="text-align: left;">0.4.1</td>
<td style="text-align: left;">Analyze Big Files Without Loading Them in Memory</td>
</tr>
<tr>
<td style="text-align: left;">brew</td>
<td style="text-align: left;">1.0-10</td>
<td style="text-align: left;">Templating Framework for Report Generation</td>
</tr>
<tr>
<td style="text-align: left;">bridgesampling</td>
<td style="text-align: left;">1.1-2</td>
<td style="text-align: left;">Bridge Sampling for Marginal Likelihoods and Bayes Factors</td>
</tr>
<tr>
<td style="text-align: left;">brio</td>
<td style="text-align: left;">1.1.5</td>
<td style="text-align: left;">Basic R Input Output</td>
</tr>
<tr>
<td style="text-align: left;">brms</td>
<td style="text-align: left;">2.22.0</td>
<td style="text-align: left;">Bayesian Regression Models using 'Stan'</td>
</tr>
<tr>
<td style="text-align: left;">Brobdingnag</td>
<td style="text-align: left;">1.2-9</td>
<td style="text-align: left;">Very Large Numbers in R</td>
</tr>
<tr>
<td style="text-align: left;">broom</td>
<td style="text-align: left;">1.0.8</td>
<td style="text-align: left;">Convert Statistical Objects into Tidy Tibbles</td>
</tr>
<tr>
<td style="text-align: left;">broom.helpers</td>
<td style="text-align: left;">1.21.0</td>
<td style="text-align: left;">Helpers for Model Coefficients Tibbles</td>
</tr>
<tr>
<td style="text-align: left;">bs4Dash</td>
<td style="text-align: left;">2.3.4</td>
<td style="text-align: left;">A 'Bootstrap 4' Version of 'shinydashboard'</td>
</tr>
<tr>
<td style="text-align: left;">BSgenome</td>
<td style="text-align: left;">1.76.0</td>
<td style="text-align: left;">Software infrastructure for efficient representation of full genomes and their SNPs</td>
</tr>
<tr>
<td style="text-align: left;">BSgenome.Hsapiens.UCSC.hg38</td>
<td style="text-align: left;">1.4.5</td>
<td style="text-align: left;">Full genomic sequences for Homo sapiens (UCSC genome hg38)</td>
</tr>
<tr>
<td style="text-align: left;">bslib</td>
<td style="text-align: left;">0.9.0</td>
<td style="text-align: left;">Custom 'Bootstrap' 'Sass' Themes for 'shiny' and 'rmarkdown'</td>
</tr>
<tr>
<td style="text-align: left;">bsseq</td>
<td style="text-align: left;">1.44.1</td>
<td style="text-align: left;">Analyze, manage and store whole-genome methylation data</td>
</tr>
<tr>
<td style="text-align: left;">ca</td>
<td style="text-align: left;">0.71.1</td>
<td style="text-align: left;">Simple, Multiple and Joint Correspondence Analysis</td>
</tr>
<tr>
<td style="text-align: left;">cachem</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">Cache R Objects with Automatic Pruning</td>
</tr>
<tr>
<td style="text-align: left;">Cairo</td>
<td style="text-align: left;">1.6-2</td>
<td style="text-align: left;">R Graphics Device using Cairo Graphics Library for Creating High-Quality Bitmap (PNG, JPEG, TIFF), Vector (PDF, SVG, PostScript) and Display (X11 and Win32) Output</td>
</tr>
<tr>
<td style="text-align: left;">calibrate</td>
<td style="text-align: left;">1.7.7</td>
<td style="text-align: left;">Calibration of Scatterplot and Biplot Axes</td>
</tr>
<tr>
<td style="text-align: left;">callr</td>
<td style="text-align: left;">3.7.6</td>
<td style="text-align: left;">Call R from R</td>
</tr>
<tr>
<td style="text-align: left;">candisc</td>
<td style="text-align: left;">0.9.0</td>
<td style="text-align: left;">Visualizing Generalized Canonical Discriminant and Canonical Correlation Analysis</td>
</tr>
<tr>
<td style="text-align: left;">car</td>
<td style="text-align: left;">3.1-3</td>
<td style="text-align: left;">Companion to Applied Regression</td>
</tr>
<tr>
<td style="text-align: left;">carData</td>
<td style="text-align: left;">3.0-5</td>
<td style="text-align: left;">Companion to Applied Regression Data Sets</td>
</tr>
<tr>
<td style="text-align: left;">cards</td>
<td style="text-align: left;">0.6.0</td>
<td style="text-align: left;">Analysis Results Data</td>
</tr>
<tr>
<td style="text-align: left;">caret</td>
<td style="text-align: left;">7.0-1</td>
<td style="text-align: left;">Classification and Regression Training</td>
</tr>
<tr>
<td style="text-align: left;">caTools</td>
<td style="text-align: left;">1.18.3</td>
<td style="text-align: left;">Tools: Moving Window Statistics, GIF, Base64, ROC AUC, etc</td>
</tr>
<tr>
<td style="text-align: left;">CellChat</td>
<td style="text-align: left;">2.2.0</td>
<td style="text-align: left;">Inference and analysis of cell-cell communication from single-cell and spatially resolved transcriptomics data</td>
</tr>
<tr>
<td style="text-align: left;">celldex</td>
<td style="text-align: left;">1.18.0</td>
<td style="text-align: left;">Index of Reference Cell Type Datasets</td>
</tr>
<tr>
<td style="text-align: left;">cellranger</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">Translate Spreadsheet Cell Ranges to Rows and Columns</td>
</tr>
<tr>
<td style="text-align: left;">checkmate</td>
<td style="text-align: left;">2.3.2</td>
<td style="text-align: left;">Fast and Versatile Argument Checks</td>
</tr>
<tr>
<td style="text-align: left;">ChIPseeker</td>
<td style="text-align: left;">1.44.0</td>
<td style="text-align: left;">ChIPseeker for ChIP peak Annotation, Comparison, and Visualization</td>
</tr>
<tr>
<td style="text-align: left;">chipseq</td>
<td style="text-align: left;">1.58.0</td>
<td style="text-align: left;">chipseq: A package for analyzing chipseq data</td>
</tr>
<tr>
<td style="text-align: left;">chk</td>
<td style="text-align: left;">0.10.0</td>
<td style="text-align: left;">Check User-Supplied Function Arguments</td>
</tr>
<tr>
<td style="text-align: left;">chromVAR</td>
<td style="text-align: left;">1.30.0</td>
<td style="text-align: left;">Chromatin Variation Across Regions</td>
</tr>
<tr>
<td style="text-align: left;">chron</td>
<td style="text-align: left;">2.3-62</td>
<td style="text-align: left;">Chronological Objects which Can Handle Dates and Times</td>
</tr>
<tr>
<td style="text-align: left;">circlize</td>
<td style="text-align: left;">0.4.16</td>
<td style="text-align: left;">Circular Visualization</td>
</tr>
<tr>
<td style="text-align: left;">class</td>
<td style="text-align: left;">7.3-23</td>
<td style="text-align: left;">Functions for Classification</td>
</tr>
<tr>
<td style="text-align: left;">classInt</td>
<td style="text-align: left;">0.4-11</td>
<td style="text-align: left;">Choose Univariate Class Intervals</td>
</tr>
<tr>
<td style="text-align: left;">cli</td>
<td style="text-align: left;">3.6.5</td>
<td style="text-align: left;">Helpers for Developing Command Line Interfaces</td>
</tr>
<tr>
<td style="text-align: left;">clipr</td>
<td style="text-align: left;">0.8.0</td>
<td style="text-align: left;">Read and Write from the System Clipboard</td>
</tr>
<tr>
<td style="text-align: left;">clock</td>
<td style="text-align: left;">0.7.3</td>
<td style="text-align: left;">Date-Time Types and Tools</td>
</tr>
<tr>
<td style="text-align: left;">clue</td>
<td style="text-align: left;">0.3-66</td>
<td style="text-align: left;">Cluster Ensembles</td>
</tr>
<tr>
<td style="text-align: left;">cluster</td>
<td style="text-align: left;">2.1.8.1</td>
<td style="text-align: left;">"Finding Groups in Data": Cluster Analysis Extended Rousseeuw et al.</td>
</tr>
<tr>
<td style="text-align: left;">clustermole</td>
<td style="text-align: left;">1.1.1</td>
<td style="text-align: left;">Unbiased Single-Cell Transcriptomic Data Cell Type Identification</td>
</tr>
<tr>
<td style="text-align: left;">clusterProfiler</td>
<td style="text-align: left;">4.16.0</td>
<td style="text-align: left;">A universal enrichment tool for interpreting omics data</td>
</tr>
<tr>
<td style="text-align: left;">ClusterR</td>
<td style="text-align: left;">1.3.3</td>
<td style="text-align: left;">Gaussian Mixture Models, K-Means, Mini-Batch-Kmeans, K-Medoids and Affinity Propagation Clustering</td>
</tr>
<tr>
<td style="text-align: left;">clustree</td>
<td style="text-align: left;">0.5.1</td>
<td style="text-align: left;">Visualise Clusterings at Different Resolutions</td>
</tr>
<tr>
<td style="text-align: left;">cmprsk</td>
<td style="text-align: left;">2.2-12</td>
<td style="text-align: left;">Subdistribution Analysis of Competing Risks</td>
</tr>
<tr>
<td style="text-align: left;">CNEr</td>
<td style="text-align: left;">1.43.0</td>
<td style="text-align: left;">CNE Detection and Visualization</td>
</tr>
<tr>
<td style="text-align: left;">cocor</td>
<td style="text-align: left;">1.1-4</td>
<td style="text-align: left;">Comparing Correlations</td>
</tr>
<tr>
<td style="text-align: left;">coda</td>
<td style="text-align: left;">0.19-4.1</td>
<td style="text-align: left;">Output Analysis and Diagnostics for MCMC</td>
</tr>
<tr>
<td style="text-align: left;">codetools</td>
<td style="text-align: left;">0.2-20</td>
<td style="text-align: left;">Code Analysis Tools for R</td>
</tr>
<tr>
<td style="text-align: left;">coin</td>
<td style="text-align: left;">1.4-3</td>
<td style="text-align: left;">Conditional Inference Procedures in a Permutation Test Framework</td>
</tr>
<tr>
<td style="text-align: left;">colorRamp2</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">Generate Color Mapping Functions</td>
</tr>
<tr>
<td style="text-align: left;">colorspace</td>
<td style="text-align: left;">2.1-1</td>
<td style="text-align: left;">A Toolbox for Manipulating and Assessing Colors and Palettes</td>
</tr>
<tr>
<td style="text-align: left;">colourpicker</td>
<td style="text-align: left;">1.3.0</td>
<td style="text-align: left;">A Colour Picker Tool for Shiny and for Selecting Colours in Plots</td>
</tr>
<tr>
<td style="text-align: left;">cols4all</td>
<td style="text-align: left;">0.8</td>
<td style="text-align: left;">Colors for all</td>
</tr>
<tr>
<td style="text-align: left;">combinat</td>
<td style="text-align: left;">0.0-8</td>
<td style="text-align: left;">combinatorics utilities</td>
</tr>
<tr>
<td style="text-align: left;">commonmark</td>
<td style="text-align: left;">1.9.5</td>
<td style="text-align: left;">High Performance CommonMark and Github Markdown Rendering in R</td>
</tr>
<tr>
<td style="text-align: left;">comorbidity</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">Computing Comorbidity Scores</td>
</tr>
<tr>
<td style="text-align: left;">compiler</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">The R Compiler Package</td>
</tr>
<tr>
<td style="text-align: left;">ComplexHeatmap</td>
<td style="text-align: left;">2.24.0</td>
<td style="text-align: left;">Make Complex Heatmaps</td>
</tr>
<tr>
<td style="text-align: left;">config</td>
<td style="text-align: left;">0.3.2</td>
<td style="text-align: left;">Manage Environment Specific Configuration Values</td>
</tr>
<tr>
<td style="text-align: left;">confintr</td>
<td style="text-align: left;">1.0.2</td>
<td style="text-align: left;">Confidence Intervals</td>
</tr>
<tr>
<td style="text-align: left;">conflicted</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">An Alternative Conflict Resolution Strategy</td>
</tr>
<tr>
<td style="text-align: left;">conquer</td>
<td style="text-align: left;">1.3.3</td>
<td style="text-align: left;">Convolution-Type Smoothed Quantile Regression</td>
</tr>
<tr>
<td style="text-align: left;">coro</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">'Coroutines' for R</td>
</tr>
<tr>
<td style="text-align: left;">corpcor</td>
<td style="text-align: left;">1.6.10</td>
<td style="text-align: left;">Efficient Estimation of Covariance and (Partial) Correlation</td>
</tr>
<tr>
<td style="text-align: left;">corrplot</td>
<td style="text-align: left;">0.95</td>
<td style="text-align: left;">Visualization of a Correlation Matrix</td>
</tr>
<tr>
<td style="text-align: left;">covr</td>
<td style="text-align: left;">3.6.4</td>
<td style="text-align: left;">Test Coverage for Packages</td>
</tr>
<tr>
<td style="text-align: left;">cowplot</td>
<td style="text-align: left;">1.1.3</td>
<td style="text-align: left;">Streamlined Plot Theme and Plot Annotations for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">coxme</td>
<td style="text-align: left;">2.2-22</td>
<td style="text-align: left;">Mixed Effects Cox Models</td>
</tr>
<tr>
<td style="text-align: left;">cpp11</td>
<td style="text-align: left;">0.5.2</td>
<td style="text-align: left;">A C++11 Interface for R's C Interface</td>
</tr>
<tr>
<td style="text-align: left;">crayon</td>
<td style="text-align: left;">1.5.3</td>
<td style="text-align: left;">Colored Terminal Output</td>
</tr>
<tr>
<td style="text-align: left;">credentials</td>
<td style="text-align: left;">2.0.2</td>
<td style="text-align: left;">Tools for Managing SSH and Git Credentials</td>
</tr>
<tr>
<td style="text-align: left;">crmn</td>
<td style="text-align: left;">0.0.21</td>
<td style="text-align: left;">CCMN and Other Normalization Methods for Metabolomics Data</td>
</tr>
<tr>
<td style="text-align: left;">crossmatch</td>
<td style="text-align: left;">1.4-0</td>
<td style="text-align: left;">The Cross-Match Test</td>
</tr>
<tr>
<td style="text-align: left;">crosstalk</td>
<td style="text-align: left;">1.2.1</td>
<td style="text-align: left;">Inter-Widget Interactivity for HTML Widgets</td>
</tr>
<tr>
<td style="text-align: left;">crul</td>
<td style="text-align: left;">1.5.0</td>
<td style="text-align: left;">HTTP Client</td>
</tr>
<tr>
<td style="text-align: left;">csaw</td>
<td style="text-align: left;">1.42.0</td>
<td style="text-align: left;">ChIP-Seq Analysis with Windows</td>
</tr>
<tr>
<td style="text-align: left;">cubature</td>
<td style="text-align: left;">2.1.1</td>
<td style="text-align: left;">Adaptive Multivariate Integration over Hypercubes</td>
</tr>
<tr>
<td style="text-align: left;">curl</td>
<td style="text-align: left;">6.2.2</td>
<td style="text-align: left;">A Modern and Flexible Web Client for R</td>
</tr>
<tr>
<td style="text-align: left;">cvAUC</td>
<td style="text-align: left;">1.1.4</td>
<td style="text-align: left;">Cross-Validated Area Under the ROC Curve Confidence Intervals</td>
</tr>
<tr>
<td style="text-align: left;">data.table</td>
<td style="text-align: left;">1.17.0</td>
<td style="text-align: left;">Extension of <code>data.frame</code></td>
</tr>
<tr>
<td style="text-align: left;">data.tree</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">General Purpose Hierarchical Data Structure</td>
</tr>
<tr>
<td style="text-align: left;">datasets</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">The R Datasets Package</td>
</tr>
<tr>
<td style="text-align: left;">datawizard</td>
<td style="text-align: left;">1.0.2</td>
<td style="text-align: left;">Easy Data Wrangling and Statistical Transformations</td>
</tr>
<tr>
<td style="text-align: left;">date</td>
<td style="text-align: left;">1.2-42</td>
<td style="text-align: left;">Functions for Handling Dates</td>
</tr>
<tr>
<td style="text-align: left;">DBI</td>
<td style="text-align: left;">1.2.3</td>
<td style="text-align: left;">R Database Interface</td>
</tr>
<tr>
<td style="text-align: left;">dbparser</td>
<td style="text-align: left;">2.0.3</td>
<td style="text-align: left;">Drugs Databases Parser</td>
</tr>
<tr>
<td style="text-align: left;">dbplyr</td>
<td style="text-align: left;">2.5.0</td>
<td style="text-align: left;">A 'dplyr' Back End for Databases</td>
</tr>
<tr>
<td style="text-align: left;">dbscan</td>
<td style="text-align: left;">1.2.2</td>
<td style="text-align: left;">Density-Based Spatial Clustering of Applications with Noise (DBSCAN) and Related Algorithms</td>
</tr>
<tr>
<td style="text-align: left;">DDRTree</td>
<td style="text-align: left;">0.1.5</td>
<td style="text-align: left;">Learning Principal Graphs with DDRTree</td>
</tr>
<tr>
<td style="text-align: left;">decor</td>
<td style="text-align: left;">1.0.2</td>
<td style="text-align: left;">Retrieve Code Decorations</td>
</tr>
<tr>
<td style="text-align: left;">DelayedArray</td>
<td style="text-align: left;">0.34.1</td>
<td style="text-align: left;">A unified framework for working transparently with on-disk and in-memory array-like datasets</td>
</tr>
<tr>
<td style="text-align: left;">DelayedMatrixStats</td>
<td style="text-align: left;">1.30.0</td>
<td style="text-align: left;">Functions that Apply to Rows and Columns of 'DelayedMatrix' Objects</td>
</tr>
<tr>
<td style="text-align: left;">deldir</td>
<td style="text-align: left;">2.0-4</td>
<td style="text-align: left;">Delaunay Triangulation and Dirichlet (Voronoi) Tessellation</td>
</tr>
<tr>
<td style="text-align: left;">dendextend</td>
<td style="text-align: left;">1.19.0</td>
<td style="text-align: left;">Extending 'dendrogram' Functionality in R</td>
</tr>
<tr>
<td style="text-align: left;">densityClust</td>
<td style="text-align: left;">0.3.3</td>
<td style="text-align: left;">Clustering by Fast Search and Find of Density Peaks</td>
</tr>
<tr>
<td style="text-align: left;">densvis</td>
<td style="text-align: left;">1.18.0</td>
<td style="text-align: left;">Density-Preserving Data Visualization via Non-Linear Dimensionality Reduction</td>
</tr>
<tr>
<td style="text-align: left;">DEoptimR</td>
<td style="text-align: left;">1.1-3-1</td>
<td style="text-align: left;">Differential Evolution Optimization in Pure R</td>
</tr>
<tr>
<td style="text-align: left;">Deriv</td>
<td style="text-align: left;">4.1.6</td>
<td style="text-align: left;">Symbolic Differentiation</td>
</tr>
<tr>
<td style="text-align: left;">desc</td>
<td style="text-align: left;">1.4.3</td>
<td style="text-align: left;">Manipulate DESCRIPTION Files</td>
</tr>
<tr>
<td style="text-align: left;">DescTools</td>
<td style="text-align: left;">0.99.60</td>
<td style="text-align: left;">Tools for Descriptive Statistics</td>
</tr>
<tr>
<td style="text-align: left;">DESeq2</td>
<td style="text-align: left;">1.48.0</td>
<td style="text-align: left;">Differential gene expression analysis based on the negative binomial distribution</td>
</tr>
<tr>
<td style="text-align: left;">devtools</td>
<td style="text-align: left;">2.4.6</td>
<td style="text-align: left;">Tools to Make Developing R Packages Easier</td>
</tr>
<tr>
<td style="text-align: left;">diagram</td>
<td style="text-align: left;">1.6.5</td>
<td style="text-align: left;">Functions for Visualising Simple Graphs (Networks), Plotting Flow Diagrams</td>
</tr>
<tr>
<td style="text-align: left;">DiagrammeR</td>
<td style="text-align: left;">1.0.11</td>
<td style="text-align: left;">Graph/Network Visualization</td>
</tr>
<tr>
<td style="text-align: left;">dials</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Tools for Creating Tuning Parameter Values</td>
</tr>
<tr>
<td style="text-align: left;">DiceDesign</td>
<td style="text-align: left;">1.10</td>
<td style="text-align: left;">Designs of Computer Experiments</td>
</tr>
<tr>
<td style="text-align: left;">dichromat</td>
<td style="text-align: left;">2.0-0.1</td>
<td style="text-align: left;">Color Schemes for Dichromats</td>
</tr>
<tr>
<td style="text-align: left;">diffobj</td>
<td style="text-align: left;">0.3.6</td>
<td style="text-align: left;">Diffs for R Objects</td>
</tr>
<tr>
<td style="text-align: left;">digest</td>
<td style="text-align: left;">0.6.37</td>
<td style="text-align: left;">Create Compact Hash Digests of R Objects</td>
</tr>
<tr>
<td style="text-align: left;">dir.expiry</td>
<td style="text-align: left;">1.16.0</td>
<td style="text-align: left;">Managing Expiration for Cache Directories</td>
</tr>
<tr>
<td style="text-align: left;">DirichletMultinomial</td>
<td style="text-align: left;">1.50.0</td>
<td style="text-align: left;">Dirichlet-Multinomial Mixture Model Machine Learning for Microbiome Data</td>
</tr>
<tr>
<td style="text-align: left;">distributional</td>
<td style="text-align: left;">0.5.0</td>
<td style="text-align: left;">Vectorised Probability Distributions</td>
</tr>
<tr>
<td style="text-align: left;">DNAcopy</td>
<td style="text-align: left;">1.82.0</td>
<td style="text-align: left;">DNA Copy Number Data Analysis</td>
</tr>
<tr>
<td style="text-align: left;">doBy</td>
<td style="text-align: left;">4.6.26</td>
<td style="text-align: left;">Groupwise Statistics, LSmeans, Linear Estimates, Utilities</td>
</tr>
<tr>
<td style="text-align: left;">docopt</td>
<td style="text-align: left;">0.7.2</td>
<td style="text-align: left;">Command-Line Interface Specification Language</td>
</tr>
<tr>
<td style="text-align: left;">doFuture</td>
<td style="text-align: left;">1.0.2</td>
<td style="text-align: left;">Use Foreach to Parallelize via the Future Framework</td>
</tr>
<tr>
<td style="text-align: left;">doMC</td>
<td style="text-align: left;">1.3.8</td>
<td style="text-align: left;">Foreach Parallel Adaptor for 'parallel'</td>
</tr>
<tr>
<td style="text-align: left;">doParallel</td>
<td style="text-align: left;">1.0.17</td>
<td style="text-align: left;">Foreach Parallel Adaptor for the 'parallel' Package</td>
</tr>
<tr>
<td style="text-align: left;">doRNG</td>
<td style="text-align: left;">1.8.6.2</td>
<td style="text-align: left;">Generic Reproducible Parallel Backend for 'foreach' Loops</td>
</tr>
<tr>
<td style="text-align: left;">DOSE</td>
<td style="text-align: left;">4.2.0</td>
<td style="text-align: left;">Disease Ontology Semantic and Enrichment analysis</td>
</tr>
<tr>
<td style="text-align: left;">doSNOW</td>
<td style="text-align: left;">1.0.20</td>
<td style="text-align: left;">Foreach Parallel Adaptor for the 'snow' Package</td>
</tr>
<tr>
<td style="text-align: left;">dotCall64</td>
<td style="text-align: left;">1.2</td>
<td style="text-align: left;">Enhanced Foreign Function Interface Supporting Long Vectors</td>
</tr>
<tr>
<td style="text-align: left;">dotty</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">The Unpacking Dot Operator</td>
</tr>
<tr>
<td style="text-align: left;">DoubleExpSeq</td>
<td style="text-align: left;">1.1</td>
<td style="text-align: left;">Differential Exon Usage Test for RNA-Seq Data via Empirical Bayes Shrinkage of the Dispersion Parameter</td>
</tr>
<tr>
<td style="text-align: left;">downlit</td>
<td style="text-align: left;">0.4.4</td>
<td style="text-align: left;">Syntax Highlighting and Automatic Linking</td>
</tr>
<tr>
<td style="text-align: left;">downloader</td>
<td style="text-align: left;">0.4.1</td>
<td style="text-align: left;">Download Files over HTTP and HTTPS</td>
</tr>
<tr>
<td style="text-align: left;">dplyr</td>
<td style="text-align: left;">1.1.4</td>
<td style="text-align: left;">A Grammar of Data Manipulation</td>
</tr>
<tr>
<td style="text-align: left;">dqrng</td>
<td style="text-align: left;">0.4.1</td>
<td style="text-align: left;">Fast Pseudo Random Number Generators</td>
</tr>
<tr>
<td style="text-align: left;">DT</td>
<td style="text-align: left;">0.33</td>
<td style="text-align: left;">A Wrapper of the JavaScript Library 'DataTables'</td>
</tr>
<tr>
<td style="text-align: left;">dtplyr</td>
<td style="text-align: left;">1.3.1</td>
<td style="text-align: left;">Data Table Back-End for 'dplyr'</td>
</tr>
<tr>
<td style="text-align: left;">dygraphs</td>
<td style="text-align: left;">1.1.1.6</td>
<td style="text-align: left;">Interface to 'Dygraphs' Interactive Time Series Charting Library</td>
</tr>
<tr>
<td style="text-align: left;">dynamicTreeCut</td>
<td style="text-align: left;">1.63-1</td>
<td style="text-align: left;">Methods for Detection of Clusters in Hierarchical Clustering Dendrograms</td>
</tr>
<tr>
<td style="text-align: left;">e1071</td>
<td style="text-align: left;">1.7-16</td>
<td style="text-align: left;">Misc Functions of the Department of Statistics, Probability Theory Group (Formerly: E1071), TU Wien</td>
</tr>
<tr>
<td style="text-align: left;">edgeR</td>
<td style="text-align: left;">4.6.3</td>
<td style="text-align: left;">Empirical Analysis of Digital Gene Expression Data in R</td>
</tr>
<tr>
<td style="text-align: left;">egg</td>
<td style="text-align: left;">0.4.5</td>
<td style="text-align: left;">Extensions for 'ggplot2': Custom Geom, Custom Themes, Plot Alignment, Labelled Panels, Symmetric Scales, and Fixed Panel Size</td>
</tr>
<tr>
<td style="text-align: left;">eigenmodel</td>
<td style="text-align: left;">1.11</td>
<td style="text-align: left;">Semiparametric Factor and Regression Models for Symmetric Relational Data</td>
</tr>
<tr>
<td style="text-align: left;">ellipse</td>
<td style="text-align: left;">0.5.0</td>
<td style="text-align: left;">Functions for Drawing Ellipses and Ellipse-Like Confidence Regions</td>
</tr>
<tr>
<td style="text-align: left;">ellipsis</td>
<td style="text-align: left;">0.3.2</td>
<td style="text-align: left;">Tools for Working with ...</td>
</tr>
<tr>
<td style="text-align: left;">emdbook</td>
<td style="text-align: left;">1.3.13</td>
<td style="text-align: left;">Support Functions and Data for "Ecological Models and Data"</td>
</tr>
<tr>
<td style="text-align: left;">emmeans</td>
<td style="text-align: left;">1.11.0</td>
<td style="text-align: left;">Estimated Marginal Means, aka Least-Squares Means</td>
</tr>
<tr>
<td style="text-align: left;">emoa</td>
<td style="text-align: left;">0.5-3</td>
<td style="text-align: left;">Evolutionary Multiobjective Optimization Algorithms</td>
</tr>
<tr>
<td style="text-align: left;">EnhancedVolcano</td>
<td style="text-align: left;">1.26.0</td>
<td style="text-align: left;">Publication-ready volcano plots with enhanced colouring and labeling</td>
</tr>
<tr>
<td style="text-align: left;">EnrichedHeatmap</td>
<td style="text-align: left;">1.38.0</td>
<td style="text-align: left;">Making Enriched Heatmaps</td>
</tr>
<tr>
<td style="text-align: left;">enrichplot</td>
<td style="text-align: left;">1.28.0</td>
<td style="text-align: left;">Visualization of Functional Enrichment Result</td>
</tr>
<tr>
<td style="text-align: left;">EnsDb.Hsapiens.v86</td>
<td style="text-align: left;">2.99.0</td>
<td style="text-align: left;">Ensembl based annotation package</td>
</tr>
<tr>
<td style="text-align: left;">ensembldb</td>
<td style="text-align: left;">2.32.0</td>
<td style="text-align: left;">Utilities to create and use Ensembl-based annotation databases</td>
</tr>
<tr>
<td style="text-align: left;">EnvStats</td>
<td style="text-align: left;">3.1.0</td>
<td style="text-align: left;">Package for Environmental Statistics, Including US EPA Guidance</td>
</tr>
<tr>
<td style="text-align: left;">estimability</td>
<td style="text-align: left;">1.5.1</td>
<td style="text-align: left;">Tools for Assessing Estimability of Linear Predictions</td>
</tr>
<tr>
<td style="text-align: left;">etrunct</td>
<td style="text-align: left;">0.1</td>
<td style="text-align: left;">Computes Moments of Univariate Truncated t Distribution</td>
</tr>
<tr>
<td style="text-align: left;">eulerr</td>
<td style="text-align: left;">7.0.2</td>
<td style="text-align: left;">Area-Proportional Euler and Venn Diagrams with Ellipses</td>
</tr>
<tr>
<td style="text-align: left;">evaluate</td>
<td style="text-align: left;">1.0.3</td>
<td style="text-align: left;">Parsing and Evaluation Tools that Provide More Details than the Default</td>
</tr>
<tr>
<td style="text-align: left;">Exact</td>
<td style="text-align: left;">3.3</td>
<td style="text-align: left;">Unconditional Exact Test</td>
</tr>
<tr>
<td style="text-align: left;">exactRankTests</td>
<td style="text-align: left;">0.8-35</td>
<td style="text-align: left;">Exact Distributions for Rank and Permutation Tests</td>
</tr>
<tr>
<td style="text-align: left;">ExperimentHub</td>
<td style="text-align: left;">2.16.0</td>
<td style="text-align: left;">Client to access ExperimentHub resources</td>
</tr>
<tr>
<td style="text-align: left;">expm</td>
<td style="text-align: left;">1.0-0</td>
<td style="text-align: left;">Matrix Exponential, Log, 'etc'</td>
</tr>
<tr>
<td style="text-align: left;">extrafont</td>
<td style="text-align: left;">0.19</td>
<td style="text-align: left;">Tools for Using Fonts</td>
</tr>
<tr>
<td style="text-align: left;">extrafontdb</td>
<td style="text-align: left;">1.0</td>
<td style="text-align: left;">Package for holding the database for the extrafont package</td>
</tr>
<tr>
<td style="text-align: left;">factoextra</td>
<td style="text-align: left;">1.0.7</td>
<td style="text-align: left;">Extract and Visualize the Results of Multivariate Data Analyses</td>
</tr>
<tr>
<td style="text-align: left;">FactoMineR</td>
<td style="text-align: left;">2.11</td>
<td style="text-align: left;">Multivariate Exploratory Data Analysis and Data Mining</td>
</tr>
<tr>
<td style="text-align: left;">fANCOVA</td>
<td style="text-align: left;">0.6-1</td>
<td style="text-align: left;">Nonparametric Analysis of Covariance</td>
</tr>
<tr>
<td style="text-align: left;">fansi</td>
<td style="text-align: left;">1.0.6</td>
<td style="text-align: left;">ANSI Control Sequence Aware String Functions</td>
</tr>
<tr>
<td style="text-align: left;">farver</td>
<td style="text-align: left;">2.1.2</td>
<td style="text-align: left;">High Performance Colour Space Manipulation</td>
</tr>
<tr>
<td style="text-align: left;">fastcluster</td>
<td style="text-align: left;">1.2.6</td>
<td style="text-align: left;">Fast Hierarchical Clustering Routines for R and 'Python'</td>
</tr>
<tr>
<td style="text-align: left;">fastDummies</td>
<td style="text-align: left;">1.7.5</td>
<td style="text-align: left;">Fast Creation of Dummy (Binary) Columns and Rows from Categorical Variables</td>
</tr>
<tr>
<td style="text-align: left;">fastICA</td>
<td style="text-align: left;">1.2-7</td>
<td style="text-align: left;">FastICA Algorithms to Perform ICA and Projection Pursuit</td>
</tr>
<tr>
<td style="text-align: left;">fastmap</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">Fast Data Structures</td>
</tr>
<tr>
<td style="text-align: left;">fastmatch</td>
<td style="text-align: left;">1.1-6</td>
<td style="text-align: left;">Fast 'match()' Function</td>
</tr>
<tr>
<td style="text-align: left;">fdrtool</td>
<td style="text-align: left;">1.2.18</td>
<td style="text-align: left;">Estimation of (Local) False Discovery Rates and Higher Criticism</td>
</tr>
<tr>
<td style="text-align: left;">feather</td>
<td style="text-align: left;">0.3.5</td>
<td style="text-align: left;">R Bindings to the Feather 'API'</td>
</tr>
<tr>
<td style="text-align: left;">ff</td>
<td style="text-align: left;">4.5.2</td>
<td style="text-align: left;">Memory-Efficient Storage of Large Data on Disk and Fast Access Functions</td>
</tr>
<tr>
<td style="text-align: left;">fftwtools</td>
<td style="text-align: left;">0.9-11</td>
<td style="text-align: left;">Wrapper for 'FFTW3' Includes: One-Dimensional, Two-Dimensional, Three-Dimensional, and Multivariate Transforms</td>
</tr>
<tr>
<td style="text-align: left;">fgsea</td>
<td style="text-align: left;">1.34.2</td>
<td style="text-align: left;">Fast Gene Set Enrichment Analysis</td>
</tr>
<tr>
<td style="text-align: left;">fields</td>
<td style="text-align: left;">16.3.1</td>
<td style="text-align: left;">Tools for Spatial Data</td>
</tr>
<tr>
<td style="text-align: left;">filelock</td>
<td style="text-align: left;">1.0.3</td>
<td style="text-align: left;">Portable File Locking</td>
</tr>
<tr>
<td style="text-align: left;">findpython</td>
<td style="text-align: left;">1.0.9</td>
<td style="text-align: left;">Functions to Find an Acceptable Python Binary</td>
</tr>
<tr>
<td style="text-align: left;">fitdistrplus</td>
<td style="text-align: left;">1.2-2</td>
<td style="text-align: left;">Help to Fit of a Parametric Distribution to Non-Censored or Censored Data</td>
</tr>
<tr>
<td style="text-align: left;">flashClust</td>
<td style="text-align: left;">1.01-2</td>
<td style="text-align: left;">Implementation of optimal hierarchical clustering</td>
</tr>
<tr>
<td style="text-align: left;">flextable</td>
<td style="text-align: left;">0.9.7</td>
<td style="text-align: left;">Functions for Tabular Reporting</td>
</tr>
<tr>
<td style="text-align: left;">fmsb</td>
<td style="text-align: left;">0.7.6</td>
<td style="text-align: left;">Functions for Medical Statistics Book with some Demographic Data</td>
</tr>
<tr>
<td style="text-align: left;">FNN</td>
<td style="text-align: left;">1.1.4.1</td>
<td style="text-align: left;">Fast Nearest Neighbor Search Algorithms and Applications</td>
</tr>
<tr>
<td style="text-align: left;">fontawesome</td>
<td style="text-align: left;">0.5.3</td>
<td style="text-align: left;">Easily Work with 'Font Awesome' Icons</td>
</tr>
<tr>
<td style="text-align: left;">fontBitstreamVera</td>
<td style="text-align: left;">0.1.1</td>
<td style="text-align: left;">Fonts with 'Bitstream Vera Fonts' License</td>
</tr>
<tr>
<td style="text-align: left;">fontLiberation</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">Liberation Fonts</td>
</tr>
<tr>
<td style="text-align: left;">fontquiver</td>
<td style="text-align: left;">0.2.1</td>
<td style="text-align: left;">Set of Installed Fonts</td>
</tr>
<tr>
<td style="text-align: left;">forcats</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">Tools for Working with Categorical Variables (Factors)</td>
</tr>
<tr>
<td style="text-align: left;">foreach</td>
<td style="text-align: left;">1.5.2</td>
<td style="text-align: left;">Provides Foreach Looping Construct</td>
</tr>
<tr>
<td style="text-align: left;">foreign</td>
<td style="text-align: left;">0.8-90</td>
<td style="text-align: left;">Read Data Stored by 'Minitab', 'S', 'SAS', 'SPSS', 'Stata', 'Systat', 'Weka', 'dBase', ...</td>
</tr>
<tr>
<td style="text-align: left;">formatR</td>
<td style="text-align: left;">1.14</td>
<td style="text-align: left;">Format R Code Automatically</td>
</tr>
<tr>
<td style="text-align: left;">formattable</td>
<td style="text-align: left;">0.2.1</td>
<td style="text-align: left;">Create 'Formattable' Data Structures</td>
</tr>
<tr>
<td style="text-align: left;">Formula</td>
<td style="text-align: left;">1.2-5</td>
<td style="text-align: left;">Extended Model Formulas</td>
</tr>
<tr>
<td style="text-align: left;">formula.tools</td>
<td style="text-align: left;">1.7.1</td>
<td style="text-align: left;">Programmatic Utilities for Manipulating Formulas, Expressions, Calls, Assignments and Other R Objects</td>
</tr>
<tr>
<td style="text-align: left;">freesurferformats</td>
<td style="text-align: left;">0.1.18</td>
<td style="text-align: left;">Read and Write 'FreeSurfer' Neuroimaging File Formats</td>
</tr>
<tr>
<td style="text-align: left;">fresh</td>
<td style="text-align: left;">0.2.1</td>
<td style="text-align: left;">Create Custom 'Bootstrap' Themes to Use in 'Shiny'</td>
</tr>
<tr>
<td style="text-align: left;">fs</td>
<td style="text-align: left;">1.6.6</td>
<td style="text-align: left;">Cross-Platform File System Operations Based on 'libuv'</td>
</tr>
<tr>
<td style="text-align: left;">fsbrain</td>
<td style="text-align: left;">0.5.5</td>
<td style="text-align: left;">Managing and Visualizing Brain Surface Data</td>
</tr>
<tr>
<td style="text-align: left;">fslr</td>
<td style="text-align: left;">2.25.3</td>
<td style="text-align: left;">Wrapper Functions for 'FSL' ('FMRIB' Software Library) from Functional MRI of the Brain ('FMRIB')</td>
</tr>
<tr>
<td style="text-align: left;">fst</td>
<td style="text-align: left;">0.9.8</td>
<td style="text-align: left;">Lightning Fast Serialization of Data Frames</td>
</tr>
<tr>
<td style="text-align: left;">fstcore</td>
<td style="text-align: left;">0.10.0</td>
<td style="text-align: left;">R Bindings to the 'Fstlib' Library</td>
</tr>
<tr>
<td style="text-align: left;">furrr</td>
<td style="text-align: left;">0.3.1</td>
<td style="text-align: left;">Apply Mapping Functions in Parallel using Futures</td>
</tr>
<tr>
<td style="text-align: left;">futile.logger</td>
<td style="text-align: left;">1.4.3</td>
<td style="text-align: left;">A Logging Utility for R</td>
</tr>
<tr>
<td style="text-align: left;">futile.options</td>
<td style="text-align: left;">1.0.1</td>
<td style="text-align: left;">Futile Options Management</td>
</tr>
<tr>
<td style="text-align: left;">future</td>
<td style="text-align: left;">1.40.0</td>
<td style="text-align: left;">Unified Parallel and Distributed Processing in R for Everyone</td>
</tr>
<tr>
<td style="text-align: left;">future.apply</td>
<td style="text-align: left;">1.11.3</td>
<td style="text-align: left;">Apply Function to Elements in Parallel using Futures</td>
</tr>
<tr>
<td style="text-align: left;">gam</td>
<td style="text-align: left;">1.22-5</td>
<td style="text-align: left;">Generalized Additive Models</td>
</tr>
<tr>
<td style="text-align: left;">gamm4</td>
<td style="text-align: left;">0.2-7</td>
<td style="text-align: left;">Generalized Additive Mixed Models using 'mgcv' and 'lme4'</td>
</tr>
<tr>
<td style="text-align: left;">gargle</td>
<td style="text-align: left;">1.5.2</td>
<td style="text-align: left;">Utilities for Working with Google APIs</td>
</tr>
<tr>
<td style="text-align: left;">gbm</td>
<td style="text-align: left;">2.2.2</td>
<td style="text-align: left;">Generalized Boosted Regression Models</td>
</tr>
<tr>
<td style="text-align: left;">gclus</td>
<td style="text-align: left;">1.3.3</td>
<td style="text-align: left;">Clustering Graphics</td>
</tr>
<tr>
<td style="text-align: left;">gcrma</td>
<td style="text-align: left;">2.80.0</td>
<td style="text-align: left;">Background Adjustment Using Sequence Information</td>
</tr>
<tr>
<td style="text-align: left;">gdata</td>
<td style="text-align: left;">3.0.1</td>
<td style="text-align: left;">Various R Programming Tools for Data Manipulation</td>
</tr>
<tr>
<td style="text-align: left;">gdsfmt</td>
<td style="text-align: left;">1.44.0</td>
<td style="text-align: left;">R Interface to CoreArray Genomic Data Structure (GDS) Files</td>
</tr>
<tr>
<td style="text-align: left;">gdtools</td>
<td style="text-align: left;">0.4.2</td>
<td style="text-align: left;">Utilities for Graphical Rendering and Fonts Management</td>
</tr>
<tr>
<td style="text-align: left;">geeM</td>
<td style="text-align: left;">0.10.1</td>
<td style="text-align: left;">Solve Generalized Estimating Equations</td>
</tr>
<tr>
<td style="text-align: left;">geepack</td>
<td style="text-align: left;">1.3.12</td>
<td style="text-align: left;">Generalized Estimating Equation Package</td>
</tr>
<tr>
<td style="text-align: left;">genefilter</td>
<td style="text-align: left;">1.90.0</td>
<td style="text-align: left;">genefilter: methods for filtering genes from high-throughput experiments</td>
</tr>
<tr>
<td style="text-align: left;">geneplotter</td>
<td style="text-align: left;">1.86.0</td>
<td style="text-align: left;">Graphics related functions for Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">generics</td>
<td style="text-align: left;">0.1.3</td>
<td style="text-align: left;">Common S3 Generics not Provided by Base R Methods Related to Model Fitting</td>
</tr>
<tr>
<td style="text-align: left;">GENIE3</td>
<td style="text-align: left;">1.30.0</td>
<td style="text-align: left;">GEne Network Inference with Ensemble of trees</td>
</tr>
<tr>
<td style="text-align: left;">genio</td>
<td style="text-align: left;">1.1.2</td>
<td style="text-align: left;">Genetics Input/Output Functions</td>
</tr>
<tr>
<td style="text-align: left;">GenomeInfoDb</td>
<td style="text-align: left;">1.44.0</td>
<td style="text-align: left;">Utilities for manipulating chromosome names, including modifying them to follow a particular naming style</td>
</tr>
<tr>
<td style="text-align: left;">GenomeInfoDbData</td>
<td style="text-align: left;">1.2.14</td>
<td style="text-align: left;">Species and taxonomy ID look up tables used by GenomeInfoDb</td>
</tr>
<tr>
<td style="text-align: left;">GenomicAlignments</td>
<td style="text-align: left;">1.44.0</td>
<td style="text-align: left;">Representation and manipulation of short genomic alignments</td>
</tr>
<tr>
<td style="text-align: left;">GenomicFeatures</td>
<td style="text-align: left;">1.60.0</td>
<td style="text-align: left;">Query the gene models of a given organism/assembly</td>
</tr>
<tr>
<td style="text-align: left;">GenomicRanges</td>
<td style="text-align: left;">1.60.0</td>
<td style="text-align: left;">Representation and manipulation of genomic intervals</td>
</tr>
<tr>
<td style="text-align: left;">GenSA</td>
<td style="text-align: left;">1.1.14.1</td>
<td style="text-align: left;">R Functions for Generalized Simulated Annealing</td>
</tr>
<tr>
<td style="text-align: left;">geojsonsf</td>
<td style="text-align: left;">2.0.3</td>
<td style="text-align: left;">GeoJSON to Simple Feature Converter</td>
</tr>
<tr>
<td style="text-align: left;">GEOmetadb</td>
<td style="text-align: left;">1.70.0</td>
<td style="text-align: left;">A compilation of metadata from NCBI GEO</td>
</tr>
<tr>
<td style="text-align: left;">geometries</td>
<td style="text-align: left;">0.2.4</td>
<td style="text-align: left;">Convert Between R Objects and Geometric Structures</td>
</tr>
<tr>
<td style="text-align: left;">GEOquery</td>
<td style="text-align: left;">2.76.0</td>
<td style="text-align: left;">Get data from NCBI Gene Expression Omnibus (GEO)</td>
</tr>
<tr>
<td style="text-align: left;">geos</td>
<td style="text-align: left;">0.2.4</td>
<td style="text-align: left;">Open Source Geometry Engine ('GEOS') R API</td>
</tr>
<tr>
<td style="text-align: left;">gert</td>
<td style="text-align: left;">2.1.5</td>
<td style="text-align: left;">Simple Git Client for R</td>
</tr>
<tr>
<td style="text-align: left;">getopt</td>
<td style="text-align: left;">1.20.4</td>
<td style="text-align: left;">C-Like 'getopt' Behavior</td>
</tr>
<tr>
<td style="text-align: left;">GetoptLong</td>
<td style="text-align: left;">1.0.5</td>
<td style="text-align: left;">Parsing Command-Line Arguments and Simple Variable Interpolation</td>
</tr>
<tr>
<td style="text-align: left;">gfonts</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">Offline 'Google' Fonts for 'Markdown' and 'Shiny'</td>
</tr>
<tr>
<td style="text-align: left;">ggalluvial</td>
<td style="text-align: left;">0.12.5</td>
<td style="text-align: left;">Alluvial Plots in 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">GGally</td>
<td style="text-align: left;">2.2.1</td>
<td style="text-align: left;">Extension to 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggbeeswarm</td>
<td style="text-align: left;">0.7.2</td>
<td style="text-align: left;">Categorical Scatter (Violin Point) Plots</td>
</tr>
<tr>
<td style="text-align: left;">ggbio</td>
<td style="text-align: left;">1.56.0</td>
<td style="text-align: left;">Visualization tools for genomic data</td>
</tr>
<tr>
<td style="text-align: left;">ggdendro</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">Create Dendrograms and Tree Diagrams Using 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggdist</td>
<td style="text-align: left;">3.3.3</td>
<td style="text-align: left;">Visualizations of Distributions and Uncertainty</td>
</tr>
<tr>
<td style="text-align: left;">ggforce</td>
<td style="text-align: left;">0.4.2</td>
<td style="text-align: left;">Accelerating 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggformula</td>
<td style="text-align: left;">0.12.0</td>
<td style="text-align: left;">Formula Interface to the Grammar of Graphics</td>
</tr>
<tr>
<td style="text-align: left;">ggfun</td>
<td style="text-align: left;">0.1.8</td>
<td style="text-align: left;">Miscellaneous Functions for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggnetwork</td>
<td style="text-align: left;">0.5.13</td>
<td style="text-align: left;">Geometries to Plot Networks with 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggnewscale</td>
<td style="text-align: left;">0.5.1</td>
<td style="text-align: left;">Multiple Fill and Colour Scales in 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggplot2</td>
<td style="text-align: left;">3.5.2</td>
<td style="text-align: left;">Create Elegant Data Visualisations Using the Grammar of Graphics</td>
</tr>
<tr>
<td style="text-align: left;">ggplotify</td>
<td style="text-align: left;">0.1.2</td>
<td style="text-align: left;">Convert Plot to 'grob' or 'ggplot' Object</td>
</tr>
<tr>
<td style="text-align: left;">ggpmisc</td>
<td style="text-align: left;">0.6.1</td>
<td style="text-align: left;">Miscellaneous Extensions to 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggpp</td>
<td style="text-align: left;">0.5.8-1</td>
<td style="text-align: left;">Grammar Extensions to 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggpubr</td>
<td style="text-align: left;">0.6.0</td>
<td style="text-align: left;">'ggplot2' Based Publication Ready Plots</td>
</tr>
<tr>
<td style="text-align: left;">ggraph</td>
<td style="text-align: left;">2.2.1</td>
<td style="text-align: left;">An Implementation of Grammar of Graphics for Graphs and Networks</td>
</tr>
<tr>
<td style="text-align: left;">ggrastr</td>
<td style="text-align: left;">1.0.2</td>
<td style="text-align: left;">Rasterize Layers for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggrepel</td>
<td style="text-align: left;">0.9.6</td>
<td style="text-align: left;">Automatically Position Non-Overlapping Text Labels with 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggridges</td>
<td style="text-align: left;">0.5.6</td>
<td style="text-align: left;">Ridgeline Plots in 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggsci</td>
<td style="text-align: left;">3.2.0</td>
<td style="text-align: left;">Scientific Journal and Sci-Fi Themed Color Palettes for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggsignif</td>
<td style="text-align: left;">0.6.4</td>
<td style="text-align: left;">Significance Brackets for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggstance</td>
<td style="text-align: left;">0.3.7</td>
<td style="text-align: left;">Horizontal 'ggplot2' Components</td>
</tr>
<tr>
<td style="text-align: left;">ggstats</td>
<td style="text-align: left;">0.9.0</td>
<td style="text-align: left;">Extension to 'ggplot2' for Plotting Stats</td>
</tr>
<tr>
<td style="text-align: left;">ggtangle</td>
<td style="text-align: left;">0.0.6</td>
<td style="text-align: left;">Draw Network with Data</td>
</tr>
<tr>
<td style="text-align: left;">ggtext</td>
<td style="text-align: left;">0.1.2</td>
<td style="text-align: left;">Improved Text Rendering Support for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggthemes</td>
<td style="text-align: left;">5.1.0</td>
<td style="text-align: left;">Extra Themes, Scales and Geoms for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">ggtree</td>
<td style="text-align: left;">3.16.0</td>
<td style="text-align: left;">an R package for visualization of tree and annotation data</td>
</tr>
<tr>
<td style="text-align: left;">ggVennDiagram</td>
<td style="text-align: left;">1.5.2</td>
<td style="text-align: left;">A 'ggplot2' Implement of Venn Diagram</td>
</tr>
<tr>
<td style="text-align: left;">gh</td>
<td style="text-align: left;">1.4.1</td>
<td style="text-align: left;">'GitHub' 'API'</td>
</tr>
<tr>
<td style="text-align: left;">Giotto</td>
<td style="text-align: left;">4.2.1</td>
<td style="text-align: left;">Spatial Single-Cell Transcriptomics Toolbox</td>
</tr>
<tr>
<td style="text-align: left;">GiottoClass</td>
<td style="text-align: left;">0.4.7</td>
<td style="text-align: left;">Giotto Suite Object Definitions and Framework</td>
</tr>
<tr>
<td style="text-align: left;">GiottoUtils</td>
<td style="text-align: left;">0.2.4</td>
<td style="text-align: left;">Giotto Suite Utilities</td>
</tr>
<tr>
<td style="text-align: left;">GiottoVisuals</td>
<td style="text-align: left;">0.2.12</td>
<td style="text-align: left;">Visuals for the Giotto spatial biology analysis ecosystem</td>
</tr>
<tr>
<td style="text-align: left;">gistr</td>
<td style="text-align: left;">0.9.0</td>
<td style="text-align: left;">Work with 'GitHub' 'Gists'</td>
</tr>
<tr>
<td style="text-align: left;">gitcreds</td>
<td style="text-align: left;">0.1.2</td>
<td style="text-align: left;">Query 'git' Credentials from 'R'</td>
</tr>
<tr>
<td style="text-align: left;">glasso</td>
<td style="text-align: left;">1.11</td>
<td style="text-align: left;">Graphical Lasso: Estimation of Gaussian Graphical Models</td>
</tr>
<tr>
<td style="text-align: left;">gld</td>
<td style="text-align: left;">2.6.7</td>
<td style="text-align: left;">Estimation and Use of the Generalised (Tukey) Lambda Distribution</td>
</tr>
<tr>
<td style="text-align: left;">Glimma</td>
<td style="text-align: left;">2.18.0</td>
<td style="text-align: left;">Interactive visualizations for gene expression analysis</td>
</tr>
<tr>
<td style="text-align: left;">glmGamPoi</td>
<td style="text-align: left;">1.20.0</td>
<td style="text-align: left;">Fit a Gamma-Poisson Generalized Linear Model</td>
</tr>
<tr>
<td style="text-align: left;">glmmTMB</td>
<td style="text-align: left;">1.1.11</td>
<td style="text-align: left;">Generalized Linear Mixed Models using Template Model Builder</td>
</tr>
<tr>
<td style="text-align: left;">glmnet</td>
<td style="text-align: left;">4.1-8</td>
<td style="text-align: left;">Lasso and Elastic-Net Regularized Generalized Linear Models</td>
</tr>
<tr>
<td style="text-align: left;">GlobalAncova</td>
<td style="text-align: left;">4.26.0</td>
<td style="text-align: left;">Global test for groups of variables via model comparisons</td>
</tr>
<tr>
<td style="text-align: left;">GlobalOptions</td>
<td style="text-align: left;">0.1.2</td>
<td style="text-align: left;">Generate Functions to Get or Set Global Options</td>
</tr>
<tr>
<td style="text-align: left;">globals</td>
<td style="text-align: left;">0.17.0</td>
<td style="text-align: left;">Identify Global Objects in R Expressions</td>
</tr>
<tr>
<td style="text-align: left;">globaltest</td>
<td style="text-align: left;">5.62.0</td>
<td style="text-align: left;">Testing Groups of Covariates/Features for Association with a Response Variable, with Applications to Gene Set Testing</td>
</tr>
<tr>
<td style="text-align: left;">glue</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Interpreted String Literals</td>
</tr>
<tr>
<td style="text-align: left;">gmp</td>
<td style="text-align: left;">0.7-5</td>
<td style="text-align: left;">Multiple Precision Arithmetic</td>
</tr>
<tr>
<td style="text-align: left;">GO.db</td>
<td style="text-align: left;">3.21.0</td>
<td style="text-align: left;">A set of annotation maps describing the entire Gene Ontology</td>
</tr>
<tr>
<td style="text-align: left;">goftest</td>
<td style="text-align: left;">1.2-3</td>
<td style="text-align: left;">Classical Goodness-of-Fit Tests for Univariate Distributions</td>
</tr>
<tr>
<td style="text-align: left;">googledrive</td>
<td style="text-align: left;">2.1.1</td>
<td style="text-align: left;">An Interface to Google Drive</td>
</tr>
<tr>
<td style="text-align: left;">googlesheets4</td>
<td style="text-align: left;">1.1.1</td>
<td style="text-align: left;">Access Google Sheets using the Sheets API V4</td>
</tr>
<tr>
<td style="text-align: left;">GOSemSim</td>
<td style="text-align: left;">2.34.0</td>
<td style="text-align: left;">GO-terms Semantic Similarity Measures</td>
</tr>
<tr>
<td style="text-align: left;">gower</td>
<td style="text-align: left;">1.0.2</td>
<td style="text-align: left;">Gower's Distance</td>
</tr>
<tr>
<td style="text-align: left;">GPArotation</td>
<td style="text-align: left;">2025.3-1</td>
<td style="text-align: left;">Gradient Projection Factor Rotation</td>
</tr>
<tr>
<td style="text-align: left;">GPfit</td>
<td style="text-align: left;">1.0-9</td>
<td style="text-align: left;">Gaussian Processes Modeling</td>
</tr>
<tr>
<td style="text-align: left;">gplots</td>
<td style="text-align: left;">3.2.0</td>
<td style="text-align: left;">Various R Programming Tools for Plotting Data</td>
</tr>
<tr>
<td style="text-align: left;">gprofiler2</td>
<td style="text-align: left;">0.2.3</td>
<td style="text-align: left;">Interface to the 'g:Profiler' Toolset</td>
</tr>
<tr>
<td style="text-align: left;">graph</td>
<td style="text-align: left;">1.86.0</td>
<td style="text-align: left;">graph: A package to handle graph data structures</td>
</tr>
<tr>
<td style="text-align: left;">graphics</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">The R Graphics Package</td>
</tr>
<tr>
<td style="text-align: left;">graphite</td>
<td style="text-align: left;">1.54.0</td>
<td style="text-align: left;">GRAPH Interaction from pathway Topological Environment</td>
</tr>
<tr>
<td style="text-align: left;">graphlayouts</td>
<td style="text-align: left;">1.2.2</td>
<td style="text-align: left;">Additional Layout Algorithms for Network Visualizations</td>
</tr>
<tr>
<td style="text-align: left;">grDevices</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">The R Graphics Devices and Support for Colours and Fonts</td>
</tr>
<tr>
<td style="text-align: left;">grid</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">The Grid Graphics Package</td>
</tr>
<tr>
<td style="text-align: left;">gridBase</td>
<td style="text-align: left;">0.4-7</td>
<td style="text-align: left;">Integration of base and grid graphics</td>
</tr>
<tr>
<td style="text-align: left;">gridExtra</td>
<td style="text-align: left;">2.3</td>
<td style="text-align: left;">Miscellaneous Functions for "Grid" Graphics</td>
</tr>
<tr>
<td style="text-align: left;">gridGraphics</td>
<td style="text-align: left;">0.5-1</td>
<td style="text-align: left;">Redraw Base Graphics Using 'grid' Graphics</td>
</tr>
<tr>
<td style="text-align: left;">gridtext</td>
<td style="text-align: left;">0.1.5</td>
<td style="text-align: left;">Improved Text Rendering Support for 'Grid' Graphics</td>
</tr>
<tr>
<td style="text-align: left;">grpreg</td>
<td style="text-align: left;">3.5.0</td>
<td style="text-align: left;">Regularization Paths for Regression Models with Grouped Covariates</td>
</tr>
<tr>
<td style="text-align: left;">grr</td>
<td style="text-align: left;">0.9.5</td>
<td style="text-align: left;">Alternative Implementations of Base R Functions</td>
</tr>
<tr>
<td style="text-align: left;">GSA</td>
<td style="text-align: left;">1.03.3</td>
<td style="text-align: left;">Gene Set Analysis</td>
</tr>
<tr>
<td style="text-align: left;">GSEABase</td>
<td style="text-align: left;">1.70.0</td>
<td style="text-align: left;">Gene set enrichment data structures and methods</td>
</tr>
<tr>
<td style="text-align: left;">gson</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">Base Class and Methods for 'gson' Format</td>
</tr>
<tr>
<td style="text-align: left;">gstat</td>
<td style="text-align: left;">2.1-3</td>
<td style="text-align: left;">Spatial and Spatio-Temporal Geostatistical Modelling, Prediction and Simulation</td>
</tr>
<tr>
<td style="text-align: left;">gsubfn</td>
<td style="text-align: left;">0.7</td>
<td style="text-align: left;">Utilities for Strings and Function Arguments</td>
</tr>
<tr>
<td style="text-align: left;">GSVA</td>
<td style="text-align: left;">2.2.0</td>
<td style="text-align: left;">Gene Set Variation Analysis for Microarray and RNA-Seq Data</td>
</tr>
<tr>
<td style="text-align: left;">gt</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">Easily Create Presentation-Ready Display Tables</td>
</tr>
<tr>
<td style="text-align: left;">gtable</td>
<td style="text-align: left;">0.3.6</td>
<td style="text-align: left;">Arrange 'Grobs' in Tables</td>
</tr>
<tr>
<td style="text-align: left;">gtools</td>
<td style="text-align: left;">3.9.5</td>
<td style="text-align: left;">Various R Programming Tools</td>
</tr>
<tr>
<td style="text-align: left;">gtsummary</td>
<td style="text-align: left;">2.2.0</td>
<td style="text-align: left;">Presentation-Ready Data Summary and Analytic Result Tables</td>
</tr>
<tr>
<td style="text-align: left;">GWASExactHW</td>
<td style="text-align: left;">1.2</td>
<td style="text-align: left;">Exact Hardy-Weinburg Testing for Genome Wide Association Studies</td>
</tr>
<tr>
<td style="text-align: left;">GWASTools</td>
<td style="text-align: left;">1.54.0</td>
<td style="text-align: left;">Tools for Genome Wide Association Studies</td>
</tr>
<tr>
<td style="text-align: left;">gwasurvivr</td>
<td style="text-align: left;">1.26.0</td>
<td style="text-align: left;">gwasurvivr: an R package for genome wide survival analysis</td>
</tr>
<tr>
<td style="text-align: left;">gypsum</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Interface to the gypsum REST API</td>
</tr>
<tr>
<td style="text-align: left;">h2o</td>
<td style="text-align: left;">3.44.0.3</td>
<td style="text-align: left;">R Interface for the 'H2O' Scalable Machine Learning Platform</td>
</tr>
<tr>
<td style="text-align: left;">h5mread</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">A fast HDF5 reader</td>
</tr>
<tr>
<td style="text-align: left;">hardhat</td>
<td style="text-align: left;">1.4.1</td>
<td style="text-align: left;">Construct Modeling Packages</td>
</tr>
<tr>
<td style="text-align: left;">harmony</td>
<td style="text-align: left;">1.2.3</td>
<td style="text-align: left;">Fast, Sensitive, and Accurate Integration of Single Cell Data</td>
</tr>
<tr>
<td style="text-align: left;">hash</td>
<td style="text-align: left;">2.2.6.3</td>
<td style="text-align: left;">Full Featured Implementation of Hash Tables/Associative Arrays/Dictionaries</td>
</tr>
<tr>
<td style="text-align: left;">haven</td>
<td style="text-align: left;">2.5.4</td>
<td style="text-align: left;">Import and Export 'SPSS', 'Stata' and 'SAS' Files</td>
</tr>
<tr>
<td style="text-align: left;">HDF5Array</td>
<td style="text-align: left;">1.36.0</td>
<td style="text-align: left;">HDF5 datasets as array-like objects in R</td>
</tr>
<tr>
<td style="text-align: left;">hdf5r</td>
<td style="text-align: left;">1.3.12</td>
<td style="text-align: left;">Interface to the 'HDF5' Binary Data Format</td>
</tr>
<tr>
<td style="text-align: left;">HDO.db</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">A set of annotation maps describing the entire Human Disease Ontology</td>
</tr>
<tr>
<td style="text-align: left;">heatmaply</td>
<td style="text-align: left;">1.5.0</td>
<td style="text-align: left;">Interactive Cluster Heat Maps Using 'plotly' and 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">heplots</td>
<td style="text-align: left;">1.7.4</td>
<td style="text-align: left;">Visualizing Hypothesis Tests in Multivariate Linear Models</td>
</tr>
<tr>
<td style="text-align: left;">here</td>
<td style="text-align: left;">1.0.1</td>
<td style="text-align: left;">A Simpler Way to Find Your Files</td>
</tr>
<tr>
<td style="text-align: left;">hexbin</td>
<td style="text-align: left;">1.28.5</td>
<td style="text-align: left;">Hexagonal Binning Routines</td>
</tr>
<tr>
<td style="text-align: left;">HGNChelper</td>
<td style="text-align: left;">0.8.15</td>
<td style="text-align: left;">Identify and Correct Invalid HGNC Human Gene Symbols and MGI Mouse Gene Symbols</td>
</tr>
<tr>
<td style="text-align: left;">HiddenMarkov</td>
<td style="text-align: left;">1.8-14</td>
<td style="text-align: left;">Hidden Markov Models</td>
</tr>
<tr>
<td style="text-align: left;">highr</td>
<td style="text-align: left;">0.11</td>
<td style="text-align: left;">Syntax Highlighting for R Source Code</td>
</tr>
<tr>
<td style="text-align: left;">Hmisc</td>
<td style="text-align: left;">5.2-3</td>
<td style="text-align: left;">Harrell Miscellaneous</td>
</tr>
<tr>
<td style="text-align: left;">hms</td>
<td style="text-align: left;">1.1.3</td>
<td style="text-align: left;">Pretty Time of Day</td>
</tr>
<tr>
<td style="text-align: left;">hrbrthemes</td>
<td style="text-align: left;">0.8.7</td>
<td style="text-align: left;">Additional Themes, Theme Components and Utilities for 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">HSMMSingleCell</td>
<td style="text-align: left;">1.28.0</td>
<td style="text-align: left;">Single-cell RNA-Seq for differentiating human skeletal muscle myoblasts (HSMM)</td>
</tr>
<tr>
<td style="text-align: left;">htmlTable</td>
<td style="text-align: left;">2.4.3</td>
<td style="text-align: left;">Advanced Tables for Markdown/HTML</td>
</tr>
<tr>
<td style="text-align: left;">htmltools</td>
<td style="text-align: left;">0.5.8.1</td>
<td style="text-align: left;">Tools for HTML</td>
</tr>
<tr>
<td style="text-align: left;">htmlwidgets</td>
<td style="text-align: left;">1.6.4</td>
<td style="text-align: left;">HTML Widgets for R</td>
</tr>
<tr>
<td style="text-align: left;">httpcode</td>
<td style="text-align: left;">0.3.0</td>
<td style="text-align: left;">'HTTP' Status Code Helper</td>
</tr>
<tr>
<td style="text-align: left;">httpuv</td>
<td style="text-align: left;">1.6.16</td>
<td style="text-align: left;">HTTP and WebSocket Server Library</td>
</tr>
<tr>
<td style="text-align: left;">httr</td>
<td style="text-align: left;">1.4.7</td>
<td style="text-align: left;">Tools for Working with URLs and HTTP</td>
</tr>
<tr>
<td style="text-align: left;">httr2</td>
<td style="text-align: left;">1.1.2</td>
<td style="text-align: left;">Perform HTTP Requests and Process the Responses</td>
</tr>
<tr>
<td style="text-align: left;">hwriter</td>
<td style="text-align: left;">1.3.2.1</td>
<td style="text-align: left;">HTML Writer - Outputs R Objects in HTML Format</td>
</tr>
<tr>
<td style="text-align: left;">ica</td>
<td style="text-align: left;">1.0-3</td>
<td style="text-align: left;">Independent Component Analysis</td>
</tr>
<tr>
<td style="text-align: left;">ids</td>
<td style="text-align: left;">1.0.1</td>
<td style="text-align: left;">Generate Random Identifiers</td>
</tr>
<tr>
<td style="text-align: left;">igraph</td>
<td style="text-align: left;">2.1.4</td>
<td style="text-align: left;">Network Analysis and Visualization</td>
</tr>
<tr>
<td style="text-align: left;">imager</td>
<td style="text-align: left;">1.0.3</td>
<td style="text-align: left;">Image Processing Library Based on 'CImg'</td>
</tr>
<tr>
<td style="text-align: left;">iml</td>
<td style="text-align: left;">0.11.4</td>
<td style="text-align: left;">Interpretable Machine Learning</td>
</tr>
<tr>
<td style="text-align: left;">impute</td>
<td style="text-align: left;">1.82.0</td>
<td style="text-align: left;">impute: Imputation for microarray data</td>
</tr>
<tr>
<td style="text-align: left;">infer</td>
<td style="text-align: left;">1.0.8</td>
<td style="text-align: left;">Tidy Statistical Inference</td>
</tr>
<tr>
<td style="text-align: left;">infercnv</td>
<td style="text-align: left;">1.24.0</td>
<td style="text-align: left;">Infer Copy Number Variation from Single-Cell RNA-Seq Data</td>
</tr>
<tr>
<td style="text-align: left;">influenceR</td>
<td style="text-align: left;">0.1.5</td>
<td style="text-align: left;">Software Tools to Quantify Structural Importance of Nodes in a Network</td>
</tr>
<tr>
<td style="text-align: left;">ini</td>
<td style="text-align: left;">0.3.1</td>
<td style="text-align: left;">Read and Write '.ini' Files</td>
</tr>
<tr>
<td style="text-align: left;">inline</td>
<td style="text-align: left;">0.3.21</td>
<td style="text-align: left;">Functions to Inline C, C++, Fortran Function Calls from R</td>
</tr>
<tr>
<td style="text-align: left;">insight</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">Easy Access to Model Information for Various Model Objects</td>
</tr>
<tr>
<td style="text-align: left;">interactiveDisplayBase</td>
<td style="text-align: left;">1.46.0</td>
<td style="text-align: left;">Base package for enabling powerful shiny web displays of Bioconductor objects</td>
</tr>
<tr>
<td style="text-align: left;">interp</td>
<td style="text-align: left;">1.1-6</td>
<td style="text-align: left;">Interpolation Methods</td>
</tr>
<tr>
<td style="text-align: left;">intervals</td>
<td style="text-align: left;">0.15.5</td>
<td style="text-align: left;">Tools for Working with Points and Intervals</td>
</tr>
<tr>
<td style="text-align: left;">invgamma</td>
<td style="text-align: left;">1.1</td>
<td style="text-align: left;">The Inverse Gamma Distribution</td>
</tr>
<tr>
<td style="text-align: left;">iotools</td>
<td style="text-align: left;">0.3-5</td>
<td style="text-align: left;">I/O Tools for Streaming</td>
</tr>
<tr>
<td style="text-align: left;">ipred</td>
<td style="text-align: left;">0.9-15</td>
<td style="text-align: left;">Improved Predictors</td>
</tr>
<tr>
<td style="text-align: left;">IRanges</td>
<td style="text-align: left;">2.42.0</td>
<td style="text-align: left;">Foundation of integer range manipulation in Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">IRdisplay</td>
<td style="text-align: left;">1.1</td>
<td style="text-align: left;">'Jupyter' Display Machinery</td>
</tr>
<tr>
<td style="text-align: left;">IRkernel</td>
<td style="text-align: left;">1.3.2</td>
<td style="text-align: left;">Native R Kernel for the 'Jupyter Notebook'</td>
</tr>
<tr>
<td style="text-align: left;">irlba</td>
<td style="text-align: left;">2.3.5.1</td>
<td style="text-align: left;">Fast Truncated Singular Value Decomposition and Principal Components Analysis for Large Dense and Sparse Matrices</td>
</tr>
<tr>
<td style="text-align: left;">IsingFit</td>
<td style="text-align: left;">0.4</td>
<td style="text-align: left;">Fitting Ising Models Using the ELasso Method</td>
</tr>
<tr>
<td style="text-align: left;">IsingSampler</td>
<td style="text-align: left;">0.2.3</td>
<td style="text-align: left;">Sampling Methods and Distribution Functions for the Ising Model</td>
</tr>
<tr>
<td style="text-align: left;">isoband</td>
<td style="text-align: left;">0.2.7</td>
<td style="text-align: left;">Generate Isolines and Isobands from Regularly Spaced Elevation Grids</td>
</tr>
<tr>
<td style="text-align: left;">iterators</td>
<td style="text-align: left;">1.0.14</td>
<td style="text-align: left;">Provides Iterator Construct</td>
</tr>
<tr>
<td style="text-align: left;">ITKR</td>
<td style="text-align: left;">0.6.0.0.2</td>
<td style="text-align: left;">ITK in R</td>
</tr>
<tr>
<td style="text-align: left;">janitor</td>
<td style="text-align: left;">2.2.1</td>
<td style="text-align: left;">Simple Tools for Examining and Cleaning Dirty Data</td>
</tr>
<tr>
<td style="text-align: left;">JASPAR2020</td>
<td style="text-align: left;">0.99.10</td>
<td style="text-align: left;">Data package for JASPAR database (version 2020)</td>
</tr>
<tr>
<td style="text-align: left;">jomo</td>
<td style="text-align: left;">2.7-6</td>
<td style="text-align: left;">Multilevel Joint Modelling Multiple Imputation</td>
</tr>
<tr>
<td style="text-align: left;">jpeg</td>
<td style="text-align: left;">0.1-11</td>
<td style="text-align: left;">Read and write JPEG images</td>
</tr>
<tr>
<td style="text-align: left;">jquerylib</td>
<td style="text-align: left;">0.1.4</td>
<td style="text-align: left;">Obtain 'jQuery' as an HTML Dependency Object</td>
</tr>
<tr>
<td style="text-align: left;">jsonify</td>
<td style="text-align: left;">1.2.2</td>
<td style="text-align: left;">Convert Between 'R' Objects and Javascript Object Notation (JSON)</td>
</tr>
<tr>
<td style="text-align: left;">jsonlite</td>
<td style="text-align: left;">2.0.0</td>
<td style="text-align: left;">A Simple and Robust JSON Parser and Generator for R</td>
</tr>
<tr>
<td style="text-align: left;">jsonvalidate</td>
<td style="text-align: left;">1.5.0</td>
<td style="text-align: left;">Validate 'JSON' Schema</td>
</tr>
<tr>
<td style="text-align: left;">juicyjuice</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">Inline CSS Properties into HTML Tags Using 'juice'</td>
</tr>
<tr>
<td style="text-align: left;">kableExtra</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Construct Complex Table with 'kable' and Pipe Syntax</td>
</tr>
<tr>
<td style="text-align: left;">KEGGgraph</td>
<td style="text-align: left;">1.68.0</td>
<td style="text-align: left;">KEGGgraph: A graph approach to KEGG PATHWAY in R and Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">KEGGREST</td>
<td style="text-align: left;">1.48.0</td>
<td style="text-align: left;">Client-side REST access to the Kyoto Encyclopedia of Genes and Genomes (KEGG)</td>
</tr>
<tr>
<td style="text-align: left;">keras3</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">R Interface to 'Keras'</td>
</tr>
<tr>
<td style="text-align: left;">kernlab</td>
<td style="text-align: left;">0.9-33</td>
<td style="text-align: left;">Kernel-Based Machine Learning Lab</td>
</tr>
<tr>
<td style="text-align: left;">KernSmooth</td>
<td style="text-align: left;">2.23-26</td>
<td style="text-align: left;">Functions for Kernel Smoothing Supporting Wand &amp; Jones (1995)</td>
</tr>
<tr>
<td style="text-align: left;">kinship2</td>
<td style="text-align: left;">1.9.6.1</td>
<td style="text-align: left;">Pedigree Functions</td>
</tr>
<tr>
<td style="text-align: left;">km.ci</td>
<td style="text-align: left;">0.5-6</td>
<td style="text-align: left;">Confidence Intervals for the Kaplan-Meier Estimator</td>
</tr>
<tr>
<td style="text-align: left;">KMsurv</td>
<td style="text-align: left;">0.1-5</td>
<td style="text-align: left;">Data sets from Klein and Moeschberger (1997), Survival Analysis</td>
</tr>
<tr>
<td style="text-align: left;">knitr</td>
<td style="text-align: left;">1.50</td>
<td style="text-align: left;">A General-Purpose Package for Dynamic Report Generation in R</td>
</tr>
<tr>
<td style="text-align: left;">ks</td>
<td style="text-align: left;">1.14.3</td>
<td style="text-align: left;">Kernel Smoothing</td>
</tr>
<tr>
<td style="text-align: left;">kutils</td>
<td style="text-align: left;">1.73</td>
<td style="text-align: left;">Project Management Tools</td>
</tr>
<tr>
<td style="text-align: left;">labeling</td>
<td style="text-align: left;">0.4.3</td>
<td style="text-align: left;">Axis Labeling</td>
</tr>
<tr>
<td style="text-align: left;">labelled</td>
<td style="text-align: left;">2.14.0</td>
<td style="text-align: left;">Manipulating Labelled Data</td>
</tr>
<tr>
<td style="text-align: left;">laeken</td>
<td style="text-align: left;">0.5.3</td>
<td style="text-align: left;">Estimation of Indicators on Social Exclusion and Poverty</td>
</tr>
<tr>
<td style="text-align: left;">lambda.r</td>
<td style="text-align: left;">1.2.4</td>
<td style="text-align: left;">Modeling Data with Functional Programming</td>
</tr>
<tr>
<td style="text-align: left;">later</td>
<td style="text-align: left;">1.4.2</td>
<td style="text-align: left;">Utilities for Scheduling Functions to Execute Later with Event Loops</td>
</tr>
<tr>
<td style="text-align: left;">lattice</td>
<td style="text-align: left;">0.22-7</td>
<td style="text-align: left;">Trellis Graphics for R</td>
</tr>
<tr>
<td style="text-align: left;">latticeExtra</td>
<td style="text-align: left;">0.6-30</td>
<td style="text-align: left;">Extra Graphical Utilities Based on Lattice</td>
</tr>
<tr>
<td style="text-align: left;">lava</td>
<td style="text-align: left;">1.8.1</td>
<td style="text-align: left;">Latent Variable Models</td>
</tr>
<tr>
<td style="text-align: left;">lavaan</td>
<td style="text-align: left;">0.6-19</td>
<td style="text-align: left;">Latent Variable Analysis</td>
</tr>
<tr>
<td style="text-align: left;">lazyeval</td>
<td style="text-align: left;">0.2.2</td>
<td style="text-align: left;">Lazy (Non-Standard) Evaluation</td>
</tr>
<tr>
<td style="text-align: left;">leafem</td>
<td style="text-align: left;">0.2.3</td>
<td style="text-align: left;">'leaflet' Extensions for 'mapview'</td>
</tr>
<tr>
<td style="text-align: left;">leafgl</td>
<td style="text-align: left;">0.2.2</td>
<td style="text-align: left;">High-Performance 'WebGl' Rendering for Package 'leaflet'</td>
</tr>
<tr>
<td style="text-align: left;">leaflegend</td>
<td style="text-align: left;">1.2.1</td>
<td style="text-align: left;">Add Custom Legends to 'leaflet' Maps</td>
</tr>
<tr>
<td style="text-align: left;">leaflet</td>
<td style="text-align: left;">2.2.2</td>
<td style="text-align: left;">Create Interactive Web Maps with the JavaScript 'Leaflet' Library</td>
</tr>
<tr>
<td style="text-align: left;">leaflet.providers</td>
<td style="text-align: left;">2.0.0</td>
<td style="text-align: left;">Leaflet Providers</td>
</tr>
<tr>
<td style="text-align: left;">leafpop</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">Include Tables, Images and Graphs in Leaflet Pop-Ups</td>
</tr>
<tr>
<td style="text-align: left;">leafsync</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">Small Multiples for Leaflet Web Maps</td>
</tr>
<tr>
<td style="text-align: left;">leaps</td>
<td style="text-align: left;">3.2</td>
<td style="text-align: left;">Regression Subset Selection</td>
</tr>
<tr>
<td style="text-align: left;">leiden</td>
<td style="text-align: left;">0.4.3.1</td>
<td style="text-align: left;">R Implementation of Leiden Clustering Algorithm</td>
</tr>
<tr>
<td style="text-align: left;">leidenbase</td>
<td style="text-align: left;">0.1.35</td>
<td style="text-align: left;">R and C/C++ Wrappers to Run the Leiden find_partition() Function</td>
</tr>
<tr>
<td style="text-align: left;">lfa</td>
<td style="text-align: left;">2.8.0</td>
<td style="text-align: left;">Logistic Factor Analysis for Categorical Data</td>
</tr>
<tr>
<td style="text-align: left;">lhs</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">Latin Hypercube Samples</td>
</tr>
<tr>
<td style="text-align: left;">libcoin</td>
<td style="text-align: left;">1.0-10</td>
<td style="text-align: left;">Linear Test Statistics for Permutation Inference</td>
</tr>
<tr>
<td style="text-align: left;">libgeos</td>
<td style="text-align: left;">3.11.1-3</td>
<td style="text-align: left;">Open Source Geometry Engine ('GEOS') C API</td>
</tr>
<tr>
<td style="text-align: left;">lifecycle</td>
<td style="text-align: left;">1.0.4</td>
<td style="text-align: left;">Manage the Life Cycle of your Package Functions</td>
</tr>
<tr>
<td style="text-align: left;">limma</td>
<td style="text-align: left;">3.64.3</td>
<td style="text-align: left;">Linear Models for Microarray and Omics Data</td>
</tr>
<tr>
<td style="text-align: left;">linkcomm</td>
<td style="text-align: left;">1.0-14</td>
<td style="text-align: left;">Tools for Generating, Visualizing, and Analysing Link Communities in Networks</td>
</tr>
<tr>
<td style="text-align: left;">lisrelToR</td>
<td style="text-align: left;">0.3</td>
<td style="text-align: left;">Import Output from LISREL into R</td>
</tr>
<tr>
<td style="text-align: left;">listenv</td>
<td style="text-align: left;">0.9.1</td>
<td style="text-align: left;">Environments Behaving (Almost) as Lists</td>
</tr>
<tr>
<td style="text-align: left;">litedown</td>
<td style="text-align: left;">0.7</td>
<td style="text-align: left;">A Lightweight Version of R Markdown</td>
</tr>
<tr>
<td style="text-align: left;">lme4</td>
<td style="text-align: left;">1.1-37</td>
<td style="text-align: left;">Linear Mixed-Effects Models using 'Eigen' and S4</td>
</tr>
<tr>
<td style="text-align: left;">lmerTest</td>
<td style="text-align: left;">3.1-3</td>
<td style="text-align: left;">Tests in Linear Mixed Effects Models</td>
</tr>
<tr>
<td style="text-align: left;">lmodel2</td>
<td style="text-align: left;">1.7-4</td>
<td style="text-align: left;">Model II Regression</td>
</tr>
<tr>
<td style="text-align: left;">lmom</td>
<td style="text-align: left;">3.2</td>
<td style="text-align: left;">L-Moments</td>
</tr>
<tr>
<td style="text-align: left;">lmtest</td>
<td style="text-align: left;">0.9-40</td>
<td style="text-align: left;">Testing Linear Regression Models</td>
</tr>
<tr>
<td style="text-align: left;">lobstr</td>
<td style="text-align: left;">1.1.2</td>
<td style="text-align: left;">Visualize R Data Structures with Trees</td>
</tr>
<tr>
<td style="text-align: left;">locfdr</td>
<td style="text-align: left;">1.1-8</td>
<td style="text-align: left;">Computes Local False Discovery Rates</td>
</tr>
<tr>
<td style="text-align: left;">locfit</td>
<td style="text-align: left;">1.5-9.12</td>
<td style="text-align: left;">Local Regression, Likelihood and Density Estimation</td>
</tr>
<tr>
<td style="text-align: left;">logger</td>
<td style="text-align: left;">0.4.0</td>
<td style="text-align: left;">A Lightweight, Modern and Flexible Logging Utility</td>
</tr>
<tr>
<td style="text-align: left;">logistf</td>
<td style="text-align: left;">1.26.1</td>
<td style="text-align: left;">Firth's Bias-Reduced Logistic Regression</td>
</tr>
<tr>
<td style="text-align: left;">loo</td>
<td style="text-align: left;">2.8.0</td>
<td style="text-align: left;">Efficient Leave-One-Out Cross-Validation and WAIC for Bayesian Models</td>
</tr>
<tr>
<td style="text-align: left;">loupeR</td>
<td style="text-align: left;">1.1.4</td>
<td style="text-align: left;">Converts Seurat objects to 10x Genomics Loupe files</td>
</tr>
<tr>
<td style="text-align: left;">lubridate</td>
<td style="text-align: left;">1.9.4</td>
<td style="text-align: left;">Make Dealing with Dates a Little Easier</td>
</tr>
<tr>
<td style="text-align: left;">lwgeom</td>
<td style="text-align: left;">0.2-14</td>
<td style="text-align: left;">Bindings to Selected 'liblwgeom' Functions for Simple Features</td>
</tr>
<tr>
<td style="text-align: left;">magick</td>
<td style="text-align: left;">2.8.6</td>
<td style="text-align: left;">Advanced Graphics and Image-Processing in R</td>
</tr>
<tr>
<td style="text-align: left;">magrittr</td>
<td style="text-align: left;">2.0.3</td>
<td style="text-align: left;">A Forward-Pipe Operator for R</td>
</tr>
<tr>
<td style="text-align: left;">MALDIquant</td>
<td style="text-align: left;">1.22.3</td>
<td style="text-align: left;">Quantitative Analysis of Mass Spectrometry Data</td>
</tr>
<tr>
<td style="text-align: left;">manhattanly</td>
<td style="text-align: left;">0.3.0</td>
<td style="text-align: left;">Interactive Q-Q and Manhattan Plots Using 'plotly.js'</td>
</tr>
<tr>
<td style="text-align: left;">manipulateWidget</td>
<td style="text-align: left;">0.11.1</td>
<td style="text-align: left;">Add Even More Interactivity to Interactive Charts</td>
</tr>
<tr>
<td style="text-align: left;">mapproj</td>
<td style="text-align: left;">1.2.11</td>
<td style="text-align: left;">Map Projections</td>
</tr>
<tr>
<td style="text-align: left;">maps</td>
<td style="text-align: left;">3.4.2.1</td>
<td style="text-align: left;">Draw Geographical Maps</td>
</tr>
<tr>
<td style="text-align: left;">mapview</td>
<td style="text-align: left;">2.11.2</td>
<td style="text-align: left;">Interactive Viewing of Spatial Data in R</td>
</tr>
<tr>
<td style="text-align: left;">marginaleffects</td>
<td style="text-align: left;">0.25.1</td>
<td style="text-align: left;">Predictions, Comparisons, Slopes, Marginal Means, and Hypothesis Tests</td>
</tr>
<tr>
<td style="text-align: left;">markdown</td>
<td style="text-align: left;">2.0</td>
<td style="text-align: left;">Render Markdown with 'commonmark'</td>
</tr>
<tr>
<td style="text-align: left;">MASS</td>
<td style="text-align: left;">7.3-65</td>
<td style="text-align: left;">Support Functions and Datasets for Venables and Ripley's MASS</td>
</tr>
<tr>
<td style="text-align: left;">mathjaxr</td>
<td style="text-align: left;">1.6-0</td>
<td style="text-align: left;">Using 'Mathjax' in Rd Files</td>
</tr>
<tr>
<td style="text-align: left;">Matrix</td>
<td style="text-align: left;">1.7-3</td>
<td style="text-align: left;">Sparse and Dense Matrix Classes and Methods</td>
</tr>
<tr>
<td style="text-align: left;">matrixcalc</td>
<td style="text-align: left;">1.0-6</td>
<td style="text-align: left;">Collection of Functions for Matrix Calculations</td>
</tr>
<tr>
<td style="text-align: left;">MatrixGenerics</td>
<td style="text-align: left;">1.20.0</td>
<td style="text-align: left;">S4 Generic Summary Statistic Functions that Operate on Matrix-Like Objects</td>
</tr>
<tr>
<td style="text-align: left;">MatrixModels</td>
<td style="text-align: left;">0.5-4</td>
<td style="text-align: left;">Modelling with Sparse and Dense Matrices</td>
</tr>
<tr>
<td style="text-align: left;">matrixStats</td>
<td style="text-align: left;">1.5.0</td>
<td style="text-align: left;">Functions that Apply to Rows and Columns of Matrices (and to Vectors)</td>
</tr>
<tr>
<td style="text-align: left;">maxLik</td>
<td style="text-align: left;">1.5-2.1</td>
<td style="text-align: left;">Maximum Likelihood Estimation and Related Tools</td>
</tr>
<tr>
<td style="text-align: left;">maxstat</td>
<td style="text-align: left;">0.7-25</td>
<td style="text-align: left;">Maximally Selected Rank Statistics</td>
</tr>
<tr>
<td style="text-align: left;">mbend</td>
<td style="text-align: left;">1.3.1</td>
<td style="text-align: left;">Matrix Bending</td>
</tr>
<tr>
<td style="text-align: left;">MCL</td>
<td style="text-align: left;">1.0</td>
<td style="text-align: left;">Markov Cluster Algorithm</td>
</tr>
<tr>
<td style="text-align: left;">mclogit</td>
<td style="text-align: left;">0.9.6</td>
<td style="text-align: left;">Multinomial Logit Models, with or without Random Effects or Overdispersion</td>
</tr>
<tr>
<td style="text-align: left;">mclust</td>
<td style="text-align: left;">6.1.1</td>
<td style="text-align: left;">Gaussian Mixture Modelling for Model-Based Clustering, Classification, and Density Estimation</td>
</tr>
<tr>
<td style="text-align: left;">memisc</td>
<td style="text-align: left;">0.99.31.8.3</td>
<td style="text-align: left;">Management of Survey Data and Presentation of Analysis Results</td>
</tr>
<tr>
<td style="text-align: left;">memoise</td>
<td style="text-align: left;">2.0.1</td>
<td style="text-align: left;">'Memoisation' of Functions</td>
</tr>
<tr>
<td style="text-align: left;">memuse</td>
<td style="text-align: left;">4.2-3</td>
<td style="text-align: left;">Memory Estimation Utilities</td>
</tr>
<tr>
<td style="text-align: left;">MESS</td>
<td style="text-align: left;">0.5.12</td>
<td style="text-align: left;">Miscellaneous Esoteric Statistical Scripts</td>
</tr>
<tr>
<td style="text-align: left;">MetaboAnalystR</td>
<td style="text-align: left;">4.2.0</td>
<td style="text-align: left;">An R Package for Comprehensive Analysis of Metabolomics Data</td>
</tr>
<tr>
<td style="text-align: left;">MetaboCoreUtils</td>
<td style="text-align: left;">1.16.1</td>
<td style="text-align: left;">Core Utils for Metabolomics Data</td>
</tr>
<tr>
<td style="text-align: left;">metadat</td>
<td style="text-align: left;">1.4-0</td>
<td style="text-align: left;">Meta-Analysis Datasets</td>
</tr>
<tr>
<td style="text-align: left;">metafor</td>
<td style="text-align: left;">4.8-0</td>
<td style="text-align: left;">Meta-Analysis Package for R</td>
</tr>
<tr>
<td style="text-align: left;">metapod</td>
<td style="text-align: left;">1.16.0</td>
<td style="text-align: left;">Meta-Analyses on P-Values of Differential Analyses</td>
</tr>
<tr>
<td style="text-align: left;">methods</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">Formal Methods and Classes</td>
</tr>
<tr>
<td style="text-align: left;">Metrics</td>
<td style="text-align: left;">0.1.4</td>
<td style="text-align: left;">Evaluation Metrics for Machine Learning</td>
</tr>
<tr>
<td style="text-align: left;">mets</td>
<td style="text-align: left;">1.3.6</td>
<td style="text-align: left;">Analysis of Multivariate Event Times</td>
</tr>
<tr>
<td style="text-align: left;">mgcv</td>
<td style="text-align: left;">1.9-3</td>
<td style="text-align: left;">Mixed GAM Computation Vehicle with Automatic Smoothness Estimation</td>
</tr>
<tr>
<td style="text-align: left;">mgm</td>
<td style="text-align: left;">1.2-15</td>
<td style="text-align: left;">Estimating Time-Varying k-Order Mixed Graphical Models</td>
</tr>
<tr>
<td style="text-align: left;">mi</td>
<td style="text-align: left;">1.1</td>
<td style="text-align: left;">Missing Data Imputation and Model Checking</td>
</tr>
<tr>
<td style="text-align: left;">mice</td>
<td style="text-align: left;">3.17.0</td>
<td style="text-align: left;">Multivariate Imputation by Chained Equations</td>
</tr>
<tr>
<td style="text-align: left;">microbenchmark</td>
<td style="text-align: left;">1.5.0</td>
<td style="text-align: left;">Accurate Timing Functions</td>
</tr>
<tr>
<td style="text-align: left;">mime</td>
<td style="text-align: left;">0.13</td>
<td style="text-align: left;">Map Filenames to MIME Types</td>
</tr>
<tr>
<td style="text-align: left;">miniUI</td>
<td style="text-align: left;">0.1.2</td>
<td style="text-align: left;">Shiny UI Widgets for Small Screens</td>
</tr>
<tr>
<td style="text-align: left;">minqa</td>
<td style="text-align: left;">1.2.8</td>
<td style="text-align: left;">Derivative-Free Optimization Algorithms by Quadratic Approximation</td>
</tr>
<tr>
<td style="text-align: left;">misc3d</td>
<td style="text-align: left;">0.9-1</td>
<td style="text-align: left;">Miscellaneous 3D Plots</td>
</tr>
<tr>
<td style="text-align: left;">miscTools</td>
<td style="text-align: left;">0.6-28</td>
<td style="text-align: left;">Miscellaneous Tools and Utilities</td>
</tr>
<tr>
<td style="text-align: left;">mitml</td>
<td style="text-align: left;">0.4-5</td>
<td style="text-align: left;">Tools for Multiple Imputation in Multilevel Modeling</td>
</tr>
<tr>
<td style="text-align: left;">mitools</td>
<td style="text-align: left;">2.4</td>
<td style="text-align: left;">Tools for Multiple Imputation of Missing Data</td>
</tr>
<tr>
<td style="text-align: left;">mixsqp</td>
<td style="text-align: left;">0.3-54</td>
<td style="text-align: left;">Sequential Quadratic Programming for Fast Maximum-Likelihood Estimation of Mixture Proportions</td>
</tr>
<tr>
<td style="text-align: left;">mixtools</td>
<td style="text-align: left;">2.0.0.1</td>
<td style="text-align: left;">Tools for Analyzing Finite Mixture Models</td>
</tr>
<tr>
<td style="text-align: left;">mlr</td>
<td style="text-align: left;">2.19.2</td>
<td style="text-align: left;">Machine Learning in R</td>
</tr>
<tr>
<td style="text-align: left;">mlrMBO</td>
<td style="text-align: left;">1.1.5.1</td>
<td style="text-align: left;">Bayesian Optimization and Model-Based Optimization of Expensive Black-Box Functions</td>
</tr>
<tr>
<td style="text-align: left;">mnormt</td>
<td style="text-align: left;">2.1.1</td>
<td style="text-align: left;">The Multivariate Normal and t Distributions, and Their Truncated Versions</td>
</tr>
<tr>
<td style="text-align: left;">modeldata</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Data Sets Useful for Modeling Examples</td>
</tr>
<tr>
<td style="text-align: left;">modelenv</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">Provide Tools to Register Models for Use in 'tidymodels'</td>
</tr>
<tr>
<td style="text-align: left;">ModelMetrics</td>
<td style="text-align: left;">1.2.2.2</td>
<td style="text-align: left;">Rapid Calculation of Model Metrics</td>
</tr>
<tr>
<td style="text-align: left;">modelr</td>
<td style="text-align: left;">0.1.11</td>
<td style="text-align: left;">Modelling Functions that Work with the Pipe</td>
</tr>
<tr>
<td style="text-align: left;">modeltools</td>
<td style="text-align: left;">0.2-24</td>
<td style="text-align: left;">Tools and Classes for Statistical Models</td>
</tr>
<tr>
<td style="text-align: left;">monocle</td>
<td style="text-align: left;">2.36.0</td>
<td style="text-align: left;">Clustering, differential expression, and trajectory analysis for single- cell RNA-Seq</td>
</tr>
<tr>
<td style="text-align: left;">monocle3</td>
<td style="text-align: left;">1.3.7</td>
<td style="text-align: left;">Clustering, Differential Expression, and Trajectory Analysis for Single-Cell RNA-Seq</td>
</tr>
<tr>
<td style="text-align: left;">mosaic</td>
<td style="text-align: left;">1.9.1</td>
<td style="text-align: left;">Project MOSAIC Statistics and Mathematics Teaching Utilities</td>
</tr>
<tr>
<td style="text-align: left;">mosaicCore</td>
<td style="text-align: left;">0.9.4.0</td>
<td style="text-align: left;">Common Utilities for Other MOSAIC-Family Packages</td>
</tr>
<tr>
<td style="text-align: left;">mosaicData</td>
<td style="text-align: left;">0.20.4</td>
<td style="text-align: left;">Project MOSAIC Data Sets</td>
</tr>
<tr>
<td style="text-align: left;">motifmatchr</td>
<td style="text-align: left;">1.30.0</td>
<td style="text-align: left;">Fast Motif Matching in R</td>
</tr>
<tr>
<td style="text-align: left;">MsCoreUtils</td>
<td style="text-align: left;">1.20.0</td>
<td style="text-align: left;">Core Utils for Mass Spectrometry Data</td>
</tr>
<tr>
<td style="text-align: left;">msigdbr</td>
<td style="text-align: left;">10.0.2</td>
<td style="text-align: left;">MSigDB Gene Sets for Multiple Organisms in a Tidy Data Format</td>
</tr>
<tr>
<td style="text-align: left;">MSnbase</td>
<td style="text-align: left;">2.34.1</td>
<td style="text-align: left;">Base Functions and Classes for Mass Spectrometry and Proteomics</td>
</tr>
<tr>
<td style="text-align: left;">multcomp</td>
<td style="text-align: left;">1.4-28</td>
<td style="text-align: left;">Simultaneous Inference in General Parametric Models</td>
</tr>
<tr>
<td style="text-align: left;">multcompView</td>
<td style="text-align: left;">0.1-10</td>
<td style="text-align: left;">Visualizations of Paired Comparisons</td>
</tr>
<tr>
<td style="text-align: left;">MultiAssayExperiment</td>
<td style="text-align: left;">1.34.0</td>
<td style="text-align: left;">Software for the integration of multi-omics experiments in Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">multicool</td>
<td style="text-align: left;">1.0.1</td>
<td style="text-align: left;">Permutations of Multisets in Cool-Lex Order</td>
</tr>
<tr>
<td style="text-align: left;">multicross</td>
<td style="text-align: left;">2.1.0</td>
<td style="text-align: left;">A Graph-Based Test for Comparing Multivariate Distributions in the Multi Sample Framework</td>
</tr>
<tr>
<td style="text-align: left;">multtest</td>
<td style="text-align: left;">2.64.0</td>
<td style="text-align: left;">Resampling-based multiple hypothesis testing</td>
</tr>
<tr>
<td style="text-align: left;">munsell</td>
<td style="text-align: left;">0.5.1</td>
<td style="text-align: left;">Utilities for Using Munsell Colours</td>
</tr>
<tr>
<td style="text-align: left;">muscat</td>
<td style="text-align: left;">1.22.0</td>
<td style="text-align: left;">Multi-sample multi-group scRNA-seq data analysis tools</td>
</tr>
<tr>
<td style="text-align: left;">mutoss</td>
<td style="text-align: left;">0.1-13</td>
<td style="text-align: left;">Unified Multiple Testing Procedures</td>
</tr>
<tr>
<td style="text-align: left;">mvtnorm</td>
<td style="text-align: left;">1.3-3</td>
<td style="text-align: left;">Multivariate Normal and t Distributions</td>
</tr>
<tr>
<td style="text-align: left;">mzID</td>
<td style="text-align: left;">1.46.0</td>
<td style="text-align: left;">An mzIdentML parser for R</td>
</tr>
<tr>
<td style="text-align: left;">mzR</td>
<td style="text-align: left;">2.42.0</td>
<td style="text-align: left;">parser for netCDF, mzXML and mzML and mzIdentML files (mass spectrometry data)</td>
</tr>
<tr>
<td style="text-align: left;">nabor</td>
<td style="text-align: left;">0.5.0</td>
<td style="text-align: left;">Wraps 'libnabo', a Fast K Nearest Neighbour Library for Low Dimensions</td>
</tr>
<tr>
<td style="text-align: left;">nanoarrow</td>
<td style="text-align: left;">0.6.0-1</td>
<td style="text-align: left;">Interface to the 'nanoarrow' 'C' Library</td>
</tr>
<tr>
<td style="text-align: left;">NanoMethViz</td>
<td style="text-align: left;">3.4.0</td>
<td style="text-align: left;">Visualise methylation data from Oxford Nanopore sequencing</td>
</tr>
<tr>
<td style="text-align: left;">nbpMatching</td>
<td style="text-align: left;">1.5.6</td>
<td style="text-align: left;">Functions for Optimal Non-Bipartite Matching</td>
</tr>
<tr>
<td style="text-align: left;">ncdf4</td>
<td style="text-align: left;">1.24</td>
<td style="text-align: left;">Interface to Unidata netCDF (Version 4 or Earlier) Format Data Files</td>
</tr>
<tr>
<td style="text-align: left;">network</td>
<td style="text-align: left;">1.19.0</td>
<td style="text-align: left;">Classes for Relational Data</td>
</tr>
<tr>
<td style="text-align: left;">NetworkComparisonTest</td>
<td style="text-align: left;">2.2.2</td>
<td style="text-align: left;">Statistical Comparison of Two Networks Based on Several Invariance Measures</td>
</tr>
<tr>
<td style="text-align: left;">networkD3</td>
<td style="text-align: left;">0.4.1</td>
<td style="text-align: left;">D3 JavaScript Network Graphs from R</td>
</tr>
<tr>
<td style="text-align: left;">NetworkToolbox</td>
<td style="text-align: left;">1.4.2</td>
<td style="text-align: left;">Methods and Measures for Brain, Cognitive, and Psychometric Network Analysis</td>
</tr>
<tr>
<td style="text-align: left;">networktools</td>
<td style="text-align: left;">1.6.0</td>
<td style="text-align: left;">Tools for Identifying Important Nodes in Networks</td>
</tr>
<tr>
<td style="text-align: left;">neurobase</td>
<td style="text-align: left;">1.32.4</td>
<td style="text-align: left;">'Neuroconductor' Base Package with Helper Functions for 'nifti' Objects</td>
</tr>
<tr>
<td style="text-align: left;">nleqslv</td>
<td style="text-align: left;">3.3.5</td>
<td style="text-align: left;">Solve Systems of Nonlinear Equations</td>
</tr>
<tr>
<td style="text-align: left;">nlme</td>
<td style="text-align: left;">3.1-168</td>
<td style="text-align: left;">Linear and Nonlinear Mixed Effects Models</td>
</tr>
<tr>
<td style="text-align: left;">nloptr</td>
<td style="text-align: left;">2.2.1</td>
<td style="text-align: left;">R Interface to NLopt</td>
</tr>
<tr>
<td style="text-align: left;">NMF</td>
<td style="text-align: left;">0.28</td>
<td style="text-align: left;">Algorithms and Framework for Nonnegative Matrix Factorization (NMF)</td>
</tr>
<tr>
<td style="text-align: left;">nnet</td>
<td style="text-align: left;">7.3-20</td>
<td style="text-align: left;">Feed-Forward Neural Networks and Multinomial Log-Linear Models</td>
</tr>
<tr>
<td style="text-align: left;">nnls</td>
<td style="text-align: left;">1.6</td>
<td style="text-align: left;">The Lawson-Hanson Algorithm for Non-Negative Least Squares (NNLS)</td>
</tr>
<tr>
<td style="text-align: left;">nortest</td>
<td style="text-align: left;">1.0-4</td>
<td style="text-align: left;">Tests for Normality</td>
</tr>
<tr>
<td style="text-align: left;">numDeriv</td>
<td style="text-align: left;">2016.8-1.1</td>
<td style="text-align: left;">Accurate Numerical Derivatives</td>
</tr>
<tr>
<td style="text-align: left;">NxtIRFdata</td>
<td style="text-align: left;">1.14.0</td>
<td style="text-align: left;">Data for NxtIRF</td>
</tr>
<tr>
<td style="text-align: left;">odbc</td>
<td style="text-align: left;">1.6.1</td>
<td style="text-align: left;">Connect to ODBC Compatible Databases (using the DBI Interface)</td>
</tr>
<tr>
<td style="text-align: left;">officer</td>
<td style="text-align: left;">0.6.8</td>
<td style="text-align: left;">Manipulation of Microsoft Word and PowerPoint Documents</td>
</tr>
<tr>
<td style="text-align: left;">oligo</td>
<td style="text-align: left;">1.72.0</td>
<td style="text-align: left;">Preprocessing tools for oligonucleotide arrays</td>
</tr>
<tr>
<td style="text-align: left;">oligoClasses</td>
<td style="text-align: left;">1.70.0</td>
<td style="text-align: left;">Classes for high-throughput arrays supported by oligo and crlmm</td>
</tr>
<tr>
<td style="text-align: left;">ompBAM</td>
<td style="text-align: left;">1.12.0</td>
<td style="text-align: left;">C++ Library for OpenMP-based multi-threaded sequential profiling of Binary Alignment Map (BAM) files</td>
</tr>
<tr>
<td style="text-align: left;">OpenMx</td>
<td style="text-align: left;">2.21.13</td>
<td style="text-align: left;">Extended Structural Equation Modelling</td>
</tr>
<tr>
<td style="text-align: left;">openssl</td>
<td style="text-align: left;">2.3.2</td>
<td style="text-align: left;">Toolkit for Encryption, Signatures and Certificates Based on OpenSSL</td>
</tr>
<tr>
<td style="text-align: left;">openxlsx</td>
<td style="text-align: left;">4.2.8</td>
<td style="text-align: left;">Read, Write and Edit xlsx Files</td>
</tr>
<tr>
<td style="text-align: left;">operator.tools</td>
<td style="text-align: left;">1.6.3</td>
<td style="text-align: left;">Utilities for Working with R's Operators</td>
</tr>
<tr>
<td style="text-align: left;">optimx</td>
<td style="text-align: left;">2025-4.9</td>
<td style="text-align: left;">Expanded Replacement and Extension of the 'optim' Function</td>
</tr>
<tr>
<td style="text-align: left;">optparse</td>
<td style="text-align: left;">1.7.5</td>
<td style="text-align: left;">Command Line Option Parser</td>
</tr>
<tr>
<td style="text-align: left;">orca</td>
<td style="text-align: left;">1.1-3</td>
<td style="text-align: left;">Computation of Graphlet Orbit Counts in Sparse Graphs</td>
</tr>
<tr>
<td style="text-align: left;">ordinal</td>
<td style="text-align: left;">2023.12-4.1</td>
<td style="text-align: left;">Regression Models for Ordinal Data</td>
</tr>
<tr>
<td style="text-align: left;">org.Hs.eg.db</td>
<td style="text-align: left;">3.21.0</td>
<td style="text-align: left;">Genome wide annotation for Human</td>
</tr>
<tr>
<td style="text-align: left;">org.Mm.eg.db</td>
<td style="text-align: left;">3.21.0</td>
<td style="text-align: left;">Genome wide annotation for Mouse</td>
</tr>
<tr>
<td style="text-align: left;">OrganismDbi</td>
<td style="text-align: left;">1.50.0</td>
<td style="text-align: left;">Software to enable the smooth interfacing of different database packages</td>
</tr>
<tr>
<td style="text-align: left;">oro.nifti</td>
<td style="text-align: left;">0.11.4</td>
<td style="text-align: left;">Rigorous - 'NIfTI' + 'ANALYZE' + 'AFNI' : Input / Output</td>
</tr>
<tr>
<td style="text-align: left;">orthopolynom</td>
<td style="text-align: left;">1.0-6.1</td>
<td style="text-align: left;">Collection of Functions for Orthogonal and Orthonormal Polynomials</td>
</tr>
<tr>
<td style="text-align: left;">otel</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">OpenTelemetry R API</td>
</tr>
<tr>
<td style="text-align: left;">outliers</td>
<td style="text-align: left;">0.15</td>
<td style="text-align: left;">Tests for Outliers</td>
</tr>
<tr>
<td style="text-align: left;">packrat</td>
<td style="text-align: left;">0.9.2</td>
<td style="text-align: left;">A Dependency Management System for Projects and their R Package Dependencies</td>
</tr>
<tr>
<td style="text-align: left;">pak</td>
<td style="text-align: left;">0.8.0.2</td>
<td style="text-align: left;">Another Approach to Package Installation</td>
</tr>
<tr>
<td style="text-align: left;">pals</td>
<td style="text-align: left;">1.10</td>
<td style="text-align: left;">Color Palettes, Colormaps, and Tools to Evaluate Them</td>
</tr>
<tr>
<td style="text-align: left;">pammtools</td>
<td style="text-align: left;">0.7.3</td>
<td style="text-align: left;">Piece-Wise Exponential Additive Mixed Modeling Tools for Survival Analysis</td>
</tr>
<tr>
<td style="text-align: left;">pan</td>
<td style="text-align: left;">1.9</td>
<td style="text-align: left;">Multiple Imputation for Multivariate Panel or Clustered Data</td>
</tr>
<tr>
<td style="text-align: left;">parallel</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">Support for Parallel Computation in R</td>
</tr>
<tr>
<td style="text-align: left;">parallelDist</td>
<td style="text-align: left;">0.2.6</td>
<td style="text-align: left;">Parallel Distance Matrix Computation using Multiple Threads</td>
</tr>
<tr>
<td style="text-align: left;">parallelly</td>
<td style="text-align: left;">1.43.0</td>
<td style="text-align: left;">Enhancing the 'parallel' Package</td>
</tr>
<tr>
<td style="text-align: left;">parallelMap</td>
<td style="text-align: left;">1.5.1</td>
<td style="text-align: left;">Unified Interface to Parallelization Back-Ends</td>
</tr>
<tr>
<td style="text-align: left;">ParamHelpers</td>
<td style="text-align: left;">1.14.2</td>
<td style="text-align: left;">Helpers for Parameters in Black-Box Optimization, Tuning and Machine Learning</td>
</tr>
<tr>
<td style="text-align: left;">paran</td>
<td style="text-align: left;">1.5.4</td>
<td style="text-align: left;">Horn's Test of Principal Components/Factors</td>
</tr>
<tr>
<td style="text-align: left;">parsnip</td>
<td style="text-align: left;">1.3.1</td>
<td style="text-align: left;">A Common API to Modeling and Analysis Functions</td>
</tr>
<tr>
<td style="text-align: left;">patchwork</td>
<td style="text-align: left;">1.3.0</td>
<td style="text-align: left;">The Composer of Plots</td>
</tr>
<tr>
<td style="text-align: left;">pathview</td>
<td style="text-align: left;">1.48.0</td>
<td style="text-align: left;">a tool set for pathway based data integration and visualization</td>
</tr>
<tr>
<td style="text-align: left;">pbapply</td>
<td style="text-align: left;">1.7-2</td>
<td style="text-align: left;">Adding Progress Bar to '*apply' Functions</td>
</tr>
<tr>
<td style="text-align: left;">pbdZMQ</td>
<td style="text-align: left;">0.3-14</td>
<td style="text-align: left;">Programming with Big Data -- Interface to 'ZeroMQ'</td>
</tr>
<tr>
<td style="text-align: left;">pbivnorm</td>
<td style="text-align: left;">0.6.0</td>
<td style="text-align: left;">Vectorized Bivariate Normal CDF</td>
</tr>
<tr>
<td style="text-align: left;">pbkrtest</td>
<td style="text-align: left;">0.5.4</td>
<td style="text-align: left;">Parametric Bootstrap, Kenward-Roger and Satterthwaite Based Methods for Test in Mixed Models</td>
</tr>
<tr>
<td style="text-align: left;">pbmcapply</td>
<td style="text-align: left;">1.5.1</td>
<td style="text-align: left;">Tracking the Progress of Mc*pply with Progress Bar</td>
</tr>
<tr>
<td style="text-align: left;">pcaMethods</td>
<td style="text-align: left;">2.0.0</td>
<td style="text-align: left;">A collection of PCA methods</td>
</tr>
<tr>
<td style="text-align: left;">pdist</td>
<td style="text-align: left;">1.2.1</td>
<td style="text-align: left;">Partitioned Distance Function</td>
</tr>
<tr>
<td style="text-align: left;">pec</td>
<td style="text-align: left;">2023.04.12</td>
<td style="text-align: left;">Prediction Error Curves for Risk Prediction Models in Survival Analysis</td>
</tr>
<tr>
<td style="text-align: left;">performance</td>
<td style="text-align: left;">0.13.0</td>
<td style="text-align: left;">Assessment of Regression Models Performance</td>
</tr>
<tr>
<td style="text-align: left;">permute</td>
<td style="text-align: left;">0.9-7</td>
<td style="text-align: left;">Functions for Generating Restricted Permutations of Data</td>
</tr>
<tr>
<td style="text-align: left;">pfamAnalyzeR</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Identification of domain isotypes in pfam data</td>
</tr>
<tr>
<td style="text-align: left;">pheatmap</td>
<td style="text-align: left;">1.0.12</td>
<td style="text-align: left;">Pretty Heatmaps</td>
</tr>
<tr>
<td style="text-align: left;">phia</td>
<td style="text-align: left;">0.3-1</td>
<td style="text-align: left;">Post-Hoc Interaction Analysis</td>
</tr>
<tr>
<td style="text-align: left;">phyclust</td>
<td style="text-align: left;">0.1-34</td>
<td style="text-align: left;">Phylogenetic Clustering (Phyloclustering)</td>
</tr>
<tr>
<td style="text-align: left;">pillar</td>
<td style="text-align: left;">1.10.2</td>
<td style="text-align: left;">Coloured Formatting for Columns</td>
</tr>
<tr>
<td style="text-align: left;">pinfsc50</td>
<td style="text-align: left;">1.3.0</td>
<td style="text-align: left;">Sequence ('FASTA'), Annotation ('GFF') and Variants ('VCF') for 17 Samples of 'P. Infestans" and 1 'P. Mirabilis'</td>
</tr>
<tr>
<td style="text-align: left;">pixmap</td>
<td style="text-align: left;">0.4-13</td>
<td style="text-align: left;">Bitmap Images / Pixel Maps</td>
</tr>
<tr>
<td style="text-align: left;">pkgbuild</td>
<td style="text-align: left;">1.4.8</td>
<td style="text-align: left;">Find Tools Needed to Build R Packages</td>
</tr>
<tr>
<td style="text-align: left;">pkgconfig</td>
<td style="text-align: left;">2.0.3</td>
<td style="text-align: left;">Private Configuration for 'R' Packages</td>
</tr>
<tr>
<td style="text-align: left;">pkgdown</td>
<td style="text-align: left;">2.1.3</td>
<td style="text-align: left;">Make Static HTML Documentation for a Package</td>
</tr>
<tr>
<td style="text-align: left;">pkgfilecache</td>
<td style="text-align: left;">0.1.5</td>
<td style="text-align: left;">Download and Manage Optional Package Data</td>
</tr>
<tr>
<td style="text-align: left;">pkgload</td>
<td style="text-align: left;">1.4.1</td>
<td style="text-align: left;">Simulate Package Installation and Attach</td>
</tr>
<tr>
<td style="text-align: left;">PKI</td>
<td style="text-align: left;">0.1-14</td>
<td style="text-align: left;">Public Key Infrastucture for R Based on the X.509 Standard</td>
</tr>
<tr>
<td style="text-align: left;">plink</td>
<td style="text-align: left;">1.5-1</td>
<td style="text-align: left;">IRT Separate Calibration Linking Methods</td>
</tr>
<tr>
<td style="text-align: left;">plogr</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">The 'plog' C++ Logging Library</td>
</tr>
<tr>
<td style="text-align: left;">plot3D</td>
<td style="text-align: left;">1.4.1</td>
<td style="text-align: left;">Plotting Multi-Dimensional Data</td>
</tr>
<tr>
<td style="text-align: left;">plotly</td>
<td style="text-align: left;">4.10.4</td>
<td style="text-align: left;">Create Interactive Web Graphics via 'plotly.js'</td>
</tr>
<tr>
<td style="text-align: left;">plotrix</td>
<td style="text-align: left;">3.8-4</td>
<td style="text-align: left;">Various Plotting Functions</td>
</tr>
<tr>
<td style="text-align: left;">pls</td>
<td style="text-align: left;">2.8-5</td>
<td style="text-align: left;">Partial Least Squares and Principal Component Regression</td>
</tr>
<tr>
<td style="text-align: left;">plyr</td>
<td style="text-align: left;">1.8.9</td>
<td style="text-align: left;">Tools for Splitting, Applying and Combining Data</td>
</tr>
<tr>
<td style="text-align: left;">png</td>
<td style="text-align: left;">0.1-8</td>
<td style="text-align: left;">Read and write PNG images</td>
</tr>
<tr>
<td style="text-align: left;">polspline</td>
<td style="text-align: left;">1.1.25</td>
<td style="text-align: left;">Polynomial Spline Routines</td>
</tr>
<tr>
<td style="text-align: left;">polyclip</td>
<td style="text-align: left;">1.10-7</td>
<td style="text-align: left;">Polygon Clipping</td>
</tr>
<tr>
<td style="text-align: left;">polylabelr</td>
<td style="text-align: left;">0.3.0</td>
<td style="text-align: left;">Find the Pole of Inaccessibility (Visual Center) of a Polygon</td>
</tr>
<tr>
<td style="text-align: left;">polynom</td>
<td style="text-align: left;">1.4-1</td>
<td style="text-align: left;">A Collection of Functions to Implement a Class for Univariate Polynomial Manipulations</td>
</tr>
<tr>
<td style="text-align: left;">pool</td>
<td style="text-align: left;">1.0.4</td>
<td style="text-align: left;">Object Pooling</td>
</tr>
<tr>
<td style="text-align: left;">posterior</td>
<td style="text-align: left;">1.6.1</td>
<td style="text-align: left;">Tools for Working with Posterior Distributions</td>
</tr>
<tr>
<td style="text-align: left;">poweRlaw</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">Analysis of Heavy Tailed Distributions</td>
</tr>
<tr>
<td style="text-align: left;">ppcor</td>
<td style="text-align: left;">1.1</td>
<td style="text-align: left;">Partial and Semi-Partial (Part) Correlation</td>
</tr>
<tr>
<td style="text-align: left;">pracma</td>
<td style="text-align: left;">2.4.4</td>
<td style="text-align: left;">Practical Numerical Math Functions</td>
</tr>
<tr>
<td style="text-align: left;">praise</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">Praise Users</td>
</tr>
<tr>
<td style="text-align: left;">preprocessCore</td>
<td style="text-align: left;">1.70.0</td>
<td style="text-align: left;">A collection of pre-processing functions</td>
</tr>
<tr>
<td style="text-align: left;">presto</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">Fast Functions for Differential Expression using Wilcox and AUC</td>
</tr>
<tr>
<td style="text-align: left;">prettyunits</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">Pretty, Human Readable Formatting of Quantities</td>
</tr>
<tr>
<td style="text-align: left;">pROC</td>
<td style="text-align: left;">1.18.5</td>
<td style="text-align: left;">Display and Analyze ROC Curves</td>
</tr>
<tr>
<td style="text-align: left;">processx</td>
<td style="text-align: left;">3.8.6</td>
<td style="text-align: left;">Execute and Control System Processes</td>
</tr>
<tr>
<td style="text-align: left;">prodlim</td>
<td style="text-align: left;">2025.04.28</td>
<td style="text-align: left;">Product-Limit Estimation for Censored Event History Analysis</td>
</tr>
<tr>
<td style="text-align: left;">profileplyr</td>
<td style="text-align: left;">1.24.0</td>
<td style="text-align: left;">Visualization and annotation of read signal over genomic ranges with profileplyr</td>
</tr>
<tr>
<td style="text-align: left;">profvis</td>
<td style="text-align: left;">0.4.0</td>
<td style="text-align: left;">Interactive Visualizations for Profiling R Code</td>
</tr>
<tr>
<td style="text-align: left;">progress</td>
<td style="text-align: left;">1.2.3</td>
<td style="text-align: left;">Terminal Progress Bars</td>
</tr>
<tr>
<td style="text-align: left;">progressr</td>
<td style="text-align: left;">0.15.1</td>
<td style="text-align: left;">An Inclusive, Unifying API for Progress Updates</td>
</tr>
<tr>
<td style="text-align: left;">promises</td>
<td style="text-align: left;">1.3.2</td>
<td style="text-align: left;">Abstractions for Promise-Based Asynchronous Programming</td>
</tr>
<tr>
<td style="text-align: left;">proteoDA</td>
<td style="text-align: left;">1.0.1</td>
<td style="text-align: left;">Streamlined Differential Abundance Analysis of Proteomic Data</td>
</tr>
<tr>
<td style="text-align: left;">ProtGenerics</td>
<td style="text-align: left;">1.40.0</td>
<td style="text-align: left;">Generic infrastructure for Bioconductor mass spectrometry packages</td>
</tr>
<tr>
<td style="text-align: left;">proto</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">Prototype Object-Based Programming</td>
</tr>
<tr>
<td style="text-align: left;">proxy</td>
<td style="text-align: left;">0.4-27</td>
<td style="text-align: left;">Distance and Similarity Measures</td>
</tr>
<tr>
<td style="text-align: left;">pryr</td>
<td style="text-align: left;">0.1.6</td>
<td style="text-align: left;">Tools for Computing on the Language</td>
</tr>
<tr>
<td style="text-align: left;">ps</td>
<td style="text-align: left;">1.9.1</td>
<td style="text-align: left;">List, Query, Manipulate System Processes</td>
</tr>
<tr>
<td style="text-align: left;">pscl</td>
<td style="text-align: left;">1.5.9</td>
<td style="text-align: left;">Political Science Computational Laboratory</td>
</tr>
<tr>
<td style="text-align: left;">PSMatch</td>
<td style="text-align: left;">1.12.0</td>
<td style="text-align: left;">Handling and Managing Peptide Spectrum Matches</td>
</tr>
<tr>
<td style="text-align: left;">psych</td>
<td style="text-align: left;">2.5.3</td>
<td style="text-align: left;">Procedures for Psychological, Psychometric, and Personality Research</td>
</tr>
<tr>
<td style="text-align: left;">Publish</td>
<td style="text-align: left;">2023.01.17</td>
<td style="text-align: left;">Format Output of Various Routines in a Suitable Way for Reports and Publication</td>
</tr>
<tr>
<td style="text-align: left;">purrr</td>
<td style="text-align: left;">1.0.4</td>
<td style="text-align: left;">Functional Programming Tools</td>
</tr>
<tr>
<td style="text-align: left;">pwalign</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Perform pairwise sequence alignments</td>
</tr>
<tr>
<td style="text-align: left;">pwr</td>
<td style="text-align: left;">1.3-0</td>
<td style="text-align: left;">Basic Functions for Power Analysis</td>
</tr>
<tr>
<td style="text-align: left;">qap</td>
<td style="text-align: left;">0.1-2</td>
<td style="text-align: left;">Heuristics for the Quadratic Assignment Problem (QAP)</td>
</tr>
<tr>
<td style="text-align: left;">qdapRegex</td>
<td style="text-align: left;">0.7.10</td>
<td style="text-align: left;">Regular Expression Removal, Extraction, and Replacement Tools</td>
</tr>
<tr>
<td style="text-align: left;">QFeatures</td>
<td style="text-align: left;">1.18.0</td>
<td style="text-align: left;">Quantitative features for mass spectrometry data</td>
</tr>
<tr>
<td style="text-align: left;">qgraph</td>
<td style="text-align: left;">1.9.8</td>
<td style="text-align: left;">Graph Plotting Methods, Psychometric Data Visualization and Graphical Model Estimation</td>
</tr>
<tr>
<td style="text-align: left;">qqconf</td>
<td style="text-align: left;">1.3.2</td>
<td style="text-align: left;">Creates Simultaneous Testing Bands for QQ-Plots</td>
</tr>
<tr>
<td style="text-align: left;">qqman</td>
<td style="text-align: left;">0.1.9</td>
<td style="text-align: left;">Q-Q and Manhattan Plots for GWAS Data</td>
</tr>
<tr>
<td style="text-align: left;">qs</td>
<td style="text-align: left;">0.27.3</td>
<td style="text-align: left;">Quick Serialization of R Objects</td>
</tr>
<tr>
<td style="text-align: left;">quadprog</td>
<td style="text-align: left;">1.5-8</td>
<td style="text-align: left;">Functions to Solve Quadratic Programming Problems</td>
</tr>
<tr>
<td style="text-align: left;">quantreg</td>
<td style="text-align: left;">6.1</td>
<td style="text-align: left;">Quantile Regression</td>
</tr>
<tr>
<td style="text-align: left;">quantsmooth</td>
<td style="text-align: left;">1.74.0</td>
<td style="text-align: left;">Quantile smoothing and genomic visualization of array data</td>
</tr>
<tr>
<td style="text-align: left;">questionr</td>
<td style="text-align: left;">0.8.0</td>
<td style="text-align: left;">Functions to Make Surveys Processing Easier</td>
</tr>
<tr>
<td style="text-align: left;">QuickJSR</td>
<td style="text-align: left;">1.7.0</td>
<td style="text-align: left;">Interface for the 'QuickJS' Lightweight 'JavaScript' Engine</td>
</tr>
<tr>
<td style="text-align: left;">qvalue</td>
<td style="text-align: left;">2.40.0</td>
<td style="text-align: left;">Q-value estimation for false discovery rate control</td>
</tr>
<tr>
<td style="text-align: left;">R.cache</td>
<td style="text-align: left;">0.16.0</td>
<td style="text-align: left;">Fast and Light-Weight Caching (Memoization) of Objects and Results to Speed Up Computations</td>
</tr>
<tr>
<td style="text-align: left;">R.matlab</td>
<td style="text-align: left;">3.7.0</td>
<td style="text-align: left;">Read and Write MAT Files and Call MATLAB from Within R</td>
</tr>
<tr>
<td style="text-align: left;">R.methodsS3</td>
<td style="text-align: left;">1.8.2</td>
<td style="text-align: left;">S3 Methods Simplified</td>
</tr>
<tr>
<td style="text-align: left;">R.oo</td>
<td style="text-align: left;">1.27.0</td>
<td style="text-align: left;">R Object-Oriented Programming with or without References</td>
</tr>
<tr>
<td style="text-align: left;">R.utils</td>
<td style="text-align: left;">2.13.0</td>
<td style="text-align: left;">Various Programming Utilities</td>
</tr>
<tr>
<td style="text-align: left;">R2HTML</td>
<td style="text-align: left;">2.3.4</td>
<td style="text-align: left;">HTML Exportation for R Objects</td>
</tr>
<tr>
<td style="text-align: left;">R2jags</td>
<td style="text-align: left;">0.8-9</td>
<td style="text-align: left;">Using R to Run 'JAGS'</td>
</tr>
<tr>
<td style="text-align: left;">R2WinBUGS</td>
<td style="text-align: left;">2.1-22.1</td>
<td style="text-align: left;">Running 'WinBUGS' and 'OpenBUGS' from 'R' / 'S-PLUS'</td>
</tr>
<tr>
<td style="text-align: left;">R6</td>
<td style="text-align: left;">2.6.1</td>
<td style="text-align: left;">Encapsulated Classes with Reference Semantics</td>
</tr>
<tr>
<td style="text-align: left;">ragg</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Graphic Devices Based on AGG</td>
</tr>
<tr>
<td style="text-align: left;">randomForest</td>
<td style="text-align: left;">4.7-1.2</td>
<td style="text-align: left;">Breiman and Cutlers Random Forests for Classification and Regression</td>
</tr>
<tr>
<td style="text-align: left;">randomForestSRC</td>
<td style="text-align: left;">3.3.3</td>
<td style="text-align: left;">Fast Unified Random Forests for Survival, Regression, and Classification (RF-SRC)</td>
</tr>
<tr>
<td style="text-align: left;">ranger</td>
<td style="text-align: left;">0.17.0</td>
<td style="text-align: left;">A Fast Implementation of Random Forests</td>
</tr>
<tr>
<td style="text-align: left;">RANN</td>
<td style="text-align: left;">2.6.2</td>
<td style="text-align: left;">Fast Nearest Neighbour Search (Wraps ANN Library) Using L2 Metric</td>
</tr>
<tr>
<td style="text-align: left;">rapidjsonr</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">'Rapidjson' C++ Header Files</td>
</tr>
<tr>
<td style="text-align: left;">RApiSerialize</td>
<td style="text-align: left;">0.1.4</td>
<td style="text-align: left;">R API Serialization</td>
</tr>
<tr>
<td style="text-align: left;">rappdirs</td>
<td style="text-align: left;">0.3.3</td>
<td style="text-align: left;">Application Directories: Determine Where to Save Data, Caches, and Logs</td>
</tr>
<tr>
<td style="text-align: left;">raster</td>
<td style="text-align: left;">3.6-32</td>
<td style="text-align: left;">Geographic Data Analysis and Modeling</td>
</tr>
<tr>
<td style="text-align: left;">RBGL</td>
<td style="text-align: left;">1.84.0</td>
<td style="text-align: left;">An interface to the BOOST graph library</td>
</tr>
<tr>
<td style="text-align: left;">rbibutils</td>
<td style="text-align: left;">2.3</td>
<td style="text-align: left;">Read 'Bibtex' Files and Convert Between Bibliography Formats</td>
</tr>
<tr>
<td style="text-align: left;">rbokeh</td>
<td style="text-align: left;">0.5.2</td>
<td style="text-align: left;">R Interface for Bokeh</td>
</tr>
<tr>
<td style="text-align: left;">RcisTarget</td>
<td style="text-align: left;">1.28.0</td>
<td style="text-align: left;">RcisTarget Identify transcription factor binding motifs enriched on a list of genes or genomic regions</td>
</tr>
<tr>
<td style="text-align: left;">rcmdcheck</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Run 'R CMD check' from 'R' and Capture Results</td>
</tr>
<tr>
<td style="text-align: left;">RColorBrewer</td>
<td style="text-align: left;">1.1-3</td>
<td style="text-align: left;">ColorBrewer Palettes</td>
</tr>
<tr>
<td style="text-align: left;">Rcpp</td>
<td style="text-align: left;">1.0.14</td>
<td style="text-align: left;">Seamless R and C++ Integration</td>
</tr>
<tr>
<td style="text-align: left;">RcppAnnoy</td>
<td style="text-align: left;">0.0.22</td>
<td style="text-align: left;">'Rcpp' Bindings for 'Annoy', a Library for Approximate Nearest Neighbors</td>
</tr>
<tr>
<td style="text-align: left;">RcppArmadillo</td>
<td style="text-align: left;">14.4.2-1</td>
<td style="text-align: left;">'Rcpp' Integration for the 'Armadillo' Templated Linear Algebra Library</td>
</tr>
<tr>
<td style="text-align: left;">RcppEigen</td>
<td style="text-align: left;">0.3.4.0.2</td>
<td style="text-align: left;">'Rcpp' Integration for the 'Eigen' Templated Linear Algebra Library</td>
</tr>
<tr>
<td style="text-align: left;">RcppGSL</td>
<td style="text-align: left;">0.3.13</td>
<td style="text-align: left;">'Rcpp' Integration for 'GNU GSL' Vectors and Matrices</td>
</tr>
<tr>
<td style="text-align: left;">RcppHNSW</td>
<td style="text-align: left;">0.6.0</td>
<td style="text-align: left;">'Rcpp' Bindings for 'hnswlib', a Library for Approximate Nearest Neighbors</td>
</tr>
<tr>
<td style="text-align: left;">RcppML</td>
<td style="text-align: left;">0.3.7</td>
<td style="text-align: left;">Rcpp Machine Learning Library</td>
</tr>
<tr>
<td style="text-align: left;">RcppNumerical</td>
<td style="text-align: left;">0.6-0</td>
<td style="text-align: left;">'Rcpp' Integration for Numerical Computing Libraries</td>
</tr>
<tr>
<td style="text-align: left;">RcppParallel</td>
<td style="text-align: left;">5.1.10</td>
<td style="text-align: left;">Parallel Programming Tools for 'Rcpp'</td>
</tr>
<tr>
<td style="text-align: left;">RcppProgress</td>
<td style="text-align: left;">0.4.2</td>
<td style="text-align: left;">An Interruptible Progress Bar with OpenMP Support for C++ in R Packages</td>
</tr>
<tr>
<td style="text-align: left;">RcppRoll</td>
<td style="text-align: left;">0.3.1</td>
<td style="text-align: left;">Efficient Rolling / Windowed Operations</td>
</tr>
<tr>
<td style="text-align: left;">RcppTOML</td>
<td style="text-align: left;">0.2.3</td>
<td style="text-align: left;">'Rcpp' Bindings to Parser for "Tom's Obvious Markup Language"</td>
</tr>
<tr>
<td style="text-align: left;">RcppZiggurat</td>
<td style="text-align: left;">0.1.8</td>
<td style="text-align: left;">'Rcpp' Integration of Different "Ziggurat" Normal RNG Implementations</td>
</tr>
<tr>
<td style="text-align: left;">RCurl</td>
<td style="text-align: left;">1.98-1.17</td>
<td style="text-align: left;">General Network (HTTP/FTP/...) Client Interface for R</td>
</tr>
<tr>
<td style="text-align: left;">RCy3</td>
<td style="text-align: left;">2.28.0</td>
<td style="text-align: left;">Functions to Access and Control Cytoscape</td>
</tr>
<tr>
<td style="text-align: left;">Rdpack</td>
<td style="text-align: left;">2.6.4</td>
<td style="text-align: left;">Update and Manipulate Rd Documentation Objects</td>
</tr>
<tr>
<td style="text-align: left;">reactable</td>
<td style="text-align: left;">0.4.4</td>
<td style="text-align: left;">Interactive Data Tables for R</td>
</tr>
<tr>
<td style="text-align: left;">reactome.db</td>
<td style="text-align: left;">1.92.0</td>
<td style="text-align: left;">A set of annotation maps for reactome</td>
</tr>
<tr>
<td style="text-align: left;">ReactomePA</td>
<td style="text-align: left;">1.52.0</td>
<td style="text-align: left;">Reactome Pathway Analysis</td>
</tr>
<tr>
<td style="text-align: left;">reactR</td>
<td style="text-align: left;">0.6.1</td>
<td style="text-align: left;">React Helpers</td>
</tr>
<tr>
<td style="text-align: left;">readbitmap</td>
<td style="text-align: left;">0.1.5</td>
<td style="text-align: left;">Simple Unified Interface to Read Bitmap Images (BMP,JPEG,PNG,TIFF)</td>
</tr>
<tr>
<td style="text-align: left;">readr</td>
<td style="text-align: left;">2.1.5</td>
<td style="text-align: left;">Read Rectangular Text Data</td>
</tr>
<tr>
<td style="text-align: left;">readxl</td>
<td style="text-align: left;">1.4.5</td>
<td style="text-align: left;">Read Excel Files</td>
</tr>
<tr>
<td style="text-align: left;">recipes</td>
<td style="text-align: left;">1.3.0</td>
<td style="text-align: left;">Preprocessing and Feature Engineering Steps for Modeling</td>
</tr>
<tr>
<td style="text-align: left;">reformulas</td>
<td style="text-align: left;">0.4.0</td>
<td style="text-align: left;">Machinery for Processing Random Effect Formulas</td>
</tr>
<tr>
<td style="text-align: left;">registry</td>
<td style="text-align: left;">0.5-1</td>
<td style="text-align: left;">Infrastructure for R Package Registries</td>
</tr>
<tr>
<td style="text-align: left;">relsurv</td>
<td style="text-align: left;">2.3-2</td>
<td style="text-align: left;">Relative Survival</td>
</tr>
<tr>
<td style="text-align: left;">remaCor</td>
<td style="text-align: left;">0.0.18</td>
<td style="text-align: left;">Random Effects Meta-Analysis for Correlated Test Statistics</td>
</tr>
<tr>
<td style="text-align: left;">rematch</td>
<td style="text-align: left;">2.0.0</td>
<td style="text-align: left;">Match Regular Expressions with a Nicer 'API'</td>
</tr>
<tr>
<td style="text-align: left;">rematch2</td>
<td style="text-align: left;">2.1.2</td>
<td style="text-align: left;">Tidy Output from Regular Expression Matching</td>
</tr>
<tr>
<td style="text-align: left;">remotes</td>
<td style="text-align: left;">2.5.0</td>
<td style="text-align: left;">R Package Installation from Remote Repositories, Including 'GitHub'</td>
</tr>
<tr>
<td style="text-align: left;">rentrez</td>
<td style="text-align: left;">1.2.3</td>
<td style="text-align: left;">'Entrez' in R</td>
</tr>
<tr>
<td style="text-align: left;">renv</td>
<td style="text-align: left;">1.1.4</td>
<td style="text-align: left;">Project Environments</td>
</tr>
<tr>
<td style="text-align: left;">repr</td>
<td style="text-align: left;">1.1.7</td>
<td style="text-align: left;">Serializable Representations</td>
</tr>
<tr>
<td style="text-align: left;">reprex</td>
<td style="text-align: left;">2.1.1</td>
<td style="text-align: left;">Prepare Reproducible Example Code via the Clipboard</td>
</tr>
<tr>
<td style="text-align: left;">reshape</td>
<td style="text-align: left;">0.8.9</td>
<td style="text-align: left;">Flexibly Reshape Data</td>
</tr>
<tr>
<td style="text-align: left;">reshape2</td>
<td style="text-align: left;">1.4.4</td>
<td style="text-align: left;">Flexibly Reshape Data: A Reboot of the Reshape Package</td>
</tr>
<tr>
<td style="text-align: left;">ResidualMatrix</td>
<td style="text-align: left;">1.18.0</td>
<td style="text-align: left;">Creating a DelayedMatrix of Regression Residuals</td>
</tr>
<tr>
<td style="text-align: left;">restfulr</td>
<td style="text-align: left;">0.0.15</td>
<td style="text-align: left;">R Interface to RESTful Web Services</td>
</tr>
<tr>
<td style="text-align: left;">reticulate</td>
<td style="text-align: left;">1.43.0</td>
<td style="text-align: left;">Interface to 'Python'</td>
</tr>
<tr>
<td style="text-align: left;">rex</td>
<td style="text-align: left;">1.2.1</td>
<td style="text-align: left;">Friendly Regular Expressions</td>
</tr>
<tr>
<td style="text-align: left;">Rfast</td>
<td style="text-align: left;">2.1.5.1</td>
<td style="text-align: left;">A Collection of Efficient and Extremely Fast R Functions</td>
</tr>
<tr>
<td style="text-align: left;">rgl</td>
<td style="text-align: left;">1.3.18</td>
<td style="text-align: left;">3D Visualization Using OpenGL</td>
</tr>
<tr>
<td style="text-align: left;">Rgraphviz</td>
<td style="text-align: left;">2.52.0</td>
<td style="text-align: left;">Provides plotting capabilities for R graph objects</td>
</tr>
<tr>
<td style="text-align: left;">rGREAT</td>
<td style="text-align: left;">2.10.0</td>
<td style="text-align: left;">GREAT Analysis - Functional Enrichment on Genomic Regions</td>
</tr>
<tr>
<td style="text-align: left;">rhandsontable</td>
<td style="text-align: left;">0.3.8</td>
<td style="text-align: left;">Interface to the 'Handsontable.js' Library</td>
</tr>
<tr>
<td style="text-align: left;">rhdf5</td>
<td style="text-align: left;">2.52.0</td>
<td style="text-align: left;">R Interface to HDF5</td>
</tr>
<tr>
<td style="text-align: left;">rhdf5filters</td>
<td style="text-align: left;">1.20.0</td>
<td style="text-align: left;">HDF5 Compression Filters</td>
</tr>
<tr>
<td style="text-align: left;">Rhdf5lib</td>
<td style="text-align: left;">1.30.0</td>
<td style="text-align: left;">hdf5 library as an R package</td>
</tr>
<tr>
<td style="text-align: left;">RhpcBLASctl</td>
<td style="text-align: left;">0.23-42</td>
<td style="text-align: left;">Control the Number of Threads on 'BLAS'</td>
</tr>
<tr>
<td style="text-align: left;">Rhtslib</td>
<td style="text-align: left;">3.4.0</td>
<td style="text-align: left;">HTSlib high-throughput sequencing library as an R package</td>
</tr>
<tr>
<td style="text-align: left;">RInside</td>
<td style="text-align: left;">0.2.19</td>
<td style="text-align: left;">C++ Classes to Embed R in C++ (and C) Applications</td>
</tr>
<tr>
<td style="text-align: left;">rintrojs</td>
<td style="text-align: left;">0.3.4</td>
<td style="text-align: left;">Wrapper for the 'Intro.js' Library</td>
</tr>
<tr>
<td style="text-align: left;">rio</td>
<td style="text-align: left;">1.2.3</td>
<td style="text-align: left;">A Swiss-Army Knife for Data I/O</td>
</tr>
<tr>
<td style="text-align: left;">RISCA</td>
<td style="text-align: left;">1.0.7</td>
<td style="text-align: left;">Causal Inference and Prediction in Cohort-Based Analyses</td>
</tr>
<tr>
<td style="text-align: left;">riskRegression</td>
<td style="text-align: left;">2023.12.21</td>
<td style="text-align: left;">Risk Regression Models and Prediction Scores for Survival Analysis with Competing Risks</td>
</tr>
<tr>
<td style="text-align: left;">RITAN</td>
<td style="text-align: left;">1.32.0</td>
<td style="text-align: left;">Rapid Integration of Term Annotation and Network resources</td>
</tr>
<tr>
<td style="text-align: left;">RITANdata</td>
<td style="text-align: left;">1.32.0</td>
<td style="text-align: left;">This package contains reference annotation and network data sets</td>
</tr>
<tr>
<td style="text-align: left;">rjags</td>
<td style="text-align: left;">4-17</td>
<td style="text-align: left;">Bayesian Graphical Models using MCMC</td>
</tr>
<tr>
<td style="text-align: left;">rJava</td>
<td style="text-align: left;">1.0-11</td>
<td style="text-align: left;">Low-Level R to Java Interface</td>
</tr>
<tr>
<td style="text-align: left;">rjson</td>
<td style="text-align: left;">0.2.23</td>
<td style="text-align: left;">JSON for R</td>
</tr>
<tr>
<td style="text-align: left;">RJSONIO</td>
<td style="text-align: left;">2.0.0</td>
<td style="text-align: left;">Serialize R Objects to JSON, JavaScript Object Notation</td>
</tr>
<tr>
<td style="text-align: left;">rlang</td>
<td style="text-align: left;">1.1.6</td>
<td style="text-align: left;">Functions for Base Types and Core R and 'Tidyverse' Features</td>
</tr>
<tr>
<td style="text-align: left;">rlist</td>
<td style="text-align: left;">0.4.6.2</td>
<td style="text-align: left;">A Toolbox for Non-Tabular Data Manipulation</td>
</tr>
<tr>
<td style="text-align: left;">RMariaDB</td>
<td style="text-align: left;">1.3.4</td>
<td style="text-align: left;">Database Interface and MariaDB Driver</td>
</tr>
<tr>
<td style="text-align: left;">rmarkdown</td>
<td style="text-align: left;">2.29</td>
<td style="text-align: left;">Dynamic Documents for R</td>
</tr>
<tr>
<td style="text-align: left;">rmeta</td>
<td style="text-align: left;">3.0</td>
<td style="text-align: left;">Meta-Analysis</td>
</tr>
<tr>
<td style="text-align: left;">Rmisc</td>
<td style="text-align: left;">1.5.1</td>
<td style="text-align: left;">Ryan Miscellaneous</td>
</tr>
<tr>
<td style="text-align: left;">Rmpi</td>
<td style="text-align: left;">0.7-3.3</td>
<td style="text-align: left;">Interface (Wrapper) to MPI (Message-Passing Interface)</td>
</tr>
<tr>
<td style="text-align: left;">rms</td>
<td style="text-align: left;">8.0-0</td>
<td style="text-align: left;">Regression Modeling Strategies</td>
</tr>
<tr>
<td style="text-align: left;">RMySQL</td>
<td style="text-align: left;">0.11.1</td>
<td style="text-align: left;">Database Interface and 'MySQL' Driver for R</td>
</tr>
<tr>
<td style="text-align: left;">Rnanoflann</td>
<td style="text-align: left;">0.0.3</td>
<td style="text-align: left;">Extremely Fast Nearest Neighbor Search</td>
</tr>
<tr>
<td style="text-align: left;">rngtools</td>
<td style="text-align: left;">1.5.2</td>
<td style="text-align: left;">Utility Functions for Working with Random Number Generators</td>
</tr>
<tr>
<td style="text-align: left;">RNifti</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Fast R and C++ Access to NIfTI Images</td>
</tr>
<tr>
<td style="text-align: left;">robustbase</td>
<td style="text-align: left;">0.99-4-1</td>
<td style="text-align: left;">Basic Robust Statistics</td>
</tr>
<tr>
<td style="text-align: left;">rockchalk</td>
<td style="text-align: left;">1.8.157</td>
<td style="text-align: left;">Regression Estimation and Presentation</td>
</tr>
<tr>
<td style="text-align: left;">ROCR</td>
<td style="text-align: left;">1.0-11</td>
<td style="text-align: left;">Visualizing the Performance of Scoring Classifiers</td>
</tr>
<tr>
<td style="text-align: left;">rootSolve</td>
<td style="text-align: left;">1.8.2.4</td>
<td style="text-align: left;">Nonlinear Root Finding, Equilibrium and Steady-State Analysis of Ordinary Differential Equations</td>
</tr>
<tr>
<td style="text-align: left;">ROpenCVLite</td>
<td style="text-align: left;">4.90.2</td>
<td style="text-align: left;">Helper Package for Installing OpenCV with R</td>
</tr>
<tr>
<td style="text-align: left;">roxygen2</td>
<td style="text-align: left;">7.3.3</td>
<td style="text-align: left;">In-Line Documentation for R</td>
</tr>
<tr>
<td style="text-align: left;">rpart</td>
<td style="text-align: left;">4.1.24</td>
<td style="text-align: left;">Recursive Partitioning and Regression Trees</td>
</tr>
<tr>
<td style="text-align: left;">rpf</td>
<td style="text-align: left;">1.0.14</td>
<td style="text-align: left;">Response Probability Functions</td>
</tr>
<tr>
<td style="text-align: left;">RPresto</td>
<td style="text-align: left;">1.4.7</td>
<td style="text-align: left;">DBI Connector to Presto</td>
</tr>
<tr>
<td style="text-align: left;">rprojroot</td>
<td style="text-align: left;">2.1.1</td>
<td style="text-align: left;">Finding Files in Project Subdirectories</td>
</tr>
<tr>
<td style="text-align: left;">rsample</td>
<td style="text-align: left;">1.3.0</td>
<td style="text-align: left;">General Resampling Infrastructure</td>
</tr>
<tr>
<td style="text-align: left;">Rsamtools</td>
<td style="text-align: left;">2.24.0</td>
<td style="text-align: left;">Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import</td>
</tr>
<tr>
<td style="text-align: left;">rsconnect</td>
<td style="text-align: left;">1.3.4</td>
<td style="text-align: left;">Deploy Docs, Apps, and APIs to 'Posit Connect', 'shinyapps.io', and 'RPubs'</td>
</tr>
<tr>
<td style="text-align: left;">Rserve</td>
<td style="text-align: left;">1.8-15</td>
<td style="text-align: left;">Versatile R Server</td>
</tr>
<tr>
<td style="text-align: left;">RSpectra</td>
<td style="text-align: left;">0.16-2</td>
<td style="text-align: left;">Solvers for Large-Scale Eigenvalue and SVD Problems</td>
</tr>
<tr>
<td style="text-align: left;">RSQLite</td>
<td style="text-align: left;">2.3.9</td>
<td style="text-align: left;">SQLite Interface for R</td>
</tr>
<tr>
<td style="text-align: left;">rstan</td>
<td style="text-align: left;">2.32.7</td>
<td style="text-align: left;">R Interface to Stan</td>
</tr>
<tr>
<td style="text-align: left;">rstantools</td>
<td style="text-align: left;">2.4.0</td>
<td style="text-align: left;">Tools for Developing R Packages Interfacing with 'Stan'</td>
</tr>
<tr>
<td style="text-align: left;">rstatix</td>
<td style="text-align: left;">0.7.2</td>
<td style="text-align: left;">Pipe-Friendly Framework for Basic Statistical Tests</td>
</tr>
<tr>
<td style="text-align: left;">rstudioapi</td>
<td style="text-align: left;">0.17.1</td>
<td style="text-align: left;">Safely Access the RStudio API</td>
</tr>
<tr>
<td style="text-align: left;">Rsubread</td>
<td style="text-align: left;">2.22.1</td>
<td style="text-align: left;">Mapping, quantification and variant analysis of sequencing data</td>
</tr>
<tr>
<td style="text-align: left;">rsvd</td>
<td style="text-align: left;">1.0.5</td>
<td style="text-align: left;">Randomized Singular Value Decomposition</td>
</tr>
<tr>
<td style="text-align: left;">rtracklayer</td>
<td style="text-align: left;">1.68.0</td>
<td style="text-align: left;">R interface to genome annotation files and the UCSC genome browser</td>
</tr>
<tr>
<td style="text-align: left;">Rtsne</td>
<td style="text-align: left;">0.17</td>
<td style="text-align: left;">T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation</td>
</tr>
<tr>
<td style="text-align: left;">Rttf2pt1</td>
<td style="text-align: left;">1.3.12</td>
<td style="text-align: left;">'ttf2pt1' Program</td>
</tr>
<tr>
<td style="text-align: left;">RUnit</td>
<td style="text-align: left;">0.4.33</td>
<td style="text-align: left;">R Unit Test Framework</td>
</tr>
<tr>
<td style="text-align: left;">rvcheck</td>
<td style="text-align: left;">0.2.1</td>
<td style="text-align: left;">R/Package Version Check</td>
</tr>
<tr>
<td style="text-align: left;">RVenn</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">Set Operations for Many Sets</td>
</tr>
<tr>
<td style="text-align: left;">rversions</td>
<td style="text-align: left;">2.1.2</td>
<td style="text-align: left;">Query 'R' Versions, Including 'r-release' and 'r-oldrel'</td>
</tr>
<tr>
<td style="text-align: left;">rvest</td>
<td style="text-align: left;">1.0.4</td>
<td style="text-align: left;">Easily Harvest (Scrape) Web Pages</td>
</tr>
<tr>
<td style="text-align: left;">s2</td>
<td style="text-align: left;">1.1.7</td>
<td style="text-align: left;">Spherical Geometry Operators Using the S2 Geometry Library</td>
</tr>
<tr>
<td style="text-align: left;">S4Arrays</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">Foundation of array-like containers in Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">S4Vectors</td>
<td style="text-align: left;">0.46.0</td>
<td style="text-align: left;">Foundation of vector-like and list-like containers in Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">S7</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">An Object Oriented System Meant to Become a Successor to S3 and S4</td>
</tr>
<tr>
<td style="text-align: left;">safetensors</td>
<td style="text-align: left;">0.1.2</td>
<td style="text-align: left;">Safetensors File Format</td>
</tr>
<tr>
<td style="text-align: left;">samr</td>
<td style="text-align: left;">3.0</td>
<td style="text-align: left;">SAM: Significance Analysis of Microarrays</td>
</tr>
<tr>
<td style="text-align: left;">sandwich</td>
<td style="text-align: left;">3.1-1</td>
<td style="text-align: left;">Robust Covariance Matrix Estimators</td>
</tr>
<tr>
<td style="text-align: left;">sass</td>
<td style="text-align: left;">0.4.10</td>
<td style="text-align: left;">Syntactically Awesome Style Sheets ('Sass')</td>
</tr>
<tr>
<td style="text-align: left;">satellite</td>
<td style="text-align: left;">1.0.5</td>
<td style="text-align: left;">Handling and Manipulating Remote Sensing Data</td>
</tr>
<tr>
<td style="text-align: left;">satuRn</td>
<td style="text-align: left;">1.16.0</td>
<td style="text-align: left;">Scalable Analysis of Differential Transcript Usage for Bulk and Single-Cell RNA-sequencing Applications</td>
</tr>
<tr>
<td style="text-align: left;">ScaledMatrix</td>
<td style="text-align: left;">1.16.0</td>
<td style="text-align: left;">Creating a DelayedMatrix of Scaled and Centered Values</td>
</tr>
<tr>
<td style="text-align: left;">scales</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Scale Functions for Visualization</td>
</tr>
<tr>
<td style="text-align: left;">scam</td>
<td style="text-align: left;">1.2-18</td>
<td style="text-align: left;">Shape Constrained Additive Models</td>
</tr>
<tr>
<td style="text-align: left;">scater</td>
<td style="text-align: left;">1.36.0</td>
<td style="text-align: left;">Single-Cell Analysis Toolkit for Gene Expression Data in R</td>
</tr>
<tr>
<td style="text-align: left;">scattermore</td>
<td style="text-align: left;">1.2</td>
<td style="text-align: left;">Scatterplots with More Points</td>
</tr>
<tr>
<td style="text-align: left;">scatterpie</td>
<td style="text-align: left;">0.2.4</td>
<td style="text-align: left;">Scatter Pie Plot</td>
</tr>
<tr>
<td style="text-align: left;">scatterplot3d</td>
<td style="text-align: left;">0.3-44</td>
<td style="text-align: left;">3D Scatter Plot</td>
</tr>
<tr>
<td style="text-align: left;">SCENIC</td>
<td style="text-align: left;">1.3.1</td>
<td style="text-align: left;">SCENIC (Single Cell rEgulatory Network Inference and Clustering)</td>
</tr>
<tr>
<td style="text-align: left;">scider</td>
<td style="text-align: left;">1.6.0</td>
<td style="text-align: left;">Spatial cell-type inter-correlation by density in R</td>
</tr>
<tr>
<td style="text-align: left;">SCopeLoomR</td>
<td style="text-align: left;">0.13.0</td>
<td style="text-align: left;">Build .loom files (compatible with SCope) and extract data from .loom files.</td>
</tr>
<tr>
<td style="text-align: left;">SCPA</td>
<td style="text-align: left;">1.6.2</td>
<td style="text-align: left;">Single Cell Pathway Analysis</td>
</tr>
<tr>
<td style="text-align: left;">scrime</td>
<td style="text-align: left;">1.3.5</td>
<td style="text-align: left;">Analysis of High-Dimensional Categorical Data Such as SNP Data</td>
</tr>
<tr>
<td style="text-align: left;">sctransform</td>
<td style="text-align: left;">0.4.1</td>
<td style="text-align: left;">Variance Stabilizing Transformations for Single Cell UMI Data</td>
</tr>
<tr>
<td style="text-align: left;">scuttle</td>
<td style="text-align: left;">1.18.0</td>
<td style="text-align: left;">Single-Cell RNA-Seq Analysis Utilities</td>
</tr>
<tr>
<td style="text-align: left;">segmented</td>
<td style="text-align: left;">2.1-4</td>
<td style="text-align: left;">Regression Models with Break-Points / Change-Points Estimation (with Possibly Random Effects)</td>
</tr>
<tr>
<td style="text-align: left;">selectr</td>
<td style="text-align: left;">0.4-2</td>
<td style="text-align: left;">Translate CSS Selectors to XPath Expressions</td>
</tr>
<tr>
<td style="text-align: left;">sem</td>
<td style="text-align: left;">3.1-16</td>
<td style="text-align: left;">Structural Equation Models</td>
</tr>
<tr>
<td style="text-align: left;">semPlot</td>
<td style="text-align: left;">1.1.6</td>
<td style="text-align: left;">Path Diagrams and Visual Analysis of Various SEM Packages' Output</td>
</tr>
<tr>
<td style="text-align: left;">sendmailR</td>
<td style="text-align: left;">1.4-0</td>
<td style="text-align: left;">Send Email Using R</td>
</tr>
<tr>
<td style="text-align: left;">seqinr</td>
<td style="text-align: left;">4.2-36</td>
<td style="text-align: left;">Biological Sequences Retrieval and Analysis</td>
</tr>
<tr>
<td style="text-align: left;">seqLogo</td>
<td style="text-align: left;">1.74.0</td>
<td style="text-align: left;">Sequence logos for DNA sequence alignments</td>
</tr>
<tr>
<td style="text-align: left;">seqminer</td>
<td style="text-align: left;">9.7</td>
<td style="text-align: left;">Efficiently Read Sequence Data (VCF Format, BCF Format, METAL Format and BGEN Format) into R</td>
</tr>
<tr>
<td style="text-align: left;">seriation</td>
<td style="text-align: left;">1.5.7</td>
<td style="text-align: left;">Infrastructure for Ordering Objects Using Seriation</td>
</tr>
<tr>
<td style="text-align: left;">servr</td>
<td style="text-align: left;">0.32</td>
<td style="text-align: left;">A Simple HTTP Server to Serve Static Files or Dynamic Documents</td>
</tr>
<tr>
<td style="text-align: left;">sessioninfo</td>
<td style="text-align: left;">1.2.3</td>
<td style="text-align: left;">R Session Information</td>
</tr>
<tr>
<td style="text-align: left;">Seurat</td>
<td style="text-align: left;">5.3.0</td>
<td style="text-align: left;">Tools for Single Cell Genomics</td>
</tr>
<tr>
<td style="text-align: left;">SeuratData</td>
<td style="text-align: left;">0.2.2.9002</td>
<td style="text-align: left;">Install and Manage Seurat Datasets</td>
</tr>
<tr>
<td style="text-align: left;">SeuratDisk</td>
<td style="text-align: left;">0.0.0.9021</td>
<td style="text-align: left;">Interfaces for HDF5-Based Single Cell File Formats</td>
</tr>
<tr>
<td style="text-align: left;">SeuratObject</td>
<td style="text-align: left;">5.1.0</td>
<td style="text-align: left;">Data Structures for Single Cell Data</td>
</tr>
<tr>
<td style="text-align: left;">SeuratWrappers</td>
<td style="text-align: left;">0.4.0</td>
<td style="text-align: left;">Community-Provided Methods and Extensions for the Seurat Object</td>
</tr>
<tr>
<td style="text-align: left;">sf</td>
<td style="text-align: left;">1.0-20</td>
<td style="text-align: left;">Simple Features for R</td>
</tr>
<tr>
<td style="text-align: left;">sfd</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">Space-Filling Design Library</td>
</tr>
<tr>
<td style="text-align: left;">sfheaders</td>
<td style="text-align: left;">0.4.4</td>
<td style="text-align: left;">Converts Between R Objects and Simple Feature Objects</td>
</tr>
<tr>
<td style="text-align: left;">sftime</td>
<td style="text-align: left;">0.3.0</td>
<td style="text-align: left;">Classes and Methods for Simple Feature Objects that Have a Time Column</td>
</tr>
<tr>
<td style="text-align: left;">shadowtext</td>
<td style="text-align: left;">0.1.4</td>
<td style="text-align: left;">Shadow Text Grob and Layer</td>
</tr>
<tr>
<td style="text-align: left;">shape</td>
<td style="text-align: left;">1.4.6.1</td>
<td style="text-align: left;">Functions for Plotting Graphical Shapes, Colors</td>
</tr>
<tr>
<td style="text-align: left;">shiny</td>
<td style="text-align: left;">1.10.0</td>
<td style="text-align: left;">Web Application Framework for R</td>
</tr>
<tr>
<td style="text-align: left;">shinyAce</td>
<td style="text-align: left;">0.4.4</td>
<td style="text-align: left;">Ace Editor Bindings for Shiny</td>
</tr>
<tr>
<td style="text-align: left;">shinyBS</td>
<td style="text-align: left;">0.61.1</td>
<td style="text-align: left;">Twitter Bootstrap Components for Shiny</td>
</tr>
<tr>
<td style="text-align: left;">shinycssloaders</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">Add Loading Animations to a 'shiny' Output While It's Recalculating</td>
</tr>
<tr>
<td style="text-align: left;">shinydashboard</td>
<td style="text-align: left;">0.7.3</td>
<td style="text-align: left;">Create Dashboards with 'Shiny'</td>
</tr>
<tr>
<td style="text-align: left;">shinyFiles</td>
<td style="text-align: left;">0.9.3</td>
<td style="text-align: left;">A Server-Side File System Viewer for Shiny</td>
</tr>
<tr>
<td style="text-align: left;">shinyjs</td>
<td style="text-align: left;">2.1.0</td>
<td style="text-align: left;">Easily Improve the User Experience of Your Shiny Apps in Seconds</td>
</tr>
<tr>
<td style="text-align: left;">shinystan</td>
<td style="text-align: left;">2.6.0</td>
<td style="text-align: left;">Interactive Visual and Numerical Diagnostics and Posterior Analysis for Bayesian Models</td>
</tr>
<tr>
<td style="text-align: left;">shinythemes</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">Themes for Shiny</td>
</tr>
<tr>
<td style="text-align: left;">shinyWidgets</td>
<td style="text-align: left;">0.9.0</td>
<td style="text-align: left;">Custom Inputs Widgets for Shiny</td>
</tr>
<tr>
<td style="text-align: left;">ShortRead</td>
<td style="text-align: left;">1.66.0</td>
<td style="text-align: left;">FASTQ input and manipulation</td>
</tr>
<tr>
<td style="text-align: left;">siggenes</td>
<td style="text-align: left;">1.82.0</td>
<td style="text-align: left;">Multiple Testing using SAM and Efron's Empirical Bayes Approaches</td>
</tr>
<tr>
<td style="text-align: left;">Signac</td>
<td style="text-align: left;">1.14.0</td>
<td style="text-align: left;">Analysis of Single-Cell Chromatin Data</td>
</tr>
<tr>
<td style="text-align: left;">simdistr</td>
<td style="text-align: left;">1.0.1</td>
<td style="text-align: left;">Assessment of Data Trial Distributions According to the Carlisle-Stouffer Method</td>
</tr>
<tr>
<td style="text-align: left;">SingleCellExperiment</td>
<td style="text-align: left;">1.30.0</td>
<td style="text-align: left;">S4 Classes for Single Cell Data</td>
</tr>
<tr>
<td style="text-align: left;">SingleR</td>
<td style="text-align: left;">2.10.0</td>
<td style="text-align: left;">Reference-Based Single-Cell RNA-Seq Annotation</td>
</tr>
<tr>
<td style="text-align: left;">singscore</td>
<td style="text-align: left;">1.28.0</td>
<td style="text-align: left;">Rank-based single-sample gene set scoring method</td>
</tr>
<tr>
<td style="text-align: left;">sitmo</td>
<td style="text-align: left;">2.0.2</td>
<td style="text-align: left;">Parallel Pseudo Random Number Generator (PPRNG) 'sitmo' Header Files</td>
</tr>
<tr>
<td style="text-align: left;">SKAT</td>
<td style="text-align: left;">2.2.5</td>
<td style="text-align: left;">SNP-Set (Sequence) Kernel Association Test</td>
</tr>
<tr>
<td style="text-align: left;">slam</td>
<td style="text-align: left;">0.1-55</td>
<td style="text-align: left;">Sparse Lightweight Arrays and Matrices</td>
</tr>
<tr>
<td style="text-align: left;">slider</td>
<td style="text-align: left;">0.3.2</td>
<td style="text-align: left;">Sliding Window Functions</td>
</tr>
<tr>
<td style="text-align: left;">smacof</td>
<td style="text-align: left;">2.1-7</td>
<td style="text-align: left;">Multidimensional Scaling</td>
</tr>
<tr>
<td style="text-align: left;">smoof</td>
<td style="text-align: left;">1.6.0.3</td>
<td style="text-align: left;">Single and Multi-Objective Optimization Test Functions</td>
</tr>
<tr>
<td style="text-align: left;">sn</td>
<td style="text-align: left;">2.1.1</td>
<td style="text-align: left;">The Skew-Normal and Related Distributions Such as the Skew-t and the SUN</td>
</tr>
<tr>
<td style="text-align: left;">sna</td>
<td style="text-align: left;">2.8</td>
<td style="text-align: left;">Tools for Social Network Analysis</td>
</tr>
<tr>
<td style="text-align: left;">snakecase</td>
<td style="text-align: left;">0.11.1</td>
<td style="text-align: left;">Convert Strings into any Case</td>
</tr>
<tr>
<td style="text-align: left;">snow</td>
<td style="text-align: left;">0.4-4</td>
<td style="text-align: left;">Simple Network of Workstations</td>
</tr>
<tr>
<td style="text-align: left;">SNPRelate</td>
<td style="text-align: left;">1.42.0</td>
<td style="text-align: left;">Parallel Computing Toolset for Relatedness and Principal Component Analysis of SNP Data</td>
</tr>
<tr>
<td style="text-align: left;">soGGi</td>
<td style="text-align: left;">1.40.0</td>
<td style="text-align: left;">Visualise ChIP-seq, MNase-seq and motif occurrence as aggregate plots Summarised Over Grouped Genomic Intervals</td>
</tr>
<tr>
<td style="text-align: left;">sourcetools</td>
<td style="text-align: left;">0.1.7-1</td>
<td style="text-align: left;">Tools for Reading, Tokenizing and Parsing R Code</td>
</tr>
<tr>
<td style="text-align: left;">sp</td>
<td style="text-align: left;">2.2-0</td>
<td style="text-align: left;">Classes and Methods for Spatial Data</td>
</tr>
<tr>
<td style="text-align: left;">spacesXYZ</td>
<td style="text-align: left;">1.5-1</td>
<td style="text-align: left;">CIE XYZ and some of Its Derived Color Spaces</td>
</tr>
<tr>
<td style="text-align: left;">spacetime</td>
<td style="text-align: left;">1.3-3</td>
<td style="text-align: left;">Classes and Methods for Spatio-Temporal Data</td>
</tr>
<tr>
<td style="text-align: left;">spam</td>
<td style="text-align: left;">2.11-1</td>
<td style="text-align: left;">SPArse Matrix</td>
</tr>
<tr>
<td style="text-align: left;">SparseArray</td>
<td style="text-align: left;">1.8.0</td>
<td style="text-align: left;">High-performance sparse data representation and manipulation in R</td>
</tr>
<tr>
<td style="text-align: left;">SparseM</td>
<td style="text-align: left;">1.84-2</td>
<td style="text-align: left;">Sparse Linear Algebra</td>
</tr>
<tr>
<td style="text-align: left;">sparseMatrixStats</td>
<td style="text-align: left;">1.20.0</td>
<td style="text-align: left;">Summary Statistics for Rows and Columns of Sparse Matrices</td>
</tr>
<tr>
<td style="text-align: left;">sparsesvd</td>
<td style="text-align: left;">0.2-2</td>
<td style="text-align: left;">Sparse Truncated Singular Value Decomposition (from 'SVDLIBC')</td>
</tr>
<tr>
<td style="text-align: left;">sparsevctrs</td>
<td style="text-align: left;">0.3.3</td>
<td style="text-align: left;">Sparse Vectors for Use in Data Frames</td>
</tr>
<tr>
<td style="text-align: left;">SPAtest</td>
<td style="text-align: left;">3.1.2</td>
<td style="text-align: left;">Score Test and Meta-Analysis Based on Saddlepoint Approximation</td>
</tr>
<tr>
<td style="text-align: left;">spatial</td>
<td style="text-align: left;">7.3-18</td>
<td style="text-align: left;">Functions for Kriging and Point Pattern Analysis</td>
</tr>
<tr>
<td style="text-align: left;">SpatialExperiment</td>
<td style="text-align: left;">1.18.0</td>
<td style="text-align: left;">S4 Class for Spatially Resolved -omics Data</td>
</tr>
<tr>
<td style="text-align: left;">spatstat</td>
<td style="text-align: left;">3.3-2</td>
<td style="text-align: left;">Spatial Point Pattern Analysis, Model-Fitting, Simulation, Tests</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.data</td>
<td style="text-align: left;">3.1-6</td>
<td style="text-align: left;">Datasets for 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.explore</td>
<td style="text-align: left;">3.4-2</td>
<td style="text-align: left;">Exploratory Data Analysis for the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.geom</td>
<td style="text-align: left;">3.3-6</td>
<td style="text-align: left;">Geometrical Functionality of the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.linnet</td>
<td style="text-align: left;">3.2-5</td>
<td style="text-align: left;">Linear Networks Functionality of the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.model</td>
<td style="text-align: left;">3.3-5</td>
<td style="text-align: left;">Parametric Statistical Modelling and Inference for the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.random</td>
<td style="text-align: left;">3.3-3</td>
<td style="text-align: left;">Random Generation Functionality for the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.sparse</td>
<td style="text-align: left;">3.1-0</td>
<td style="text-align: left;">Sparse Three-Dimensional Arrays and Linear Algebra Utilities</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.univar</td>
<td style="text-align: left;">3.1-2</td>
<td style="text-align: left;">One-Dimensional Probability Distribution Support for the 'spatstat' Family</td>
</tr>
<tr>
<td style="text-align: left;">spatstat.utils</td>
<td style="text-align: left;">3.1-3</td>
<td style="text-align: left;">Utility Functions for 'spatstat'</td>
</tr>
<tr>
<td style="text-align: left;">spData</td>
<td style="text-align: left;">2.3.4</td>
<td style="text-align: left;">Datasets for Spatial Analysis</td>
</tr>
<tr>
<td style="text-align: left;">spdep</td>
<td style="text-align: left;">1.3-11</td>
<td style="text-align: left;">Spatial Dependence: Weighting Schemes, Statistics</td>
</tr>
<tr>
<td style="text-align: left;">Spectra</td>
<td style="text-align: left;">1.18.2</td>
<td style="text-align: left;">Spectra Infrastructure for Mass Spectrometry Data</td>
</tr>
<tr>
<td style="text-align: left;">speedglm</td>
<td style="text-align: left;">0.3-5</td>
<td style="text-align: left;">Fitting Linear and Generalized Linear Models to Large Data Sets</td>
</tr>
<tr>
<td style="text-align: left;">SpliceWiz</td>
<td style="text-align: left;">1.10.0</td>
<td style="text-align: left;">interactive analysis and visualization of alternative splicing in R</td>
</tr>
<tr>
<td style="text-align: left;">splines</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">Regression Spline Functions and Classes</td>
</tr>
<tr>
<td style="text-align: left;">splitstackshape</td>
<td style="text-align: left;">1.4.8</td>
<td style="text-align: left;">Stack and Reshape Datasets After Splitting Concatenated Values</td>
</tr>
<tr>
<td style="text-align: left;">splus2R</td>
<td style="text-align: left;">1.3-5</td>
<td style="text-align: left;">Supplemental S-PLUS Functionality in R</td>
</tr>
<tr>
<td style="text-align: left;">sqldf</td>
<td style="text-align: left;">0.4-11</td>
<td style="text-align: left;">Manipulate R Data Frames Using SQL</td>
</tr>
<tr>
<td style="text-align: left;">SQUAREM</td>
<td style="text-align: left;">2021.1</td>
<td style="text-align: left;">Squared Extrapolation Methods for Accelerating EM-Like Monotone Algorithms</td>
</tr>
<tr>
<td style="text-align: left;">squash</td>
<td style="text-align: left;">1.0.9</td>
<td style="text-align: left;">Color-Based Plots for Multivariate Visualization</td>
</tr>
<tr>
<td style="text-align: left;">StanHeaders</td>
<td style="text-align: left;">2.32.10</td>
<td style="text-align: left;">C++ Header Files for Stan</td>
</tr>
<tr>
<td style="text-align: left;">stapler</td>
<td style="text-align: left;">0.8.0</td>
<td style="text-align: left;">Simultaneous Truth and Performance Level Estimation</td>
</tr>
<tr>
<td style="text-align: left;">stars</td>
<td style="text-align: left;">0.6-8</td>
<td style="text-align: left;">Spatiotemporal Arrays, Raster and Vector Data Cubes</td>
</tr>
<tr>
<td style="text-align: left;">statmod</td>
<td style="text-align: left;">1.5.0</td>
<td style="text-align: left;">Statistical Modeling</td>
</tr>
<tr>
<td style="text-align: left;">statnet.common</td>
<td style="text-align: left;">4.11.0</td>
<td style="text-align: left;">Common R Scripts and Utilities Used by the Statnet Project Software</td>
</tr>
<tr>
<td style="text-align: left;">stats</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">The R Stats Package</td>
</tr>
<tr>
<td style="text-align: left;">stats4</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">Statistical Functions using S4 Classes</td>
</tr>
<tr>
<td style="text-align: left;">STRINGdb</td>
<td style="text-align: left;">2.20.0</td>
<td style="text-align: left;">STRINGdb - Protein-Protein Interaction Networks and Functional Enrichment Analysis</td>
</tr>
<tr>
<td style="text-align: left;">stringdist</td>
<td style="text-align: left;">0.9.15</td>
<td style="text-align: left;">Approximate String Matching, Fuzzy Text Search, and String Distance Functions</td>
</tr>
<tr>
<td style="text-align: left;">stringfish</td>
<td style="text-align: left;">0.17.0</td>
<td style="text-align: left;">Alt String Implementation</td>
</tr>
<tr>
<td style="text-align: left;">stringi</td>
<td style="text-align: left;">1.8.7</td>
<td style="text-align: left;">Fast and Portable Character String Processing Facilities</td>
</tr>
<tr>
<td style="text-align: left;">stringr</td>
<td style="text-align: left;">1.5.1</td>
<td style="text-align: left;">Simple, Consistent Wrappers for Common String Operations</td>
</tr>
<tr>
<td style="text-align: left;">StructuralVariantAnnotation</td>
<td style="text-align: left;">1.24.0</td>
<td style="text-align: left;">Variant annotations for structural variants</td>
</tr>
<tr>
<td style="text-align: left;">styler</td>
<td style="text-align: left;">1.10.3</td>
<td style="text-align: left;">Non-Invasive Pretty Printing of R Code</td>
</tr>
<tr>
<td style="text-align: left;">SummarizedExperiment</td>
<td style="text-align: left;">1.38.0</td>
<td style="text-align: left;">A container (S4 class) for matrix-like assays</td>
</tr>
<tr>
<td style="text-align: left;">SuperLearner</td>
<td style="text-align: left;">2.0-29</td>
<td style="text-align: left;">Super Learner Prediction</td>
</tr>
<tr>
<td style="text-align: left;">survey</td>
<td style="text-align: left;">4.4-2</td>
<td style="text-align: left;">Analysis of Complex Survey Samples</td>
</tr>
<tr>
<td style="text-align: left;">survival</td>
<td style="text-align: left;">3.8-3</td>
<td style="text-align: left;">Survival Analysis</td>
</tr>
<tr>
<td style="text-align: left;">survminer</td>
<td style="text-align: left;">0.5.0</td>
<td style="text-align: left;">Drawing Survival Curves using 'ggplot2'</td>
</tr>
<tr>
<td style="text-align: left;">survMisc</td>
<td style="text-align: left;">0.5.6</td>
<td style="text-align: left;">Miscellaneous Functions for Survival Data</td>
</tr>
<tr>
<td style="text-align: left;">sva</td>
<td style="text-align: left;">3.56.0</td>
<td style="text-align: left;">Surrogate Variable Analysis</td>
</tr>
<tr>
<td style="text-align: left;">svglite</td>
<td style="text-align: left;">2.1.3</td>
<td style="text-align: left;">An 'SVG' Graphics Device</td>
</tr>
<tr>
<td style="text-align: left;">svUnit</td>
<td style="text-align: left;">1.0.6</td>
<td style="text-align: left;">'SciViews' - Unit, Integration and System Testing</td>
</tr>
<tr>
<td style="text-align: left;">sys</td>
<td style="text-align: left;">3.4.3</td>
<td style="text-align: left;">Powerful and Reliable Tools for Running System Commands in R</td>
</tr>
<tr>
<td style="text-align: left;">systemfonts</td>
<td style="text-align: left;">1.2.2</td>
<td style="text-align: left;">System Native Font Finding</td>
</tr>
<tr>
<td style="text-align: left;">systemPipeR</td>
<td style="text-align: left;">2.14.0</td>
<td style="text-align: left;">systemPipeR: Workflow Environment for Data Analysis and Report Generation</td>
</tr>
<tr>
<td style="text-align: left;">tcltk</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">Tcl/Tk Interface</td>
</tr>
<tr>
<td style="text-align: left;">tensor</td>
<td style="text-align: left;">1.5</td>
<td style="text-align: left;">Tensor product of arrays</td>
</tr>
<tr>
<td style="text-align: left;">tensorA</td>
<td style="text-align: left;">0.36.2.1</td>
<td style="text-align: left;">Advanced Tensor Arithmetic with Named Indices</td>
</tr>
<tr>
<td style="text-align: left;">tensorflow</td>
<td style="text-align: left;">2.20.0</td>
<td style="text-align: left;">R Interface to 'TensorFlow'</td>
</tr>
<tr>
<td style="text-align: left;">terra</td>
<td style="text-align: left;">1.8-42</td>
<td style="text-align: left;">Spatial Data Analysis</td>
</tr>
<tr>
<td style="text-align: left;">testthat</td>
<td style="text-align: left;">3.2.3</td>
<td style="text-align: left;">Unit Testing for R</td>
</tr>
<tr>
<td style="text-align: left;">textshaping</td>
<td style="text-align: left;">1.0.0</td>
<td style="text-align: left;">Bindings to the 'HarfBuzz' and 'Fribidi' Libraries for Text Shaping</td>
</tr>
<tr>
<td style="text-align: left;">tfautograph</td>
<td style="text-align: left;">0.3.2</td>
<td style="text-align: left;">Autograph R for 'Tensorflow'</td>
</tr>
<tr>
<td style="text-align: left;">TFBSTools</td>
<td style="text-align: left;">1.46.0</td>
<td style="text-align: left;">Software Package for Transcription Factor Binding Site (TFBS) Analysis</td>
</tr>
<tr>
<td style="text-align: left;">TFisher</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">Optimal Thresholding Fisher's P-Value Combination Method</td>
</tr>
<tr>
<td style="text-align: left;">TFMPvalue</td>
<td style="text-align: left;">0.0.9</td>
<td style="text-align: left;">Efficient and Accurate P-Value Computation for Position Weight Matrices</td>
</tr>
<tr>
<td style="text-align: left;">tfruns</td>
<td style="text-align: left;">1.5.4</td>
<td style="text-align: left;">Training Run Tools for 'TensorFlow'</td>
</tr>
<tr>
<td style="text-align: left;">TH.data</td>
<td style="text-align: left;">1.1-3</td>
<td style="text-align: left;">TH's Data Archive</td>
</tr>
<tr>
<td style="text-align: left;">threejs</td>
<td style="text-align: left;">0.3.4</td>
<td style="text-align: left;">Interactive 3D Scatter Plots, Networks and Globes</td>
</tr>
<tr>
<td style="text-align: left;">tibble</td>
<td style="text-align: left;">3.2.1</td>
<td style="text-align: left;">Simple Data Frames</td>
</tr>
<tr>
<td style="text-align: left;">tidybayes</td>
<td style="text-align: left;">3.0.7</td>
<td style="text-align: left;">Tidy Data and 'Geoms' for Bayesian Models</td>
</tr>
<tr>
<td style="text-align: left;">tidycmprsk</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">Competing Risks Estimation</td>
</tr>
<tr>
<td style="text-align: left;">tidygraph</td>
<td style="text-align: left;">1.3.1</td>
<td style="text-align: left;">A Tidy API for Graph Manipulation</td>
</tr>
<tr>
<td style="text-align: left;">tidymodels</td>
<td style="text-align: left;">1.3.0</td>
<td style="text-align: left;">Easily Install and Load the 'Tidymodels' Packages</td>
</tr>
<tr>
<td style="text-align: left;">tidyr</td>
<td style="text-align: left;">1.3.1</td>
<td style="text-align: left;">Tidy Messy Data</td>
</tr>
<tr>
<td style="text-align: left;">tidyselect</td>
<td style="text-align: left;">1.2.1</td>
<td style="text-align: left;">Select from a Set of Strings</td>
</tr>
<tr>
<td style="text-align: left;">tidytree</td>
<td style="text-align: left;">0.4.6</td>
<td style="text-align: left;">A Tidy Tool for Phylogenetic Tree Data Manipulation</td>
</tr>
<tr>
<td style="text-align: left;">tidyverse</td>
<td style="text-align: left;">2.0.0</td>
<td style="text-align: left;">Easily Install and Load the 'Tidyverse'</td>
</tr>
<tr>
<td style="text-align: left;">tiff</td>
<td style="text-align: left;">0.1-12</td>
<td style="text-align: left;">Read and Write TIFF Images</td>
</tr>
<tr>
<td style="text-align: left;">timechange</td>
<td style="text-align: left;">0.3.0</td>
<td style="text-align: left;">Efficient Manipulation of Date-Times</td>
</tr>
<tr>
<td style="text-align: left;">timeDate</td>
<td style="text-align: left;">4041.110</td>
<td style="text-align: left;">Rmetrics - Chronological and Calendar Objects</td>
</tr>
<tr>
<td style="text-align: left;">timereg</td>
<td style="text-align: left;">2.0.6</td>
<td style="text-align: left;">Flexible Regression Models for Survival Data</td>
</tr>
<tr>
<td style="text-align: left;">tinytex</td>
<td style="text-align: left;">0.57</td>
<td style="text-align: left;">Helper Functions to Install and Maintain TeX Live, and Compile LaTeX Documents</td>
</tr>
<tr>
<td style="text-align: left;">tippy</td>
<td style="text-align: left;">0.1.0</td>
<td style="text-align: left;">Add Tooltips to 'R markdown' Documents or 'Shiny' Apps</td>
</tr>
<tr>
<td style="text-align: left;">tmap</td>
<td style="text-align: left;">4.0</td>
<td style="text-align: left;">Thematic Maps</td>
</tr>
<tr>
<td style="text-align: left;">tmaptools</td>
<td style="text-align: left;">3.2</td>
<td style="text-align: left;">Thematic Map Tools</td>
</tr>
<tr>
<td style="text-align: left;">tmvnsim</td>
<td style="text-align: left;">1.0-2</td>
<td style="text-align: left;">Truncated Multivariate Normal Simulation</td>
</tr>
<tr>
<td style="text-align: left;">tools</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">Tools for Package Development</td>
</tr>
<tr>
<td style="text-align: left;">topGO</td>
<td style="text-align: left;">2.59.0</td>
<td style="text-align: left;">Enrichment Analysis for Gene Ontology</td>
</tr>
<tr>
<td style="text-align: left;">torch</td>
<td style="text-align: left;">0.14.2</td>
<td style="text-align: left;">Tensors and Neural Networks with 'GPU' Acceleration</td>
</tr>
<tr>
<td style="text-align: left;">treeio</td>
<td style="text-align: left;">1.32.0</td>
<td style="text-align: left;">Base Classes and Functions for Phylogenetic Tree Input and Output</td>
</tr>
<tr>
<td style="text-align: left;">triebeard</td>
<td style="text-align: left;">0.4.1</td>
<td style="text-align: left;">'Radix' Trees in 'Rcpp'</td>
</tr>
<tr>
<td style="text-align: left;">truncnorm</td>
<td style="text-align: left;">1.0-9</td>
<td style="text-align: left;">Truncated Normal Distribution</td>
</tr>
<tr>
<td style="text-align: left;">TSP</td>
<td style="text-align: left;">1.2-4</td>
<td style="text-align: left;">Traveling Salesperson Problem (TSP)</td>
</tr>
<tr>
<td style="text-align: left;">tune</td>
<td style="text-align: left;">1.3.0</td>
<td style="text-align: left;">Tidy Tuning Tools</td>
</tr>
<tr>
<td style="text-align: left;">twang</td>
<td style="text-align: left;">2.6.1</td>
<td style="text-align: left;">Toolkit for Weighting and Analysis of Nonequivalent Groups</td>
</tr>
<tr>
<td style="text-align: left;">tweenr</td>
<td style="text-align: left;">2.0.3</td>
<td style="text-align: left;">Interpolate Data for Smooth Animations</td>
</tr>
<tr>
<td style="text-align: left;">TxDb.Hsapiens.UCSC.hg19.knownGene</td>
<td style="text-align: left;">3.2.2</td>
<td style="text-align: left;">Annotation package for TxDb object(s)</td>
</tr>
<tr>
<td style="text-align: left;">TxDb.Hsapiens.UCSC.hg38.knownGene</td>
<td style="text-align: left;">3.21.0</td>
<td style="text-align: left;">Annotation package for TxDb object(s)</td>
</tr>
<tr>
<td style="text-align: left;">TxDb.Mmusculus.UCSC.mm10.knownGene</td>
<td style="text-align: left;">3.10.0</td>
<td style="text-align: left;">Annotation package for TxDb object(s)</td>
</tr>
<tr>
<td style="text-align: left;">TxDb.Mmusculus.UCSC.mm9.knownGene</td>
<td style="text-align: left;">3.2.2</td>
<td style="text-align: left;">Annotation package for TxDb object(s)</td>
</tr>
<tr>
<td style="text-align: left;">txdbmaker</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Tools for making TxDb objects from genomic annotations</td>
</tr>
<tr>
<td style="text-align: left;">tximeta</td>
<td style="text-align: left;">1.26.0</td>
<td style="text-align: left;">Transcript Quantification Import with Automatic Metadata</td>
</tr>
<tr>
<td style="text-align: left;">tximport</td>
<td style="text-align: left;">1.36.0</td>
<td style="text-align: left;">Import and summarize transcript-level estimates for transcript- and gene-level analysis</td>
</tr>
<tr>
<td style="text-align: left;">tzdb</td>
<td style="text-align: left;">0.5.0</td>
<td style="text-align: left;">Time Zone Database Information</td>
</tr>
<tr>
<td style="text-align: left;">ucminf</td>
<td style="text-align: left;">1.2.2</td>
<td style="text-align: left;">General-Purpose Unconstrained Non-Linear Optimization</td>
</tr>
<tr>
<td style="text-align: left;">UCSC.utils</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">Low-level utilities to retrieve data from the UCSC Genome Browser</td>
</tr>
<tr>
<td style="text-align: left;">UCSCXenaTools</td>
<td style="text-align: left;">1.4.8</td>
<td style="text-align: left;">Download and Explore Datasets from UCSC Xena Data Hubs</td>
</tr>
<tr>
<td style="text-align: left;">umap</td>
<td style="text-align: left;">0.2.10.0</td>
<td style="text-align: left;">Uniform Manifold Approximation and Projection</td>
</tr>
<tr>
<td style="text-align: left;">UniprotR</td>
<td style="text-align: left;">2.4.0</td>
<td style="text-align: left;">Retrieving Information of Proteins from Uniprot</td>
</tr>
<tr>
<td style="text-align: left;">units</td>
<td style="text-align: left;">0.8-7</td>
<td style="text-align: left;">Measurement Units for R Vectors</td>
</tr>
<tr>
<td style="text-align: left;">UpSetR</td>
<td style="text-align: left;">1.4.0</td>
<td style="text-align: left;">A More Scalable Alternative to Venn and Euler Diagrams for Visualizing Intersecting Sets</td>
</tr>
<tr>
<td style="text-align: left;">urlchecker</td>
<td style="text-align: left;">1.0.1</td>
<td style="text-align: left;">Run CRAN URL Checks from Older R Versions</td>
</tr>
<tr>
<td style="text-align: left;">urltools</td>
<td style="text-align: left;">1.7.3</td>
<td style="text-align: left;">Vectorised Tools for URL Handling and Parsing</td>
</tr>
<tr>
<td style="text-align: left;">useful</td>
<td style="text-align: left;">1.2.6.1</td>
<td style="text-align: left;">A Collection of Handy, Useful Functions</td>
</tr>
<tr>
<td style="text-align: left;">usethis</td>
<td style="text-align: left;">3.2.1</td>
<td style="text-align: left;">Automate Package and Project Setup</td>
</tr>
<tr>
<td style="text-align: left;">utf8</td>
<td style="text-align: left;">1.2.4</td>
<td style="text-align: left;">Unicode Text Processing</td>
</tr>
<tr>
<td style="text-align: left;">utils</td>
<td style="text-align: left;">4.5.0</td>
<td style="text-align: left;">The R Utils Package</td>
</tr>
<tr>
<td style="text-align: left;">uuid</td>
<td style="text-align: left;">1.2-1</td>
<td style="text-align: left;">Tools for Generating and Handling of UUIDs</td>
</tr>
<tr>
<td style="text-align: left;">uwot</td>
<td style="text-align: left;">0.2.3</td>
<td style="text-align: left;">The Uniform Manifold Approximation and Projection (UMAP) Method for Dimensionality Reduction</td>
</tr>
<tr>
<td style="text-align: left;">V8</td>
<td style="text-align: left;">6.0.3</td>
<td style="text-align: left;">Embedded JavaScript and WebAssembly Engine for R</td>
</tr>
<tr>
<td style="text-align: left;">variables</td>
<td style="text-align: left;">1.1-1</td>
<td style="text-align: left;">Variable Descriptions</td>
</tr>
<tr>
<td style="text-align: left;">variancePartition</td>
<td style="text-align: left;">1.38.0</td>
<td style="text-align: left;">Quantify and interpret drivers of variation in multilevel gene expression experiments</td>
</tr>
<tr>
<td style="text-align: left;">VariantAnnotation</td>
<td style="text-align: left;">1.54.0</td>
<td style="text-align: left;">Annotation of Genetic Variants</td>
</tr>
<tr>
<td style="text-align: left;">vcd</td>
<td style="text-align: left;">1.4-13</td>
<td style="text-align: left;">Visualizing Categorical Data</td>
</tr>
<tr>
<td style="text-align: left;">vcfR</td>
<td style="text-align: left;">1.15.0</td>
<td style="text-align: left;">Manipulate and Visualize VCF Data</td>
</tr>
<tr>
<td style="text-align: left;">vctrs</td>
<td style="text-align: left;">0.6.5</td>
<td style="text-align: left;">Vector Helpers</td>
</tr>
<tr>
<td style="text-align: left;">vegan</td>
<td style="text-align: left;">2.6-10</td>
<td style="text-align: left;">Community Ecology Package</td>
</tr>
<tr>
<td style="text-align: left;">velocyto.R</td>
<td style="text-align: left;">0.6</td>
<td style="text-align: left;">RNA velocity estimation in R</td>
</tr>
<tr>
<td style="text-align: left;">venn</td>
<td style="text-align: left;">1.12</td>
<td style="text-align: left;">Draw Venn Diagrams</td>
</tr>
<tr>
<td style="text-align: left;">VennDiagram</td>
<td style="text-align: left;">1.7.3</td>
<td style="text-align: left;">Generate High-Resolution Venn and Euler Plots</td>
</tr>
<tr>
<td style="text-align: left;">VGAM</td>
<td style="text-align: left;">1.1-13</td>
<td style="text-align: left;">Vector Generalized Linear and Additive Models</td>
</tr>
<tr>
<td style="text-align: left;">VIM</td>
<td style="text-align: left;">6.2.2</td>
<td style="text-align: left;">Visualization and Imputation of Missing Values</td>
</tr>
<tr>
<td style="text-align: left;">vipor</td>
<td style="text-align: left;">0.4.7</td>
<td style="text-align: left;">Plot Categorical Data Using Quasirandom Noise and Density Estimates</td>
</tr>
<tr>
<td style="text-align: left;">viridis</td>
<td style="text-align: left;">0.6.5</td>
<td style="text-align: left;">Colorblind-Friendly Color Maps for R</td>
</tr>
<tr>
<td style="text-align: left;">viridisLite</td>
<td style="text-align: left;">0.4.2</td>
<td style="text-align: left;">Colorblind-Friendly Color Maps (Lite Version)</td>
</tr>
<tr>
<td style="text-align: left;">visNetwork</td>
<td style="text-align: left;">2.1.2</td>
<td style="text-align: left;">Network Visualization using 'vis.js' Library</td>
</tr>
<tr>
<td style="text-align: left;">vroom</td>
<td style="text-align: left;">1.6.5</td>
<td style="text-align: left;">Read and Write Rectangular Text Data Quickly</td>
</tr>
<tr>
<td style="text-align: left;">vsn</td>
<td style="text-align: left;">3.76.0</td>
<td style="text-align: left;">Variance stabilization and calibration for microarray data</td>
</tr>
<tr>
<td style="text-align: left;">waiter</td>
<td style="text-align: left;">0.2.5</td>
<td style="text-align: left;">Loading Screen for 'Shiny'</td>
</tr>
<tr>
<td style="text-align: left;">waldo</td>
<td style="text-align: left;">0.6.1</td>
<td style="text-align: left;">Find Differences Between R Objects</td>
</tr>
<tr>
<td style="text-align: left;">warp</td>
<td style="text-align: left;">0.2.1</td>
<td style="text-align: left;">Group Dates</td>
</tr>
<tr>
<td style="text-align: left;">webshot</td>
<td style="text-align: left;">0.5.5</td>
<td style="text-align: left;">Take Screenshots of Web Pages</td>
</tr>
<tr>
<td style="text-align: left;">weights</td>
<td style="text-align: left;">1.0.4</td>
<td style="text-align: left;">Weighting and Weighted Statistics</td>
</tr>
<tr>
<td style="text-align: left;">WGCNA</td>
<td style="text-align: left;">1.73</td>
<td style="text-align: left;">Weighted Correlation Network Analysis</td>
</tr>
<tr>
<td style="text-align: left;">whisker</td>
<td style="text-align: left;">0.4.1</td>
<td style="text-align: left;">{{mustache}} for R, Logicless Templating</td>
</tr>
<tr>
<td style="text-align: left;">WhiteStripe</td>
<td style="text-align: left;">2.4.3</td>
<td style="text-align: left;">White Matter Normalization for Magnetic Resonance Images</td>
</tr>
<tr>
<td style="text-align: left;">widgetframe</td>
<td style="text-align: left;">0.3.1</td>
<td style="text-align: left;">'Htmlwidgets' in Responsive 'iframes'</td>
</tr>
<tr>
<td style="text-align: left;">withr</td>
<td style="text-align: left;">3.0.2</td>
<td style="text-align: left;">Run Code 'With' Temporarily Modified Global State</td>
</tr>
<tr>
<td style="text-align: left;">wk</td>
<td style="text-align: left;">0.9.4</td>
<td style="text-align: left;">Lightweight Well-Known Geometry Parsing</td>
</tr>
<tr>
<td style="text-align: left;">wordcloud</td>
<td style="text-align: left;">2.6</td>
<td style="text-align: left;">Word Clouds</td>
</tr>
<tr>
<td style="text-align: left;">workflows</td>
<td style="text-align: left;">1.2.0</td>
<td style="text-align: left;">Modeling Workflows</td>
</tr>
<tr>
<td style="text-align: left;">workflowsets</td>
<td style="text-align: left;">1.1.0</td>
<td style="text-align: left;">Create a Collection of 'tidymodels' Workflows</td>
</tr>
<tr>
<td style="text-align: left;">writexl</td>
<td style="text-align: left;">1.5.4</td>
<td style="text-align: left;">Export Data Frames to Excel 'xlsx' Format</td>
</tr>
<tr>
<td style="text-align: left;">xfun</td>
<td style="text-align: left;">0.52</td>
<td style="text-align: left;">Supporting Functions for Packages Maintained by 'Yihui Xie'</td>
</tr>
<tr>
<td style="text-align: left;">xgboost</td>
<td style="text-align: left;">1.7.10.1</td>
<td style="text-align: left;">Extreme Gradient Boosting</td>
</tr>
<tr>
<td style="text-align: left;">xlsx</td>
<td style="text-align: left;">0.6.5</td>
<td style="text-align: left;">Read, Write, Format Excel 2007 and Excel 97/2000/XP/2003 Files</td>
</tr>
<tr>
<td style="text-align: left;">xlsxjars</td>
<td style="text-align: left;">0.6.1</td>
<td style="text-align: left;">Package required POI jars for the xlsx package</td>
</tr>
<tr>
<td style="text-align: left;">XML</td>
<td style="text-align: left;">3.99-0.18</td>
<td style="text-align: left;">Tools for Parsing and Generating XML Within R and S-Plus</td>
</tr>
<tr>
<td style="text-align: left;">xml2</td>
<td style="text-align: left;">1.3.8</td>
<td style="text-align: left;">Parse XML</td>
</tr>
<tr>
<td style="text-align: left;">xopen</td>
<td style="text-align: left;">1.0.1</td>
<td style="text-align: left;">Open System Files, 'URLs', Anything</td>
</tr>
<tr>
<td style="text-align: left;">xtable</td>
<td style="text-align: left;">1.8-4</td>
<td style="text-align: left;">Export Tables to LaTeX or HTML</td>
</tr>
<tr>
<td style="text-align: left;">xts</td>
<td style="text-align: left;">0.14.1</td>
<td style="text-align: left;">eXtensible Time Series</td>
</tr>
<tr>
<td style="text-align: left;">XVector</td>
<td style="text-align: left;">0.48.0</td>
<td style="text-align: left;">Foundation of external vector representation and manipulation in Bioconductor</td>
</tr>
<tr>
<td style="text-align: left;">yaml</td>
<td style="text-align: left;">2.3.10</td>
<td style="text-align: left;">Methods to Convert R Data to YAML and Back</td>
</tr>
<tr>
<td style="text-align: left;">yardstick</td>
<td style="text-align: left;">1.3.2</td>
<td style="text-align: left;">Tidy Characterizations of Model Performance</td>
</tr>
<tr>
<td style="text-align: left;">yulab.utils</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">Supporting Functions for Packages Maintained by 'YuLab-SMU'</td>
</tr>
<tr>
<td style="text-align: left;">zeallot</td>
<td style="text-align: left;">0.2.0</td>
<td style="text-align: left;">Multiple, Unpacking, and Destructuring Assignment</td>
</tr>
<tr>
<td style="text-align: left;">zellkonverter</td>
<td style="text-align: left;">1.18.0</td>
<td style="text-align: left;">Conversion Between scRNA-seq Objects</td>
</tr>
<tr>
<td style="text-align: left;">zigg</td>
<td style="text-align: left;">0.0.2</td>
<td style="text-align: left;">Lightweight Interfaces to the 'Ziggurat' Pseudo Random Number Generator</td>
</tr>
<tr>
<td style="text-align: left;">zip</td>
<td style="text-align: left;">2.3.2</td>
<td style="text-align: left;">Cross-Platform 'zip' Compression</td>
</tr>
<tr>
<td style="text-align: left;">zlibbioc</td>
<td style="text-align: left;">1.54.0</td>
<td style="text-align: left;">An R packaged zlib-1.2.5</td>
</tr>
<tr>
<td style="text-align: left;">zoo</td>
<td style="text-align: left;">1.8-14</td>
<td style="text-align: left;">S3 Infrastructure for Regular and Irregular Time Series (Z's Ordered Observations)</td>
</tr>
</tbody>
</table>







  
    
  
  


  <aside class="md-source-file">
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Last update">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21 13.1c-.1 0-.3.1-.4.2l-1 1 2.1 2.1 1-1c.2-.2.2-.6 0-.8l-1.3-1.3c-.1-.1-.2-.2-.4-.2m-1.9 1.8-6.1 6V23h2.1l6.1-6.1zM12.5 7v5.2l4 2.4-1 1L11 13V7zM11 21.9c-5.1-.5-9-4.8-9-9.9C2 6.5 6.5 2 12 2c5.3 0 9.6 4.1 10 9.3-.3-.1-.6-.2-1-.2s-.7.1-1 .2C19.6 7.2 16.2 4 12 4c-4.4 0-8 3.6-8 8 0 4.1 3.1 7.5 7.1 7.9l-.1.2z"></path></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date" title="February 17, 2026 18:58:19 UTC">February 17, 2026</span>
  </span>

    
    
    
    
  </aside>


  




                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"></path></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        
<div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://github.com/mcw-rcc" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M173.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6m-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3m44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9M252.8 8C114.1 8 8 113.3 8 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C436.2 457.8 504 362.9 504 252 504 113.3 391.5 8 252.8 8M105.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1m-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7m32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1m-11.4-14.7c-1.6 1-1.6 3.6 0 5.9s4.3 3.3 5.6 2.3c1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2"></path></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
      <div class="md-consent" data-md-component="consent" id="__consent" hidden>
        <div class="md-consent__overlay"></div>
        <aside class="md-consent__inner">
          <form class="md-consent__form md-grid md-typeset" name="consent">
            


  


<h4>Cookie consent</h4>
<p>We use cookies to recognize your repeated visits and preferences, as well as to measure the effectiveness of our documentation and whether users find what they're searching for. With your consent, you're helping us to make our documentation better.
</p>
<input class="md-toggle" type="checkbox" id="__settings">
<div class="md-consent__settings">
  <ul class="task-list">
    
    
      
        
  
  
    
    
  
  <li class="task-list-item">
    <label class="task-list-control">
      <input type="checkbox" name="analytics" checked>
      <span class="task-list-indicator"></span>
      Google Analytics
    </label>
  </li>

      
    
    
      
        
  
  
    
    
  
  <li class="task-list-item">
    <label class="task-list-control">
      <input type="checkbox" name="github" checked>
      <span class="task-list-indicator"></span>
      GitHub
    </label>
  </li>

      
    
    
  </ul>
</div>
<div class="md-consent__controls">
  
    
      <button class="md-button md-button--primary">Accept</button>
    
    
    
  
    
    
    
      <label class="md-button" for="__settings">Manage settings</label>
    
  
</div>
          </form>
        </aside>
      </div>
      <script>var consent=__md_get("__consent");if(consent)for(var input of document.forms.consent.elements)input.name&&(input.checked=consent[input.name]||!1);else"file:"!==location.protocol&&setTimeout((function(){document.querySelector("[data-md-component=consent]").hidden=!1}),250);var form=document.forms.consent;for(var action of["submit","reset"])form.addEventListener(action,(function(e){if(e.preventDefault(),"reset"===e.type)for(var n of document.forms.consent.elements)n.name&&(n.checked=!1);__md_set("__consent",Object.fromEntries(Array.from(new FormData(form).keys()).map((function(e){return[e,!0]})))),location.hash="",location.reload()}))</script>
    
    
      
      <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.indexes", "navigation.tabs", "navigation.tabs.sticky", "navigation.sections", "navigation.tracking", "navigation.top", "search.highlight", "search.share", "search.suggest", "content.code.copy"], "search": "../../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.f55a23d4.min.js"></script>
      
        <script src="../../_static/js/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
    
  
<script id="init-glightbox">const lightbox = GLightbox({"touchNavigation": true, "loop": false, "zoomable": true, "draggable": true, "openEffect": "zoom", "closeEffect": "zoom", "slideEffect": "slide"});
document$.subscribe(()=>{ lightbox.reload(); });
</script></body></html>