<!DOCTYPE html><html lang="en" class="no-js"><head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="A site for documenting MCW Research Computing systems and services.">
      
      
        <meta name="author" content="MCW RCC">
      
      
        <link rel="canonical" href="https://docs.rcc.mcw.edu/software/cluster-pkg-list/">
      
      
        <link rel="prev" href="../module-request/">
      
      
        <link rel="next" href="../r-pkg-list/">
      
      
        <link rel="alternate" type="application/rss+xml" title="RSS feed" href="../../feed_rss_created.xml">
        <link rel="alternate" type="application/rss+xml" title="RSS feed of updated content" href="../../feed_rss_updated.xml">
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.23">
    
    
      
        <title>Cluster Packages - MCW Research Computing</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.84d31ad4.min.css">
      
      
  
  
    
      
      
    
  
  
  <style>:root{--md-admonition-icon--tip:url('data:image/svg+xml;charset=utf-8,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20viewBox%3D%220%200%2024%2024%22%3E%3Cpath%20d%3D%22M12%202a7%207%200%200%201%207%207c0%202.38-1.19%204.47-3%205.74V17a1%201%200%200%201-1%201H9a1%201%200%200%201-1-1v-2.26C6.19%2013.47%205%2011.38%205%209a7%207%200%200%201%207-7M9%2021v-1h6v1a1%201%200%200%201-1%201h-4a1%201%200%200%201-1-1m3-17a5%205%200%200%200-5%205c0%202.05%201.23%203.81%203%204.58V16h4v-2.42c1.77-.77%203-2.53%203-4.58a5%205%200%200%200-5-5%22/%3E%3C/svg%3E');}</style>



    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../_static/css/extra.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  

<script id="__analytics">function __md_analytics(){function e(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],e("js",new Date),e("config","G-50ZTKYWP10"),document.addEventListener("DOMContentLoaded",(function(){document.forms.search&&document.forms.search.query.addEventListener("blur",(function(){this.value&&e("event","search",{search_term:this.value})}));document$.subscribe((function(){var t=document.forms.feedback;if(void 0!==t)for(var a of t.querySelectorAll("[type=submit]"))a.addEventListener("click",(function(a){a.preventDefault();var n=document.location.pathname,d=this.getAttribute("data-md-value");e("event","feedback",{page:n,data:d}),t.firstElementChild.disabled=!0;var r=t.querySelector(".md-feedback__note [data-md-value='"+d+"']");r&&(r.hidden=!1)})),t.hidden=!1})),location$.subscribe((function(t){e("config","G-50ZTKYWP10",{page_path:t.pathname})}))}));var t=document.createElement("script");t.async=!0,t.src="https://www.googletagmanager.com/gtag/js?id=G-50ZTKYWP10",document.getElementById("__analytics").insertAdjacentElement("afterEnd",t)}</script>
  
    <script>if("undefined"!=typeof __md_analytics){var consent=__md_get("__consent");consent&&consent.analytics&&__md_analytics()}</script>
  

    
    
    
  <link href="../../assets/stylesheets/glightbox.min.css" rel="stylesheet"><script src="../../assets/javascripts/glightbox.min.js"></script><style id="glightbox-style">
            html.glightbox-open { overflow: initial; height: 100%; }
            .gslide-title { margin-top: 0px; user-select: text; }
            .gslide-desc { color: #666; user-select: text; }
            .gslide-image img { background: white; }
            .gscrollbar-fixer { padding-right: 15px; }
            .gdesc-inner { font-size: 0.75rem; }
            body[data-md-color-scheme="slate"] .gdesc-inner { background: var(--md-default-bg-color); }
            body[data-md-color-scheme="slate"] .gslide-title { color: var(--md-default-fg-color); }
            body[data-md-color-scheme="slate"] .gslide-desc { color: var(--md-default-fg-color); }
        </style></head>
  
  
    <body dir="ltr">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#cluster-software-list" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
        <aside class="md-banner">
          <div class="md-banner__inner md-grid md-typeset">
            
            
  <a style="color: white; " href="https://docs.rcc.mcw.edu/pubs/">
    Do you rely on Research Computing resources? Please acknowledge that support in your publications. It really helps!
  </a>

          </div>
          
        </aside>
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow md-header--lifted" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="MCW Research Computing" class="md-header__button md-logo" aria-label="MCW Research Computing" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9 3v15h3V3zm3 2 4 13 3-1-4-13zM5 5v13h3V5zM3 19v2h18v-2z"></path></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"></path></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MCW Research Computing
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Cluster Packages
            
          </span>
        </div>
      </div>
    </div>
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"></path></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"></path></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"></path></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7s-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91s2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08"></path></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"></path></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/mcw-rcc/mcw-rcc.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"></path></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
    
      
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../.." class="md-tabs__link">
          
  
  
    
  
  Home

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../cluster/quickstart/" class="md-tabs__link">
          
  
  
    
  
  Cluster

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../storage/rcc-storage/" class="md-tabs__link">
          
  
  
    
  
  Data Storage

        </a>
      </li>
    
  

      
        
  
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../modules/" class="md-tabs__link">
          
  
  
    
  
  Software

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../secure-computing/reshpc/" class="md-tabs__link">
          
  
  
    
  
  ResHPC

        </a>
      </li>
    
  

      
        
  
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../learning/learning/" class="md-tabs__link">
          
  
  
    
  
  Learning

        </a>
      </li>
    
  

      
    </ul>
  </div>
</nav>
    
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="MCW Research Computing" class="md-nav__button md-logo" aria-label="MCW Research Computing" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9 3v15h3V3zm3 2 4 13 3-1-4-13zM5 5v13h3V5zM3 19v2h18v-2z"></path></svg>

    </a>
    MCW Research Computing
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/mcw-rcc/mcw-rcc.github.io/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"></path></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_1">
        
          
          <div class="md-nav__link md-nav__container">
            <a href="../.." class="md-nav__link ">
              
  
  
  <span class="md-ellipsis">
    Home
    
  </span>
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_1" id="__nav_1_label" tabindex="0">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_1_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_1">
            <span class="md-nav__icon md-icon"></span>
            Home
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
    
  
  
    <li class="md-nav__item">
      <a href="../../news/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Recent News
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../grants/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Grant Boilerplate
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../pubs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Publications
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../maintenance/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Maintenance
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../faq/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    FAQ
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2">
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Cluster
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Cluster
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/quickstart/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Quick Start
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/accounts/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Getting an Account
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/etiquette/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    User Etiquette
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/hardware/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Hardware Specs
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_5">
        
          
          <label class="md-nav__link" for="__nav_2_5" id="__nav_2_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Connecting
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_5">
            <span class="md-nav__icon md-icon"></span>
            Connecting
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/login/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Logging in
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/ondemand/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Open OnDemand
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/remote-access/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Remote Access
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_5_4">
        
          
          <label class="md-nav__link" for="__nav_2_5_4" id="__nav_2_5_4_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    SSH Clients
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="3" aria-labelledby="__nav_2_5_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_5_4">
            <span class="md-nav__icon md-icon"></span>
            SSH Clients
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/mobaxterm/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MobaXterm
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/putty/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    PuTTY
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/access/ssh-keys/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    SSH Keys
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2_6">
        
          
          <label class="md-nav__link" for="__nav_2_6" id="__nav_2_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Running Jobs
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_2_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2_6">
            <span class="md-nav__icon md-icon"></span>
            Running Jobs
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/running-jobs/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Submitting SLURM Jobs
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/storage-job/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Using Data in a Job
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/software-job/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Software in a Job
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/troubleshoot/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Troubleshoot Jobs
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cluster/jobs/xdmod/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Job Metrics
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3">
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Data Storage
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Data Storage
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/rcc-storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Storage Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/paid-storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Paid Additional Storage
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/ref-data/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Reference Data
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/mcw-storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Additional Storage Options
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_5">
        
          
          <label class="md-nav__link" for="__nav_3_5" id="__nav_3_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Data Transfer
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_5">
            <span class="md-nav__icon md-icon"></span>
            Data Transfer
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/file-transfer/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Transfer to Cluster
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/globus/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Globus
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/rclone/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Rclone
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3_6">
        
          
          <label class="md-nav__link" for="__nav_3_6" id="__nav_3_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Manage &amp; Share
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_3_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3_6">
            <span class="md-nav__icon md-icon"></span>
            Manage &amp; Share
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/data-access/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Mounting Drives
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/data-sharing/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Sharing Data
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/file-cleanup/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Cleanup &amp; Archiving
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/file-recovery/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Recovery
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../storage/file-permissions/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Permission Changes
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
      
        
        
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" checked>
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Software
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Software
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../modules/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Using Modules
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../module-request/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Requesting Software
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    
    
      
        
      
        
      
    
    
    
      
      
        
          
          
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_3" checked>
        
          
          <label class="md-nav__link" for="__nav_4_3" id="__nav_4_3_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Installed Software
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_3_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4_3">
            <span class="md-nav__icon md-icon"></span>
            Installed Software
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Cluster Packages
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Cluster Packages
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#categories" class="md-nav__link">
    <span class="md-ellipsis">
      Categories
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Categories">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#biology" class="md-nav__link">
    <span class="md-ellipsis">
      biology
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chemistry" class="md-nav__link">
    <span class="md-ellipsis">
      chemistry
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#devel" class="md-nav__link">
    <span class="md-ellipsis">
      devel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#math" class="md-nav__link">
    <span class="md-ellipsis">
      math
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#system" class="md-nav__link">
    <span class="md-ellipsis">
      system
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#viz" class="md-nav__link">
    <span class="md-ellipsis">
      viz
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../r-pkg-list/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    R Packages
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
          
          
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_4">
        
          
          <label class="md-nav__link" for="__nav_4_4" id="__nav_4_4_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    Guides
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_4">
            <span class="md-nav__icon md-icon"></span>
            Guides
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../alphafold/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    AlphaFold
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../ansys/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    ANSYS
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../conda/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Conda
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../containers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Containers
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../git/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Git and GitHub
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../advanced_git/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Git and GitHub (advanced)
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../jupyter/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Jupyter
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../lsdyna/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    LS-DYNA
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../matlab/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    MATLAB Parallel Server
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../namd/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    NAMD
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../nextflow/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Nextflow
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../python/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Python
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../pytorch/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    PyTorch
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../R/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    R
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../schrodinger/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Schr√∂dinger
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../tensorflow/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    TensorFlow
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5">
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    ResHPC
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            ResHPC
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/reshpc/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/project-account/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Accounts &amp; Projects
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/project-storage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Storage
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/mfa-login/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Logging in
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/mfa-data-transfer/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Data Transfer
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/workflow/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Run a Job
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../secure-computing/software/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Software
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
        
      
        
      
        
      
    
    
    
      
      
        
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6">
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Learning
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            Learning
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../learning/learning/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../learning/hpc_clusters/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    HPC Clusters Tutorials
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../learning/slurm/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    SLURM Guides
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../learning/programming/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Programming Guides
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#categories" class="md-nav__link">
    <span class="md-ellipsis">
      Categories
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Categories">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#biology" class="md-nav__link">
    <span class="md-ellipsis">
      biology
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#chemistry" class="md-nav__link">
    <span class="md-ellipsis">
      chemistry
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#devel" class="md-nav__link">
    <span class="md-ellipsis">
      devel
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#math" class="md-nav__link">
    <span class="md-ellipsis">
      math
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#system" class="md-nav__link">
    <span class="md-ellipsis">
      system
    </span>
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#viz" class="md-nav__link">
    <span class="md-ellipsis">
      viz
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="cluster-software-list">Cluster software list<a class="headerlink" href="#cluster-software-list" title="Permanent link">#</a></h1>
<p>Last updated: <em><span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date" title="February 17, 2026 18:58:19 UTC">February 17, 2026</span></em></p>
<!-- 
// Copyright 2014-2022 Stanford Research Computing Center
//
// The following code is a derivative work of https://raw.githubusercontent.com/stanford-rc/www.sherlock.stanford.edu/main/src/docs/software/list.md, which is licensed GPLv3. This code therefore is also licensed under the terms GPLv3.
-->
<h2 id="categories">Categories<a class="headerlink" href="#categories" title="Permanent link">#</a></h2>
<p><em>We currently provide 444 software modules, in 6 categories, covering 59 fields of science:</em></p>
<!-- markdownlint-disable MD033 -->
<ul>
<li><a href="#biology"><code>biology</code></a> <small>
    computational biology, cryo-em, genomics, imaging, neuroimaging, population genetics
  </small></li>
</ul>
<!-- markdownlint-disable MD033 -->
<ul>
<li><a href="#chemistry"><code>chemistry</code></a> <small>
    computational chemistry, electrostatics, molecular dynamics, molecular modelling, molecular visualization, tools
  </small></li>
</ul>
<!-- markdownlint-disable MD033 -->
<ul>
<li><a href="#devel"><code>devel</code></a> <small>
    compiler, data, debugger, language, lib, mpi, networking, profiling, text editor
  </small></li>
</ul>
<!-- markdownlint-disable MD033 -->
<ul>
<li><a href="#math"><code>math</code></a> <small>
    analytics, computational fluid dynamics, data analytics, deep learning, finite element analysis, gis, lib, machine learning, numerical analysis, numerical library, optimization, simulation, statistics
  </small></li>
</ul>
<!-- markdownlint-disable MD033 -->
<ul>
<li><a href="#system"><code>system</code></a> <small>
    benchmark, build, checkpointing, cloud interface, compression, containers, data management, database, document processing, file transfer, ide, lib, library, media, package manager, pipeline, profiling, remote display, scm, terminal, tools, utilities
  </small></li>
</ul>
<!-- markdownlint-disable MD033 -->
<ul>
<li><a href="#viz"><code>viz</code></a> <small>
    graphics, graphs, imaging
  </small></li>
</ul>
<h3 id="biology"><strong>biology</strong><a class="headerlink" href="#biology" title="Permanent link">#</a></h3>
<!-- markdownlint-disable MD045 MD056 MD058 -->
<table>
<thead>
<tr>
<th style="text-align: left;">Field</th>
<th style="text-align: left;">Module Name<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:110px"></a></th>
<th style="text-align: left;">Version(s)<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:90px"></a></th>
<th style="text-align: left;">URL</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>alphafold</code></td>
<td style="text-align: left;"><code>2.3.2</code><br><code>3.0.0</code><br><code>3.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/google-deepmind/alphafold3">Website</a></td>
<td style="text-align: left;">AI system developed by DeepMind that predicts a protein‚Äôs 3D structure from its amino acid sequence.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>alphamissense</code></td>
<td style="text-align: left;"><code>3.18.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/google-deepmind/alphamissense">Website</a></td>
<td style="text-align: left;">AI system developed by DeepMind that predicts a protein‚Äôs 3D structure from its amino acid sequence.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>alphapulldown</code></td>
<td style="text-align: left;"><code>1.0.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/KosinskiLab/AlphaPulldown/tree/main">Website</a></td>
<td style="text-align: left;">AlphaPulldown is a Python package that streamlines protein-protein interaction screens and high-throughput modelling of higher-order oligomers using AlphaFold-Multimer.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>bcl</code></td>
<td style="text-align: left;"><code>3.4.0</code><br></td>
<td style="text-align: left;"><a href="http://www.meilerlab.org/index.php/bclcommons/show/b_apps_id/1">Website</a></td>
<td style="text-align: left;">The Bio Chemical Library (BCL) is a software package that provides unique tools for biological research, such as protein structure determination from sparse experimental data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>boltz</code></td>
<td style="text-align: left;"><code>0.4.1</code><br><code>2.0.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/jwohlwend/boltz">Website</a></td>
<td style="text-align: left;">Boltz-1 is the state-of-the-art open-source model to predict biomolecular structures containing combinations of proteins, RNA, DNA, and other molecules.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>calibur</code></td>
<td style="text-align: left;"><code>20120117</code><br></td>
<td style="text-align: left;"><a href="https://sourceforge.net/projects/calibur/">Website</a></td>
<td style="text-align: left;">Efficient tool for finding structural candidate decoys generated from Ab Initio protein structure prediction methods.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>colabfold</code></td>
<td style="text-align: left;"><code>2.3.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/deepmind/alphafold">Website</a></td>
<td style="text-align: left;">AI system developed by DeepMind that predicts a protein‚Äôs 3D structure from its amino acid sequence.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>cookhla</code></td>
<td style="text-align: left;"><code>1.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/WansonChoi/CookHLA">Website</a></td>
<td style="text-align: left;">CookHLA is an accurate and efficient HLA imputation method.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>dssp</code></td>
<td style="text-align: left;"><code>4.4.0</code><br><code>4.4.7</code><br></td>
<td style="text-align: left;"><a href="https://pdb-redo.eu/dssp">Website</a></td>
<td style="text-align: left;">DSSP is an algorithm originally designed by Wolfgang Kabsch and Chris Sander to standardise secondary structure assigment based on atomic coordinates.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>ensembler</code></td>
<td style="text-align: left;"><code>1.0.5</code><br></td>
<td style="text-align: left;"><a href="http://ensembler.readthedocs.org/">Website</a></td>
<td style="text-align: left;">Software pipeline for automating omics-scale protein modeling and simulation setup.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>foldx</code></td>
<td style="text-align: left;"><code>4.0</code><br><code>5.0</code><br><code>5.1</code><br></td>
<td style="text-align: left;"><a href="https://foldxsuite.crg.eu/">Website</a></td>
<td style="text-align: left;">FoldX provides a fast and quantitative estimation of the importance of the interactions contributing to the stability of proteins and protein complexes.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>instanovo</code></td>
<td style="text-align: left;"><code>1.1.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/instadeepai/InstaNovo">Website</a></td>
<td style="text-align: left;">InstaNovo is a transformer neural network with the ability to translate fragment ion peaks into the sequence of amino acids that make up the studied peptides.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>interproscan</code></td>
<td style="text-align: left;"><code>5.39-77.0</code><br><code>5.50-84.0</code><br></td>
<td style="text-align: left;"><a href="https://www.ebi.ac.uk/interpro/">Website</a></td>
<td style="text-align: left;">Functional analysis of proteins by classifying them into families and predicting domains and important sites.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>modeller</code></td>
<td style="text-align: left;"><code>10.0</code><br><code>10.5</code><br></td>
<td style="text-align: left;"><a href="https://salilab.org/modeller/">Website</a></td>
<td style="text-align: left;">MODELLER is used for homology or comparative modeling of protein three-dimensional structures.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>msmbuilder</code></td>
<td style="text-align: left;"><code>3.8.0</code><br></td>
<td style="text-align: left;"><a href="http://msmbuilder.org/3.8.0/">Website</a></td>
<td style="text-align: left;">MSMBuilder is an application and python library. It builds statistical models for high-dimensional time-series. The particular focus of the package is on the analysis of atomistic simulations of biomolecular dynamics such as protein folding and conformational change.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>pdbfixer</code></td>
<td style="text-align: left;"><code>1.7.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/openmm/pdbfixer">Website</a></td>
<td style="text-align: left;">The PDBFixer is an easy to use application for fixing problems in Protein Data Bank files</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>phenix</code></td>
<td style="text-align: left;"><code>1.21.2</code><br></td>
<td style="text-align: left;"><a href="https://phenix-online.org/version_docs/1.21.2-5419/index.html">Website</a></td>
<td style="text-align: left;">Python-based Hierarchical ENvironment for Integrated Xtallography.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>phylip</code></td>
<td style="text-align: left;"><code>3.697</code><br></td>
<td style="text-align: left;"><a href="https://evolution.genetics.washington.edu/phylip.html">Website</a></td>
<td style="text-align: left;">PHYLIP (the PHYLogeny Inference Package) is a package of programs for inferring phylogenies (evolutionary trees).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>rfdiffusion</code></td>
<td style="text-align: left;"><code>1.1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/RosettaCommons/RFdiffusion">Website</a></td>
<td style="text-align: left;">RFdiffusion is an open source method for structure generation, with or without conditional information (a motif, target etc).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>rosetta</code></td>
<td style="text-align: left;"><code>3.1</code><br><code>3.13</code><br><code>3.14</code><br><code>3.15</code><br></td>
<td style="text-align: left;"><a href="https://www.rosettacommons.org/software">Website</a></td>
<td style="text-align: left;">The Rosetta software suite includes algorithms for computational modeling and analysis of protein structures.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>rosettafold2</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/uw-ipd/RoseTTAFold2/">Website</a></td>
<td style="text-align: left;">Software to predict structures.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>simvascular</code></td>
<td style="text-align: left;"><code>2025</code><br></td>
<td style="text-align: left;"><a href="https://simvascular.github.io">Website</a></td>
<td style="text-align: left;">Interactive application used to create patient-specific geometric models of human vasculature from 3D medical imaging data acquired for a specific anatomical region.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>tsl</code></td>
<td style="text-align: left;"><code>1.21</code><br></td>
<td style="text-align: left;"><a href="http://www.twosamplelogo.org/">Website</a></td>
<td style="text-align: left;">Two Sample Logo is a web-based application that calculates and visualizes differences between two sets of aligned samples of amino acids or nucleotides.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>ucbshift</code></td>
<td style="text-align: left;"><code>2.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/THGLab/CSpred/tree/SideChain?tab=readme-ov-file">Website</a></td>
<td style="text-align: left;">UCBShift is a program for predicting chemical shifts for backbone and side chain atoms of a protein in solution.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational biology</strong></td>
<td style="text-align: left;"><code>weblogo</code></td>
<td style="text-align: left;"><code>3.9.0</code><br></td>
<td style="text-align: left;"><a href="https://weblogo.threeplusone.com/">Website</a></td>
<td style="text-align: left;">Application designed to make the generation of sequence logos as easy and painless as possible.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>cryo-em</strong></td>
<td style="text-align: left;"><code>relion</code></td>
<td style="text-align: left;"><code>3.1.3</code><br><code>4.0.2</code><br></td>
<td style="text-align: left;"><a href="https://relion.readthedocs.io/">Website</a></td>
<td style="text-align: left;">RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a software package that employs an empirical Bayesian approach for electron cryo-microscopy (cryo-EM) structure determination.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>agat</code></td>
<td style="text-align: left;"><code>1.4.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/NBISweden/AGAT">Website</a></td>
<td style="text-align: left;">Suite of tools to handle gene annotations in any GTF/GFF format.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>alevinfry</code></td>
<td style="text-align: left;"><code>0.11.1</code><br></td>
<td style="text-align: left;"><a href="https://alevin-fry.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">alevin-fry is a suite of tools for the rapid, accurate and memory-frugal processing single-cell and single-nucleus sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>alphagenome</code></td>
<td style="text-align: left;"><code>0.3.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/google-deepmind/alphagenome">Website</a></td>
<td style="text-align: left;">The AlphaGenome API provides access to AlphaGenome, Google DeepMind‚Äôs unifying model for deciphering the regulatory code within DNA sequences..</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>ampliconarchitect</code></td>
<td style="text-align: left;"><code>1.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/virajbdeshpande/AmpliconArchitect">Website</a></td>
<td style="text-align: left;">Identify one or more connected, focally amplified genomic regions to elucidate the architecture of focal amplifications such as ecDNA.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>ampliconsuite</code></td>
<td style="text-align: left;"><code>1.3.5</code><br></td>
<td style="text-align: left;"><a href="https://github.com/AmpliconSuite/AmpliconSuite-pipeline">Website</a></td>
<td style="text-align: left;">A quickstart tool for AmpliconArchitect. Performs all preliminary steps required prior to running AmpliconArchitect. Previously called PrepareAA.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>analyze_motifs</code></td>
<td style="text-align: left;"><code>0.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/YeoLab/clip_analysis_legacy">Website</a></td>
<td style="text-align: left;">Runs HOMER and kmer enrichment aspects from clip_analysis_legacy to generate useful motif analyses.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>annotsv</code></td>
<td style="text-align: left;"><code>3.4.4</code><br></td>
<td style="text-align: left;"><a href="https://lbgi.fr/AnnotSV/">Website</a></td>
<td style="text-align: left;">AnnotSV is a standalone program designed for annotating and ranking Structural Variations.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>annovar</code></td>
<td style="text-align: left;"><code>042018</code><br><code>25616</code><br><code>29711</code><br></td>
<td style="text-align: left;"><a href="https://annovar.openbioinformatics.org/en/latest/">Website</a></td>
<td style="text-align: left;">ANNOVAR is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>antismash</code></td>
<td style="text-align: left;"><code>7.1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/antismash/antismash">Website</a></td>
<td style="text-align: left;">antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genomes.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>anvio</code></td>
<td style="text-align: left;"><code>8</code><br></td>
<td style="text-align: left;"><a href="https://anvio.org/">Website</a></td>
<td style="text-align: left;">Anvi‚Äôo is an open-source, community-driven analysis and visualization platform for microbial ‚Äòomics.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>aracne-ap</code></td>
<td style="text-align: left;"><code>072018</code><br></td>
<td style="text-align: left;"><a href="https://github.com/califano-lab/ARACNe-AP">Website</a></td>
<td style="text-align: left;">ARACNe-AP (Algorithm for the Reconstruction of Accurate Cellular Networks with Adaptive Partitioning)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bam-readcount</code></td>
<td style="text-align: left;"><code>0.8.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/genome/bam-readcount">Website</a></td>
<td style="text-align: left;">Utility that runs on a BAM or CRAM file and generates low-level information about sequencing data at specific nucleotide positions.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bam2fastx</code></td>
<td style="text-align: left;"><code>3.0.0</code><br></td>
<td style="text-align: left;"><a href="missing">Website</a></td>
<td style="text-align: left;">Conversion of PacBio BAM files into gzipped fasta and fastq files, including splitting of barcoded data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bamtools</code></td>
<td style="text-align: left;"><code>2.4.1</code><br><code>2.5.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/pezmaster31/bamtools">Website</a></td>
<td style="text-align: left;">C++ API and a command-line toolkit for reading, writing, and manipulating BAM (genome alignment) files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>basemount</code></td>
<td style="text-align: left;"><code>0.15.96</code><br></td>
<td style="text-align: left;"><a href="https://basemount.basespace.illumina.com/">Website</a></td>
<td style="text-align: left;">BaseMount is a tool to mount your BaseSpace Sequence Hub data as a Linux file system.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>basespace</code></td>
<td style="text-align: left;"><code>1.3.0</code><br></td>
<td style="text-align: left;"><a href="https://developer.basespace.illumina.com/docs/content/documentation/cli/cli-overview">Website</a></td>
<td style="text-align: left;">The BaseSpace Sequence Hub CLI supports scripting and programmatic access to BaseSpace Sequence Hub for automation, bulk operations, and other routine functions.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bbmap</code></td>
<td style="text-align: left;"><code>38.82</code><br><code>39.01</code><br><code>39.11</code><br></td>
<td style="text-align: left;"><a href="https://sourceforge.net/projects/bbmap/">Website</a></td>
<td style="text-align: left;">Short read aligner for DNA and RNA-seq data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bcftools</code></td>
<td style="text-align: left;"><code>1.11</code><br><code>1.15</code><br><code>1.16</code><br><code>1.19</code><br><code>1.21</code><br><code>1.22</code><br></td>
<td style="text-align: left;"><a href="http://samtools.github.io/bcftools/howtos/index.html">Website</a></td>
<td style="text-align: left;">BCFtools is a program for variant calling and manipulating files in the Variant Call Format (VCF) and its binary counterpart BCF.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bcftools-mocha</code></td>
<td style="text-align: left;"><code>1.15</code><br></td>
<td style="text-align: left;"><a href="http://samtools.github.io/bcftools/howtos/index.html">Website</a></td>
<td style="text-align: left;">BCFtools is a program for variant calling and manipulating files in the Variant Call Format (VCF) and its binary counterpart BCF.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bcl-convert</code></td>
<td style="text-align: left;"><code>4.4.6</code><br></td>
<td style="text-align: left;"><a href="https://help.dragen.illumina.com/product-guide/dragen-v4.4/bcl-conversion">Website</a></td>
<td style="text-align: left;">The Illumina BCL Convert is a standalone local software app that converts the Binary Base Call (BCL) files produced by Illumina‚Ñ¢ sequencing systems to FASTQ files..</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bcl2fastq2</code></td>
<td style="text-align: left;"><code>2.2</code><br></td>
<td style="text-align: left;"><a href="https://support.illumina.com/downloads/bcl2fastq-conversion-software-v2-20.html">Website</a></td>
<td style="text-align: left;">The Illumina bcl2fastq2 Conversion Software v2.20 demultiplexes sequencing data and converts base call (BCL) files into FASTQ files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bedops</code></td>
<td style="text-align: left;"><code>2.4.39</code><br><code>2.4.41</code><br></td>
<td style="text-align: left;"><a href="https://bedops.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">Open-source command-line toolkit that performs highly efficient and scalable Boolean and other set operations, statistical calculations, archiving, conversion and other management of genomic data of arbitrary scale.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bedtools2</code></td>
<td style="text-align: left;"><code>2.25.0</code><br><code>2.30.0</code><br></td>
<td style="text-align: left;"><a href="https://bedtools.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">The bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bgen</code></td>
<td style="text-align: left;"><code>1.1.7</code><br></td>
<td style="text-align: left;"><a href="https://www.well.ox.ac.uk/~gav/bgen_format/software.html">Website</a></td>
<td style="text-align: left;">Reference implementation of the BGEN format, written in C++.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bicseq2-norm</code></td>
<td style="text-align: left;"><code>0.2.4</code><br></td>
<td style="text-align: left;"><a href="http://compbio.med.harvard.edu/BIC-seq/">Website</a></td>
<td style="text-align: left;">BICseq2-norm is for normalizing potential biases in the sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bicseq2-seg</code></td>
<td style="text-align: left;"><code>0.7.2</code><br></td>
<td style="text-align: left;"><a href="http://compbio.med.harvard.edu/BIC-seq/">Website</a></td>
<td style="text-align: left;">BICseq2-seg is for detecting CNVs based on the normalized data given by BICseq2-norm.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bioawk</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/lh3/bioawk">Website</a></td>
<td style="text-align: left;">BWK awk modified for biological data</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>biobambam2</code></td>
<td style="text-align: left;"><code>2.0.87</code><br></td>
<td style="text-align: left;"><a href="https://github.com/gt1/biobambam2">Website</a></td>
<td style="text-align: left;">This package contains some tools for processing BAM files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>biolibc</code></td>
<td style="text-align: left;"><code>0.2.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/auerlab/biolibc">Website</a></td>
<td style="text-align: left;">Biolibc is a collection of high-quality bricks that can be used to build efficient, robust software applications to replace disposable scripts. Using biolibc, you can easily develop permanent solutions that are easy to use and install, with near-optimal performance, so that no one ever need reinvent that particular wheel. Biolibc also facilitates development of more complex applications by providing many commonly used building blocks, thus releasing you from low-level coding.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bior</code></td>
<td style="text-align: left;"><code>5.0.0</code><br></td>
<td style="text-align: left;"><a href="https://bioinformaticstools.mayo.edu/research/bior/">Website</a></td>
<td style="text-align: left;">The Biological Reference Repository (BioR) catalog format is a flexible, readable, indexable, and schema-free format for storing and rapidly accessing arbitrary structured data such as genomic features, diseases, conditions, genetic tests, and drugs.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bismark</code></td>
<td style="text-align: left;"><code>0.16.3</code><br></td>
<td style="text-align: left;"><a href="https://www.bioinformatics.babraham.ac.uk/projects/bismark/">Website</a></td>
<td style="text-align: left;">A tool to map bisulfite converted sequence reads and determine cytosine methylation states.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>blast</code></td>
<td style="text-align: left;"><code>2.9.0</code><br><code>2.12.0</code><br></td>
<td style="text-align: left;"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">Website</a></td>
<td style="text-align: left;">The Basic Local Alignment Search Tool (BLAST) finds regions of local similarity between sequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bowtie</code></td>
<td style="text-align: left;"><code>2.3.4.3</code><br><code>2.5.0</code><br><code>2.5.4</code><br></td>
<td style="text-align: left;"><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">Website</a></td>
<td style="text-align: left;">Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bracken</code></td>
<td style="text-align: left;"><code>2.6.0</code><br><code>2.8.0</code><br></td>
<td style="text-align: left;"><a href="https://ccb.jhu.edu/software/bracken/">Website</a></td>
<td style="text-align: left;">Bracken (Bayesian Reestimation of Abundance with KrakEN) is a highly accurate statistical method that computes the abundance of species in DNA sequences from a metagenomics sample.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>bwa</code></td>
<td style="text-align: left;"><code>0.7.17</code><br></td>
<td style="text-align: left;"><a href="http://bio-bwa.sourceforge.net/">Website</a></td>
<td style="text-align: left;">BWA is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cadd</code></td>
<td style="text-align: left;"><code>1.6</code><br></td>
<td style="text-align: left;"><a href="https://cadd.gs.washington.edu/">Website</a></td>
<td style="text-align: left;">CADD is a tool for scoring the deleteriousness of single nucleotide variants as well as insertion/deletions variants in the human genome.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>canu</code></td>
<td style="text-align: left;"><code>1.8</code><br></td>
<td style="text-align: left;"><a href="https://github.com/marbl/canu">Website</a></td>
<td style="text-align: left;">A single molecule sequence assembler for genomes large and small.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cava</code></td>
<td style="text-align: left;"><code>1.2.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/RahmanTeam/CAVA">Website</a></td>
<td style="text-align: left;">CAVA (Clinical Annotation of VAriants) is a lightweight, fast, flexible and easy-to-use Next Generation Sequencing (NGS) variant annotation tool.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cd-hit</code></td>
<td style="text-align: left;"><code>4.8.1</code><br></td>
<td style="text-align: left;"><a href="http://weizhong-lab.ucsd.edu/cd-hit/">Website</a></td>
<td style="text-align: left;">Program for clustering and comparing protein or nucleotide sequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cellranger</code></td>
<td style="text-align: left;"><code>3.0.0</code><br><code>4.0.0</code><br><code>5.0.1</code><br><code>6.0.0</code><br><code>7.0.0</code><br><code>8.0.0</code><br><code>9.0.0</code><br></td>
<td style="text-align: left;"><a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger">Website</a></td>
<td style="text-align: left;">Set of analysis pipelines that process Chromium single cell data to align reads, generate feature-barcode matrices, perform clustering and other secondary analysis</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cellranger-arc</code></td>
<td style="text-align: left;"><code>1.0.1</code><br><code>2.0.1</code><br><code>2.0.2</code><br></td>
<td style="text-align: left;"><a href="https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc">Website</a></td>
<td style="text-align: left;">Cell Ranger ARC is a set of analysis pipelines that process Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a variety of analyses pertaining to gene expression (GEX), chromatin accessibility, and their linkage.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cellranger-atac</code></td>
<td style="text-align: left;"><code>2.1.0</code><br></td>
<td style="text-align: left;"><a href="https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc">Website</a></td>
<td style="text-align: left;">Cell Ranger ARC is a set of analysis pipelines that process Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a variety of analyses pertaining to gene expression (GEX), chromatin accessibility, and their linkage.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cellsnp-lite</code></td>
<td style="text-align: left;"><code>1.2.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/single-cell-genetics/cellsnp-lite">Website</a></td>
<td style="text-align: left;">cellsnp-lite was initially designed to pileup the expressed alleles in single-cell or bulk RNA-seq data, which can be directly used for donor deconvolution in multiplexed single-cell RNA-seq data, particularly with vireo, which assigns cells to donors and detects doublets, even without genotyping reference.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>centrifuge</code></td>
<td style="text-align: left;"><code>1.0.3</code><br></td>
<td style="text-align: left;"><a href="https://ccb.jhu.edu/software/centrifuge/">Website</a></td>
<td style="text-align: left;">Rapid and memory-efficient system for the classification of DNA sequences from microbial samples, with better sensitivity than and comparable accuracy to other leading systems.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>circexplorer2</code></td>
<td style="text-align: left;"><code>2.3.6</code><br></td>
<td style="text-align: left;"><a href="https://circexplorer2.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">CIRCexplorer2 is a comprehensive and integrative circular RNA analysis toolset.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>circlefinder</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/pk7zuva/Circle_finder">Website</a></td>
<td style="text-align: left;">A method to identify Circular DNA (Micro DNA) from pair-end high-throughput sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>circlemap</code></td>
<td style="text-align: left;"><code>1.1.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/iprada/Circle-Map">Website</a></td>
<td style="text-align: left;">Circle-Map takes as input an alignment of reads to a reference genome (e.g. a BWA-MEM generated BAM file) and like other methods, it will use those alignments to detect cases were the read has been split into two segments (e.g. split reads) to detect genomic rearrangements supporting a circular DNA structure.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cite-seq-count</code></td>
<td style="text-align: left;"><code>1.4.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/Hoohm/CITE-seq-Count">Website</a></td>
<td style="text-align: left;">A tool that allows to get UMI counts from a single cell protein assay.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>clara-parabricks</code></td>
<td style="text-align: left;"><code>4.0.1-1</code><br><code>4.4.0</code><br><code>4.5.0</code><br></td>
<td style="text-align: left;"><a href="https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara/containers/clara-parabricks">Website</a></td>
<td style="text-align: left;">Nvidia Clara Parabricks is an accelerated compute framework that supports applications across the genomics industry, primarily supporting analytical workflows for DNA, RNA, and somatic mutation detection applications.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>clipper</code></td>
<td style="text-align: left;"><code>2.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/YeoLab/clipper/wiki/CLIPper-Home">Website</a></td>
<td style="text-align: left;">CLIPper is a tool to define peaks in your CLIP-seq dataset.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>clustal-omega</code></td>
<td style="text-align: left;"><code>1.2.4</code><br></td>
<td style="text-align: left;"><a href="https://www.ebi.ac.uk/Tools/msa/clustalo/">Website</a></td>
<td style="text-align: left;">Clustal Omega is a new multiple sequence alignment program that uses seeded guide trees and HMM profile-profile techniques to generate alignments between three or more sequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cnvkit</code></td>
<td style="text-align: left;"><code>0.9.12</code><br></td>
<td style="text-align: left;"><a href="https://cnvkit.readthedocs.io/en/stable/">Website</a></td>
<td style="text-align: left;">CNVkit is a Python library and command-line software toolkit to infer and visualize copy number from high-throughput DNA sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cobra</code></td>
<td style="text-align: left;"><code>2.0</code><br></td>
<td style="text-align: left;"><a href="https://cfce-cobra.readthedocs.io/en/latest/index.html">Website</a></td>
<td style="text-align: left;">Containerized Bioinformatics workflow for Reproducible ChIP,ATAC-seq Analysis</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>compass</code></td>
<td style="text-align: left;"><code>0.9.10.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/YosefLab/Compass">Website</a></td>
<td style="text-align: left;">In-Silico Modeling of Metabolic Heterogeneity using Single-Cell Transcriptomes</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>concoct</code></td>
<td style="text-align: left;"><code>1.1.0</code><br></td>
<td style="text-align: left;"><a href="https://concoct.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">CONCOCT ‚Äúbins‚Äù metagenomic contigs. Metagenomic binning is the process of clustering sequences into clusters corresponding to operational taxonomic units of some level.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>crossmap</code></td>
<td style="text-align: left;"><code>0.6.6</code><br></td>
<td style="text-align: left;"><a href="https://crossmap.sourceforge.net/">Website</a></td>
<td style="text-align: left;">CrossMap is a program for genome coordinates conversion between different assemblies (such as hg18 (NCBI36) &lt;=&gt; hg19 (GRCh37)). It supports commonly used file formats including BAM, CRAM, SAM, Wiggle, BigWig, BED, GFF, GTF, MAF VCF, and gVCF.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cufflinks</code></td>
<td style="text-align: left;"><code>2.2.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/cole-trapnell-lab/cufflinks">Website</a></td>
<td style="text-align: left;">Cufflinks assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cutadapt</code></td>
<td style="text-align: left;"><code>3.2</code><br><code>3.4</code><br><code>4.0</code><br></td>
<td style="text-align: left;"><a href="https://cutadapt.readthedocs.io/en/stable/">Website</a></td>
<td style="text-align: left;">Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>cytoscape</code></td>
<td style="text-align: left;"><code>3.10.3</code><br></td>
<td style="text-align: left;"><a href="https://cytoscape.org/">Website</a></td>
<td style="text-align: left;">Cytoscape is an open source software platform for visualizing complex networks and integrating these with any type of attribute data. With Cytoscape, you can load molecular and genetic interaction data sets in many standards formats, project and integrate global datasets and functional annotations, establish powerful visual mappings across these data, perform advanced analysis and modeling and visualize and analyze human-curated pathway datasets.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dapars</code></td>
<td style="text-align: left;"><code>1.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/ZhengXia/dapars">Website</a></td>
<td style="text-align: left;">Dynamitic analysis of Alternative PolyAdenylation from RNA-seq</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dcc</code></td>
<td style="text-align: left;"><code>0.8.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/dieterich-lab/DCC">Website</a></td>
<td style="text-align: left;">DCC is a python package intended to detect and quantify circRNAs with high specificity. DCC works with the STAR (Dobin et al., 2013) chimeric.out.junction files which contains chimerically aligned reads including circRNA junction spanning reads.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dcca</code></td>
<td style="text-align: left;"><code>1.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/cmzuo11/DCCA">Website</a></td>
<td style="text-align: left;">Deep cross-omics cycle attention model for joint analysis of single-cell multi-omics data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dchic</code></td>
<td style="text-align: left;"><code>2.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/ay-lab/dcHiC">Website</a></td>
<td style="text-align: left;">dcHiC is a tool for differential compartment analysis of Hi-C datasets</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>deeptools</code></td>
<td style="text-align: left;"><code>3.5.0</code><br><code>3.5.1</code><br><code>3.5.6</code><br></td>
<td style="text-align: left;"><a href="https://deeptools.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">Suite of python tools particularly developed for the efficient analysis of high-throughput sequencing data, such as ChIPseq, RNAseq or MNaseseq.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>depict</code></td>
<td style="text-align: left;"><code>1.194</code><br></td>
<td style="text-align: left;"><a href="https://github.com/perslab/depict?tab=readme-ov-file">Website</a></td>
<td style="text-align: left;">an integrative tool that employs predicted gene functions to systematically prioritize the most likely causal genes at associated loci, highlight enriched pathways and identify tissues/cell types where genes from associated loci are highly expressed.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>diamond</code></td>
<td style="text-align: left;"><code>0.9.14</code><br></td>
<td style="text-align: left;"><a href="https://github.com/bbuchfink/diamond">Website</a></td>
<td style="text-align: left;">DIAMOND is a sequence aligner for protein and translated DNA searches, designed for high performance analysis of big sequence data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>diting</code></td>
<td style="text-align: left;"><code>0.8</code><br></td>
<td style="text-align: left;"><a href="https://github.com/xuechunxu/DiTing/">Website</a></td>
<td style="text-align: left;">DiTing is designed to determine the relative abundance of metabolic and biogeochemical functional pathways in a set of given metagenomic/metatranscriptomic data. The input is expected to be a folder containing a group of paired-end clean reads. These reads will be assembled, annotated, and parsed for producing a table of relative abundance of elemental/biogeochemical cycling pathways (e.g., Nitrogen, Carbon, Sulfur) in each sample. Sketch maps and heatmaps will also be produced accordingly for comparing biogeochemical functions visually.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dnanexus</code></td>
<td style="text-align: left;"><code>0.6.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/dnanexus/dxda">Website</a></td>
<td style="text-align: left;">DNAnexus download agent.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dorado</code></td>
<td style="text-align: left;"><code>0.9.6</code><br></td>
<td style="text-align: left;"><a href="https://dorado-docs.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">Dorado is a high-performance, easy-to-use, open source basecaller for Oxford Nanopore reads.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dram</code></td>
<td style="text-align: left;"><code>1.5.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/shafferm/DRAM">Website</a></td>
<td style="text-align: left;">DRAM (Distilled and Refined Annotation of Metabolism) is a tool for annotating metagenomic assembled genomes and VirSorter identified viral contigs.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dropest</code></td>
<td style="text-align: left;"><code>0.8.5</code><br></td>
<td style="text-align: left;"><a href="https://github.com/kharchenkolab/dropEst">Website</a></td>
<td style="text-align: left;">Pipeline for estimating molecular count matrices for droplet-based single-cell RNA-seq measurements.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>dx-toolkit</code></td>
<td style="text-align: left;"><code>0.392.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/dnanexus/dx-toolkit">Website</a></td>
<td style="text-align: left;">DNAnexus Platform SDK</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>ea-utils</code></td>
<td style="text-align: left;"><code>1.04</code><br></td>
<td style="text-align: left;"><a href="https://expressionanalysis.github.io/ea-utils/">Website</a></td>
<td style="text-align: left;">Command-line tools for processing biological sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>eclipanalyzer</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/jkkbuddika/eCLIP-Data-Analyzer">Website</a></td>
<td style="text-align: left;">This repository contains a series of python based modules to automate the analysis of sequencing data generated from eCLIP (enhanced UV crosslinking and immunoprecipitation) experiments.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>eclipidrmergepeaks</code></td>
<td style="text-align: left;"><code>0.1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/YeoLab/merge_peaks">Website</a></td>
<td style="text-align: left;">CWL-defined pipeline for using IDR to produce a set of peaks given two replicate eCLIP peaks.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>eclipregionnormalize</code></td>
<td style="text-align: left;"><code>0.0.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/YeoLab/region_based_CLIP_enrichment">Website</a></td>
<td style="text-align: left;">Pipeline for region-based enrichment. This workflow is defined using CWL v1.0 spec.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>ehive</code></td>
<td style="text-align: left;"><code>2.7.0</code><br></td>
<td style="text-align: left;"><a href="https://ensembl-hive.readthedocs.io/en/version-2.7.0/">Website</a></td>
<td style="text-align: left;">EnsEMBL Hive is a system for creating and running pipelines on a distributed compute resource.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>emg-viral-pipeline</code></td>
<td style="text-align: left;"><code>0.3.0</code><br><code>2.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/EBI-Metagenomics/emg-viral-pipeline">Website</a></td>
<td style="text-align: left;">VIRify is a recently developed pipeline for the detection, annotation, and taxonomic classification of viral contigs in metagenomic and metatranscriptomic assemblies.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>enhancer-dissection</code></td>
<td style="text-align: left;"><code>1.0.0</code><br><code>1.0.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/nowling-lab/enhancer-dissection">Website</a></td>
<td style="text-align: left;">The two programs in this project display variant data, motif data (via highlights).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>ensembl-hive</code></td>
<td style="text-align: left;"><code>2.7.0</code><br></td>
<td style="text-align: left;"><a href="https://ensembl-hive.readthedocs.io/en/version-2.7.0/">Website</a></td>
<td style="text-align: left;">EnsEMBL Hive is a system for creating and running pipelines on a distributed compute resource.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>ensembl-vep</code></td>
<td style="text-align: left;"><code>103.1</code><br><code>111.0</code><br></td>
<td style="text-align: left;"><a href="https://useast.ensembl.org/info/docs/tools/vep/index.html">Website</a></td>
<td style="text-align: left;">VEP determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>esm</code></td>
<td style="text-align: left;"><code>2.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/facebookresearch/esm">Website</a></td>
<td style="text-align: left;">Evolutionary Scale Modeling (esm). Pretrained language models for proteins.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>esmfold</code></td>
<td style="text-align: left;"><code>1.0.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/facebookresearch/esm">Website</a></td>
<td style="text-align: left;">Evolutionary Scale Modeling (esm). Pretrained language models for proteins.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>express</code></td>
<td style="text-align: left;"><code>1.5.1</code><br></td>
<td style="text-align: left;"><a href="https://pachterlab.github.io/eXpress/">Website</a></td>
<td style="text-align: left;">eXpress is a streaming tool for quantifying the abundances of a set of target sequences from sampled subsequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>fastq-pair</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/linsalrob/fastq-pair">Website</a></td>
<td style="text-align: left;">Rewrite paired end fastq files to make sure that all reads have a mate and to separate out singletons..</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>fastq-screen</code></td>
<td style="text-align: left;"><code>0.15.2</code><br></td>
<td style="text-align: left;"><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/">Website</a></td>
<td style="text-align: left;">FastQ Screen allows you to screen a library of sequences in FastQ format against a set of sequence databases so you can see if the composition of the library matches with what you expect.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>fastqc</code></td>
<td style="text-align: left;"><code>0.11.9</code><br><code>0.12.1</code><br></td>
<td style="text-align: left;"><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">Website</a></td>
<td style="text-align: left;">FastQC aims to provide a simple way to do some quality control checks on raw sequence data coming from high throughput sequencing pipelines.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>fastvifi</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/sara-javadzadeh/FastViFi">Website</a></td>
<td style="text-align: left;">A software to detect viral infection and integration sites for different viruses. FastViFi relies on ViFi and Kraken2 tools.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>fiji</code></td>
<td style="text-align: left;"><code>2.16.0</code><br></td>
<td style="text-align: left;"><a href="https://imagej.net/software/fiji/">Website</a></td>
<td style="text-align: left;">Fiji is an image processing package‚Äîa batteries-included distribution of ImageJ2, bundling a lot of plugins which facilitate scientific image analysis.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>fitsne</code></td>
<td style="text-align: left;"><code>1.2.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/KlugerLab/FIt-SNE">Website</a></td>
<td style="text-align: left;">FFT-accelerated Interpolation-based t-SNE (FIt-SNE)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>fluxsimulator</code></td>
<td style="text-align: left;"><code>1.2.1</code><br></td>
<td style="text-align: left;"><a href="https://confluence.sammeth.net/display/SIM/Home">Website</a></td>
<td style="text-align: left;">The Flux Simulator aims at modeling RNA-Seq experiments in silico. Sequencing reads are produced from a reference genome according annotated transcripts.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gatk</code></td>
<td style="text-align: left;"><code>3.8</code><br><code>3.8.1</code><br><code>4.1.7.0</code><br><code>4.4.0.0</code><br><code>4.5.0.0</code><br><code>4.6.2.0</code><br></td>
<td style="text-align: left;"><a href="https://gatk.broadinstitute.org/hc/en-us">Website</a></td>
<td style="text-align: left;">The GATK is the industry standard for identifying SNPs and indels in germline DNA and RNAseq data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gcta</code></td>
<td style="text-align: left;"><code>1.94.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/jianyangqt/gcta">Website</a></td>
<td style="text-align: left;">a tool for Genome-wide Complex Trait Analysis</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gdc-client</code></td>
<td style="text-align: left;"><code>1.6.1</code><br></td>
<td style="text-align: left;"><a href="https://docs.gdc.cancer.gov/Data_Transfer_Tool/Users_Guide/Data_Download_and_Upload/">Website</a></td>
<td style="text-align: left;">Illumina Array Analysis Platform cli</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>geneformer</code></td>
<td style="text-align: left;"><code>0.1.0</code><br></td>
<td style="text-align: left;"><a href="https://huggingface.co/ctheodoris/Geneformer">Website</a></td>
<td style="text-align: left;">Geneformer is a foundation transformer model pretrained on a large-scale corpus of ~30 million single cell transcriptomes to enable context-aware predictions in settings with limited data in network biology.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>genomebrowse</code></td>
<td style="text-align: left;"><code>3.1.0</code><br></td>
<td style="text-align: left;"><a href="https://www.goldenhelix.com/products/GenomeBrowse/">Website</a></td>
<td style="text-align: left;">The free Golden Helix GenomeBrowse tool delivers stunning visualizations of your genomic data that give you the power to see what is occurring at each base pair in your samples.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>genrich</code></td>
<td style="text-align: left;"><code>0.6.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/jsh58/Genrich">Website</a></td>
<td style="text-align: left;">Genrich is a peak-caller for genomic enrichment assays e.g. ChIP-seq, ATAC-seq. It analyzes alignment files generated following the assay and produces a file detailing peaks of significant enrichment.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gethomologues</code></td>
<td style="text-align: left;"><code>3.7.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/eead-csic-compbio/get_homologues">Website</a></td>
<td style="text-align: left;">Get_Homologues is a versatile software package for pan-genome analysis.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gffcompare</code></td>
<td style="text-align: left;"><code>0.10.1</code><br></td>
<td style="text-align: left;"><a href="https://ccb.jhu.edu/software/stringtie/gffcompare.shtml">Website</a></td>
<td style="text-align: left;">The program gffcompare can be used to compare, merge, annotate and estimate accuracy of one or more GFF files (the ‚Äúquery‚Äù files), when compared with a reference annotation (also provided as GFF).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gistic</code></td>
<td style="text-align: left;"><code>2.0.23</code><br></td>
<td style="text-align: left;"><a href="https://github.com/broadinstitute/gistic2">Website</a></td>
<td style="text-align: left;">The GISTIC module identifies regions of the genome that are significantly amplified or deleted across a set of samples.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>glow</code></td>
<td style="text-align: left;"><code>2.0.0</code><br></td>
<td style="text-align: left;"><a href="https://glow.readthedocs.io/en/latest/index.html">Website</a></td>
<td style="text-align: left;">Glow is an open-source toolkit for working with genomic data at biobank-scale and beyond.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gmap-gsnap</code></td>
<td style="text-align: left;"><code>2016-05-01</code><br><code>2021-12-17</code><br></td>
<td style="text-align: left;"><a href="http://research-pub.gene.com/gmap/">Website</a></td>
<td style="text-align: left;">A genomic mapping and alignment program for mRNA and EST sequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>google-cloud-sdk</code></td>
<td style="text-align: left;"><code>335.0.0</code><br><code>393.0.0</code><br><code>525.0.0</code><br></td>
<td style="text-align: left;"><a href="https://cloud.google.com/storage/docs/gsutil">Website</a></td>
<td style="text-align: left;">gsutil is a command-line tool for interacting with Google Cloud Storage.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gossamer</code></td>
<td style="text-align: left;"><code>1.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/data61/gossamer">Website</a></td>
<td style="text-align: left;">Gossamer is an application for doing de novo assembly of high throughput sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gridss</code></td>
<td style="text-align: left;"><code>2.13.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/PapenfussLab/gridss">Website</a></td>
<td style="text-align: left;">GRIDSS is a module software suite containing tools useful for the detection of genomic rearrangements.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>gsutil</code></td>
<td style="text-align: left;"><code>5.3.4</code><br></td>
<td style="text-align: left;"><a href="https://cloud.google.com/storage/docs/gsutil">Website</a></td>
<td style="text-align: left;">gsutil is a command-line tool for interacting with Google Cloud Storage.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>hail</code></td>
<td style="text-align: left;"><code>0.2.133</code><br></td>
<td style="text-align: left;"><a href="https://hail.is/docs/0.2/index.html">Website</a></td>
<td style="text-align: left;">Hail is an open-source library for scalable data exploration and analysis, with a particular emphasis on genomics.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>haslr</code></td>
<td style="text-align: left;"><code>0.8a1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/vpc-ccg/haslr">Website</a></td>
<td style="text-align: left;">HASLR is a tool for rapid genome assembly of long sequencing reads.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>hicexplorer</code></td>
<td style="text-align: left;"><code>3.7.6</code><br></td>
<td style="text-align: left;"><a href="https://github.com/deeptools/HiCExplorer">Website</a></td>
<td style="text-align: left;">A set of tools for the analysis and visualization of chromosome conformation data</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>hisat2</code></td>
<td style="text-align: left;"><code>2.0.5</code><br><code>2.2.1</code><br></td>
<td style="text-align: left;"><a href="http://daehwankimlab.github.io/hisat2/">Website</a></td>
<td style="text-align: left;">HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes as well as to a single reference genome.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>hisatgenotype</code></td>
<td style="text-align: left;"><code>1.3.3</code><br></td>
<td style="text-align: left;"><a href="https://daehwankimlab.github.io/hisat-genotype/">Website</a></td>
<td style="text-align: left;">Next-generation genomic analysis software platform.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>hmmer</code></td>
<td style="text-align: left;"><code>3.3.2</code><br></td>
<td style="text-align: left;"><a href="http://hmmer.org/">Website</a></td>
<td style="text-align: left;">HMMER is used for searching sequence databases for sequence homologs, and for making sequence alignments.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>homer</code></td>
<td style="text-align: left;"><code>4.9.1</code><br><code>4.11.1</code><br><code>5.1</code><br></td>
<td style="text-align: left;"><a href="http://homer.ucsd.edu/homer/">Website</a></td>
<td style="text-align: left;">HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>htseq</code></td>
<td style="text-align: left;"><code>0.13.5</code><br></td>
<td style="text-align: left;"><a href="https://pypi.org/project/HTSeq/">Website</a></td>
<td style="text-align: left;">HTSeq is a Python package for analysis of high-throughput sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>htslib</code></td>
<td style="text-align: left;"><code>1.9</code><br><code>1.10.2</code><br><code>1.11</code><br><code>1.15</code><br><code>1.2</code><br><code>1.22.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/samtools/htslib">Website</a></td>
<td style="text-align: left;">HTSlib is an implementation of a unified C library for accessing common file formats, such as SAM, CRAM and VCF, used for high-throughput sequencing data, and is the core library used by samtools and bcftools.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>humann</code></td>
<td style="text-align: left;"><code>3.0.0a4</code><br><code>3.9</code><br></td>
<td style="text-align: left;"><a href="https://github.com/biobakery/humann">Website</a></td>
<td style="text-align: left;">HUMAnN 3.0 is the next generation of HUMAnN (HMP Unified Metabolic Analysis Network).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>iaap-cli</code></td>
<td style="text-align: left;"><code>1.1.0</code><br></td>
<td style="text-align: left;"><a href="https://support.illumina.com/array/array_software/illumina-array-analysis-platform.html">Website</a></td>
<td style="text-align: left;">Illumina Array Analysis Platform cli</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>idr</code></td>
<td style="text-align: left;"><code>2.0.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/nboley/idr">Website</a></td>
<td style="text-align: left;">The IDR (Irreproducible Discovery Rate) framework is a uniÔ¨Åed approach to measure the reproducibility of Ô¨Åndings identiÔ¨Åed from replicate experiments and provide highly stable thresholds based on reproducibility.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>igv</code></td>
<td style="text-align: left;"><code>2.17.4</code><br><code>2.19.4</code><br></td>
<td style="text-align: left;"><a href="https://software.broadinstitute.org/software/igv/">Website</a></td>
<td style="text-align: left;">The Integrative Genomics Viewer (IGV) is a high-performance, easy-to-use, interactive tool for the visual exploration of genomic data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>illumina-utils</code></td>
<td style="text-align: left;"><code>2.1</code><br><code>2.11</code><br></td>
<td style="text-align: left;"><a href="https://github.com/merenlab/illumina-utils">Website</a></td>
<td style="text-align: left;">Software to perform various operations on FASTQ files (such as demultiplexing raw Illumina files, merging partial or complete overlaps, and/or performing quality filtering).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>impute</code></td>
<td style="text-align: left;"><code>2.3.2</code><br></td>
<td style="text-align: left;"><a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html">Website</a></td>
<td style="text-align: left;">IMPUTE version 2 is a genotype imputation and haplotype phasing program.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>impute5</code></td>
<td style="text-align: left;"><code>1.1.5</code><br></td>
<td style="text-align: left;"><a href="https://jmarchini.org/software/#impute-5">Website</a></td>
<td style="text-align: left;">IMPUTE 5 is a genotype imputation method that can scale to reference panels with millions of samples.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>isoseq</code></td>
<td style="text-align: left;"><code>4.3.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/PacificBiosciences/IsoSeq">Website</a></td>
<td style="text-align: left;">Identify transcripts in PacBio single-molecule sequencing data</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>jvarkit</code></td>
<td style="text-align: left;"><code>2021.10.13</code><br></td>
<td style="text-align: left;"><a href="https://github.com/lindenb/jvarkit">Website</a></td>
<td style="text-align: left;">Java utilities for bioinformatics.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>k2mem</code></td>
<td style="text-align: left;"><code>1.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/davide92/K2Mem">Website</a></td>
<td style="text-align: left;">K2Mem is a variant of Kraken 2 taxonomic sequence classification system that can learn from the previous classifications.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>kaiju</code></td>
<td style="text-align: left;"><code>1.7.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/bioinformatics-centre/kaiju">Website</a></td>
<td style="text-align: left;">Kaiju is a program for the taxonomic classification of high-throughput sequencing reads, e.g., Illumina or Roche/454, from whole-genome sequencing of metagenomic DNA.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>kallisto</code></td>
<td style="text-align: left;"><code>0.46.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/pachterlab/kallisto">Website</a></td>
<td style="text-align: left;">kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>king</code></td>
<td style="text-align: left;"><code>2.1.2</code><br></td>
<td style="text-align: left;"><a href="https://www.kingrelatedness.com/">Website</a></td>
<td style="text-align: left;">Kinship-based INference for Gwas</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>kneaddata</code></td>
<td style="text-align: left;"><code>0.10.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/biobakery/kneaddata">Website</a></td>
<td style="text-align: left;">KneadData is a tool designed to perform quality control on metagenomic and metatranscriptomic sequencing data, especially data from microbiome experiments.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>kobas</code></td>
<td style="text-align: left;"><code>3.0.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/xmao/kobas">Website</a></td>
<td style="text-align: left;">KOBAS (KEGG Orthology-Based Annotation System) is a tool for the annotation of sequences by KEGG Orthology terms. KOBAS also identifies enriched pathways and uses KEGG Pathway, PID, BioCyc, Reactome, Panther and human data from OMIM, KEGG Disease, FunDO, GAD, NHGRI, and GWAS databases.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>kraken2</code></td>
<td style="text-align: left;"><code>2.1.1</code><br><code>2.1.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/DerrickWood/kraken2">Website</a></td>
<td style="text-align: left;">Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>krakenuniq</code></td>
<td style="text-align: left;"><code>0.5.8</code><br></td>
<td style="text-align: left;"><a href="https://github.com/fbreitwieser/krakenuniq">Website</a></td>
<td style="text-align: left;">Metagenomics classifier with unique k-mer counting for more specific results.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>lefse</code></td>
<td style="text-align: left;"><code>1.1.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/SegataLab/lefse">Website</a></td>
<td style="text-align: left;">LEfSe determines the features most likely to explain differences between classes by coupling standard tests for statistical significance with additional tests encoding biological consistency and effect relevance.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>lep-map</code></td>
<td style="text-align: left;"><code>3</code><br></td>
<td style="text-align: left;"><a href="https://sourceforge.net/p/lep-map3/wiki/LM3%20Home/">Website</a></td>
<td style="text-align: left;">Lep-MAP3 (LM3) is a novel linkage map construction software suite.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>lumpy</code></td>
<td style="text-align: left;"><code>0.2.13</code><br></td>
<td style="text-align: left;"><a href="https://github.com/arq5x/lumpy-sv">Website</a></td>
<td style="text-align: left;">A probabilistic framework for structural variant discovery.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>macs2</code></td>
<td style="text-align: left;"><code>2.2.9.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/macs3-project/MACS">Website</a></td>
<td style="text-align: left;">Model-based Analysis for ChIP-Seq</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>macs3</code></td>
<td style="text-align: left;"><code>3.0.2</code><br></td>
<td style="text-align: left;"><a href="https://macs3-project.github.io/MACS/">Website</a></td>
<td style="text-align: left;">Model-based Analysis for ChIP-Seq</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>maftools</code></td>
<td style="text-align: left;"><code>2.22.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/PoisonAlien/maftools">Website</a></td>
<td style="text-align: left;">Provides a comprehensive set of functions for processing MAF files and to perform most commonly used analyses in cancer genomics.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>magicblast</code></td>
<td style="text-align: left;"><code>1.5.0</code><br></td>
<td style="text-align: left;"><a href="https://ncbi.github.io/magicblast/">Website</a></td>
<td style="text-align: left;">Magic-BLAST is a tool for mapping large next-generation RNA or DNA sequencing runs against a whole genome or transcriptome.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>magma</code></td>
<td style="text-align: left;"><code>1.1</code><br></td>
<td style="text-align: left;"><a href="https://ctg.cncr.nl/software/magma">Website</a></td>
<td style="text-align: left;">MAGMA is a tool for gene analysis and generalized gene-set analysis of GWAS data. It can be used to analyse both raw genotype data as well as summary SNP p-values from a previous GWAS or meta-analysis.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>manorm</code></td>
<td style="text-align: left;"><code>1.1.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/shao-lab/MAnorm">Website</a></td>
<td style="text-align: left;">A robust model for quantitative comparison of ChIP-Seq data sets.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>manta</code></td>
<td style="text-align: left;"><code>1.6.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/Illumina/manta">Website</a></td>
<td style="text-align: left;">Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads. Manta discovers, assembles and scores large-scale SVs, medium-sized indels and large insertions within a single efficient workflow.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>megahit</code></td>
<td style="text-align: left;"><code>1.1.1</code><br><code>1.2.9</code><br></td>
<td style="text-align: left;"><a href="https://github.com/voutcn/megahit">Website</a></td>
<td style="text-align: left;">MEGAHIT is an ultra-fast and memory-efficient NGS assembler. It is optimized for metagenomes, but also works well on generic single genome assembly (small or mammalian size) and single-cell assembly.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>meme</code></td>
<td style="text-align: left;"><code>5.3.1</code><br><code>5.4.1</code><br></td>
<td style="text-align: left;"><a href="https://meme-suite.org/meme/doc/download.html">Website</a></td>
<td style="text-align: left;">The MEME Suite allows the biologist to discover novel motifs in collections of unaligned nucleotide or protein sequences, and to perform a wide variety of other motif-based analyses.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>metagenemark</code></td>
<td style="text-align: left;"><code>3.38</code><br></td>
<td style="text-align: left;"><a href="http://exon.gatech.edu/GeneMark/">Website</a></td>
<td style="text-align: left;">Metagenomic sequences can be analyzed by MetaGeneMark , the program optimized for speed.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>metaphlan</code></td>
<td style="text-align: left;"><code>4.1.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/biobakery/MetaPhlAn">Website</a></td>
<td style="text-align: left;">MetaPhlAn is a computational tool for profiling the composition of microbial communites from metagenomic shotgun sequencing data with species level resolution.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>metilene</code></td>
<td style="text-align: left;"><code>0.2.6</code><br></td>
<td style="text-align: left;"><a href="http://www.bioinf.uni-leipzig.de/Software/metilene/">Website</a></td>
<td style="text-align: left;">Fast and sensitive detection of differential DNA methylation</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>migec</code></td>
<td style="text-align: left;"><code>1.2.9</code><br></td>
<td style="text-align: left;"><a href="https://github.com/mikessh/migec">Website</a></td>
<td style="text-align: left;">Molecular Identifier Guided Error Correction pipeline</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>minpath</code></td>
<td style="text-align: left;"><code>1.5</code><br></td>
<td style="text-align: left;"><a href="https://github.com/mgtools/MinPath">Website</a></td>
<td style="text-align: left;">MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological pathways for a query dataset.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>mira</code></td>
<td style="text-align: left;"><code>4.0.2</code><br></td>
<td style="text-align: left;"><a href="https://sourceforge.net/p/mira-assembler/wiki/Home/">Website</a></td>
<td style="text-align: left;">MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>mitohpc</code></td>
<td style="text-align: left;"><code>1.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/dpuiu/MitoHPC">Website</a></td>
<td style="text-align: left;">MitoHPC is used for Calling Mitochondrial Homplasmies and Heteroplasmies, Mitochondrial High Performance Caller</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>mixcr</code></td>
<td style="text-align: left;"><code>2.1.11</code><br><code>4.1.0</code><br><code>4.7.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/milaboratory/mixcr">Website</a></td>
<td style="text-align: left;">MiXCR is a universal software for fast and accurate analysis of raw T- or B- cell receptor repertoire sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>mosaik</code></td>
<td style="text-align: left;"><code>0.9.0891</code><br></td>
<td style="text-align: left;"><a href="https://code.google.com/archive/p/mosaik-aligner/">Website</a></td>
<td style="text-align: left;">MOSAIK is a reference-guided aligner for next-generation sequencing technologies.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>mtoolbox</code></td>
<td style="text-align: left;"><code>1.2.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/mitoNGS/MToolBox">Website</a></td>
<td style="text-align: left;">A bioinformatics pipeline to analyze mtDNA from NGS data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>multiqc</code></td>
<td style="text-align: left;"><code>1.11</code><br><code>1.17</code><br><code>1.18</code><br><code>1.32</code><br></td>
<td style="text-align: left;"><a href="https://github.com/ewels/MultiQC">Website</a></td>
<td style="text-align: left;">MultiQC is a tool to create a single report with interactive plots for multiple bioinformatics analyses across many samples.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>mummer</code></td>
<td style="text-align: left;"><code>4.0.1</code><br></td>
<td style="text-align: left;"><a href="https://mummer4.github.io/index.html">Website</a></td>
<td style="text-align: left;">MUMmer is a system for rapidly aligning large DNA sequences to one another..</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>muscle</code></td>
<td style="text-align: left;"><code>3.8.1551</code><br></td>
<td style="text-align: left;"><a href="https://github.com/rcedgar/muscle">Website</a></td>
<td style="text-align: left;">MUSCLE is widely-used software for making multiple alignments of biological sequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>mutsig2cv</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/getzlab/MutSig2CV">Website</a></td>
<td style="text-align: left;">MutSig2CV analyzes somatic point mutations discovered in DNA sequencing, identifying genes mutated more often than expected by chance given inferred background mutation processes.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>ngmerge</code></td>
<td style="text-align: left;"><code>0.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/jsh58/NGmerge">Website</a></td>
<td style="text-align: left;">Merging paired-end reads and removing sequencing adapters</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>octopus</code></td>
<td style="text-align: left;"><code>0.7.4</code><br></td>
<td style="text-align: left;"><a href="https://luntergroup.github.io/octopus/">Website</a></td>
<td style="text-align: left;">Haplotype-based variant calling</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>oncoimpact</code></td>
<td style="text-align: left;"><code>0.9.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/CSB5/OncoIMPACT">Website</a></td>
<td style="text-align: left;">OncoIMPACT is a first-in-class algorithmic framework that nominates patient-specific driver genes by integratively modeling genomic mutations (point, structural and copy-number) and the resulting perturbations in transcriptional programs via defined molecular networks.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>pacbioapps</code></td>
<td style="text-align: left;"><code>latest</code><br></td>
<td style="text-align: left;"><a href="https://github.com/PacificBiosciences/pbbioconda">Website</a></td>
<td style="text-align: left;">PacBio Secondary Analysis Tools on Bioconda</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>parsebiosciences</code></td>
<td style="text-align: left;"><code>1.4.2</code><br></td>
<td style="text-align: left;"><a href="https://www.parsebiosciences.com/">Website</a></td>
<td style="text-align: left;">Parse's single cell kits facilitate scalable single cell RNA-seq, enabling you to go all the way from samples in a single cell suspension to sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>pgap</code></td>
<td style="text-align: left;"><code>2021-07-01</code><br></td>
<td style="text-align: left;"><a href="https://github.com/ncbi/pgap">Website</a></td>
<td style="text-align: left;">The NCBI Prokaryotic Genome Annotation Pipeline is designed to annotate bacterial and archaeal genomes (chromosomes and plasmids).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>phenolyzer</code></td>
<td style="text-align: left;"><code>0.4.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/WGLab/phenolyzer">Website</a></td>
<td style="text-align: left;">Phenolyzer is a software that implements phenotype-based prioritization of candidate genes for human diseases.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>phispy</code></td>
<td style="text-align: left;"><code>4.2.21</code><br></td>
<td style="text-align: left;"><a href="https://github.com/linsalrob/PhiSpy">Website</a></td>
<td style="text-align: left;">PhiSpy identifies prophages in Bacterial (and probably Archaeal) genomes. Given an annotated genome it will use several approaches to identify the most likely prophage regions..</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>picard</code></td>
<td style="text-align: left;"><code>2.6.0</code><br><code>2.25.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/broadinstitute/picard">Website</a></td>
<td style="text-align: left;">A set of command line tools (in Java) for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>pignnemmenv</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/dodaltuin/soft-tissue-pignn">Website</a></td>
<td style="text-align: left;">Physics-Informed Graph Neural Network Emulation of Soft-Tissue Mechanics (CMAME 2023)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>piranha</code></td>
<td style="text-align: left;"><code>1.2.1</code><br></td>
<td style="text-align: left;"><a href="https://smithlabresearch.org/software/piranha/">Website</a></td>
<td style="text-align: left;">Piranha is a peak-caller for CLIP- and RIP-Seq data. It takes input in BED or BAM format and identifies regions of statistically significant read enrichment.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>plink</code></td>
<td style="text-align: left;"><code>1.9.0</code><br><code>2.0.0a6.26</code><br></td>
<td style="text-align: left;"><a href="https://zzz.bwh.harvard.edu/plink/">Website</a></td>
<td style="text-align: left;">PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>prodigal</code></td>
<td style="text-align: left;"><code>2.6.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/hyattpd/Prodigal">Website</a></td>
<td style="text-align: left;">Fast, reliable protein-coding gene prediction for prokaryotic genomes.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>prokka</code></td>
<td style="text-align: left;"><code>1.14.6</code><br></td>
<td style="text-align: left;"><a href="https://github.com/tseemann/prokka">Website</a></td>
<td style="text-align: left;">Prokka is a software tool to annotate bacterial, archaeal and viral genomes quickly and produce standards-compliant output files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>prscs</code></td>
<td style="text-align: left;"><code>1.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/getian107/PRScs">Website</a></td>
<td style="text-align: left;">PRS-CS is a Python based command line tool that infers posterior SNP effect sizes under continuous shrinkage (CS) priors using GWAS summary statistics and an external LD reference panel.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>prsice2</code></td>
<td style="text-align: left;"><code>2.3.3</code><br></td>
<td style="text-align: left;"><a href="https://choishingwan.github.io/PRS-Tutorial/prsice/">Website</a></td>
<td style="text-align: left;">Polygenic Risk Score software.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>pureclip</code></td>
<td style="text-align: left;"><code>1.3.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/skrakau/PureCLIP">Website</a></td>
<td style="text-align: left;">PureCLIP is a tool to detect protein-RNA interaction footprints from single-nucleotide CLIP-seq data, such as iCLIP and eCLIP..</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>pyani</code></td>
<td style="text-align: left;"><code>0.2.13.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/widdowquinn/pyani">Website</a></td>
<td style="text-align: left;">Application and Python module for whole-genome classification of microbes using Average Nucleotide Identity.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>pyega3</code></td>
<td style="text-align: left;"><code>5.2.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/EGA-archive/ega-download-client">Website</a></td>
<td style="text-align: left;">The pyEGA3 download client is a python-based tool for viewing and downloading files from authorized EGA datasets. pyEGA3 uses the EGA Data API and has several key features.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>pyscenic</code></td>
<td style="text-align: left;"><code>0.12.1</code><br></td>
<td style="text-align: left;"><a href="https://pyscenic.readthedocs.io/en/latest/index.html">Website</a></td>
<td style="text-align: left;">pySCENIC is a lightning-fast python implementation of the SCENIC pipeline which enables biologists to infer transcription factors, gene regulatory networks and cell types from single-cell RNA-seq data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>qctool</code></td>
<td style="text-align: left;"><code>2.0.8</code><br></td>
<td style="text-align: left;"><a href="https://www.well.ox.ac.uk/~gav/qctool_v2">Website</a></td>
<td style="text-align: left;">QCTOOL is a command-line utility program for manipulation and quality control of gwas datasets and other genome-wide data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>qiime2</code></td>
<td style="text-align: left;"><code>amplicon-2024.10</code><br><code>metagenome-2024.10</code><br><code>pathogenome-2024.10</code><br></td>
<td style="text-align: left;"><a href="https://qiime2.org/">Website</a></td>
<td style="text-align: left;">Qiime2 is a next-generation microbiome bioinformatics platform.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>regenie</code></td>
<td style="text-align: left;"><code>3.1.1</code><br></td>
<td style="text-align: left;"><a href="https://rgcgithub.github.io/regenie/">Website</a></td>
<td style="text-align: left;">regenie is a C++ program for whole genome regression modelling of large genome-wide association studies.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>repeatmasker</code></td>
<td style="text-align: left;"><code>4.1.7</code><br></td>
<td style="text-align: left;"><a href="https://www.repeatmasker.org/">Website</a></td>
<td style="text-align: left;">RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>rgi</code></td>
<td style="text-align: left;"><code>6.0.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/arpcard/rgi">Website</a></td>
<td style="text-align: left;">This application is used to predict antibiotic resistome(s) from protein or nucleotide data based on homology and SNP models.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>rhapsody</code></td>
<td style="text-align: left;"><code>0.9.8</code><br></td>
<td style="text-align: left;"><a href="https://github.com/prody/rhapsody">Website</a></td>
<td style="text-align: left;">Python program, based on ProDy, for pathogenicity prediction of human missense variants.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>rmats</code></td>
<td style="text-align: left;"><code>4.3.0</code><br></td>
<td style="text-align: left;"><a href="https://rnaseq-mats.sourceforge.io/index.html">Website</a></td>
<td style="text-align: left;">MATS is a computational tool to detect differential alternative splicing events from RNA-Seq data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>rosettafold-all-atom</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/baker-laboratory/RoseTTAFold-All-Atom">Website</a></td>
<td style="text-align: left;">RoseTTAFold All-Atom is a biomolecular structure prediction neural network that can predict a broad range of biomolecular assemblies including proteins, nucleic acids, small molecules, covalent modifications and metals as outlined in the RFAA paper.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>rsem</code></td>
<td style="text-align: left;"><code>1.3.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/deweylab/RSEM">Website</a></td>
<td style="text-align: left;">RSEM RNA-Seq by Expectation-Maximization</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>rseqc</code></td>
<td style="text-align: left;"><code>3.0.1</code><br><code>4.0.0</code><br></td>
<td style="text-align: left;"><a href="http://rseqc.sourceforge.net/">Website</a></td>
<td style="text-align: left;">RNA Sequence Quality Control Package</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>rvtests</code></td>
<td style="text-align: left;"><code>2.1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/zhanxw/rvtests">Website</a></td>
<td style="text-align: left;">Rvtests, which stands for Rare Variant tests, is a flexible software package for genetic association analysis for sequence datasets.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>sai-10k-calc</code></td>
<td style="text-align: left;"><code>1.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/adavi4/SAI-10k-calc">Website</a></td>
<td style="text-align: left;">Calculator for the prediction of pseudoexonization, intron retention, and exon deletion.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>saige</code></td>
<td style="text-align: left;"><code>1.2.0</code><br></td>
<td style="text-align: left;"><a href="https://saigegit.github.io/SAIGE-doc/">Website</a></td>
<td style="text-align: left;">SAIGE is an R package developed with Rcpp for genome-wide association tests in large-scale data sets and biobanks.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>salmon</code></td>
<td style="text-align: left;"><code>1.5.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/COMBINE-lab/salmon">Website</a></td>
<td style="text-align: left;">Salmon is a wicked-fast program to produce a highly-accurate, transcript-level quantification estimates from RNA-seq data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>sambamba</code></td>
<td style="text-align: left;"><code>0.8.1</code><br><code>1.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/biod/sambamba">Website</a></td>
<td style="text-align: left;">Sambamba is a high performance highly parallel robust and fast tool (and library), written in the D programming language, for working with SAM and BAM files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>samblaster</code></td>
<td style="text-align: left;"><code>0.1.26</code><br></td>
<td style="text-align: left;"><a href="https://github.com/GregoryFaust/samblaster">Website</a></td>
<td style="text-align: left;">samblaster is a fast and flexible program for marking duplicates in read-id grouped1 paired-end SAM files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>samtools</code></td>
<td style="text-align: left;"><code>1.11</code><br><code>1.15.1</code><br><code>1.2</code><br><code>1.22.1</code><br></td>
<td style="text-align: left;"><a href="http://www.htslib.org/">Website</a></td>
<td style="text-align: left;">Samtools is a suite of programs for interacting with high-throughput sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>scgpt</code></td>
<td style="text-align: left;"><code>0.2.1</code><br></td>
<td style="text-align: left;"><a href="https://scgpt.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">scGPT, a Python package for single-cell multi-omic data analysis using pretrained foundation models.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>scvelo</code></td>
<td style="text-align: left;"><code>0.3.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/theislab/scvelo">Website</a></td>
<td style="text-align: left;">scVelo is a scalable toolkit for RNA velocity analysis in single cells, based on Bergen et al. (Nature Biotech, 2020).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>seqkit</code></td>
<td style="text-align: left;"><code>0.15.0</code><br><code>2.10.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/shenwei356/seqkit">Website</a></td>
<td style="text-align: left;">A cross-platform and ultrafast toolkit for FASTA/Q file manipulation</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>seqtk</code></td>
<td style="text-align: left;"><code>1.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/lh3/seqtk">Website</a></td>
<td style="text-align: left;">Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>shapeit</code></td>
<td style="text-align: left;"><code>2.904</code><br><code>5.1.1</code><br></td>
<td style="text-align: left;"><a href="https://odelaneau.github.io/shapeit5/">Website</a></td>
<td style="text-align: left;">SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>shapeit4</code></td>
<td style="text-align: left;"><code>4.2.2</code><br></td>
<td style="text-align: left;"><a href="https://odelaneau.github.io/shapeit4/">Website</a></td>
<td style="text-align: left;">SHAPEIT4 is a fast and accurate method for estimation of haplotypes (aka phasing) for SNP array and high coverage sequencing data. The version 4 is a refactored and improved version of the SHAPEIT algorithm with multiple key additional features.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>shapeit5</code></td>
<td style="text-align: left;"><code>5.1.1</code><br></td>
<td style="text-align: left;"><a href="https://odelaneau.github.io/shapeit4/">Website</a></td>
<td style="text-align: left;">SHAPEIT4 is a fast and accurate method for estimation of haplotypes (aka phasing) for SNP array and high coverage sequencing data. The version 4 is a refactored and improved version of the SHAPEIT algorithm with multiple key additional features.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>shiftx2</code></td>
<td style="text-align: left;"><code>1.13</code><br></td>
<td style="text-align: left;"><a href="http://www.shiftx2.ca/index.html">Website</a></td>
<td style="text-align: left;">SHIFTX2 predicts both the backbone and side chain 1H, 13C and 15N chemical shifts for proteins using their structural (PDB) coordinates as input.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>snakemake</code></td>
<td style="text-align: left;"><code>8.27.1</code><br></td>
<td style="text-align: left;"><a href="https://snakemake.readthedocs.io/en/stable/index.html">Website</a></td>
<td style="text-align: left;">The Snakemake workflow management system is a tool to create reproducible and scalable data analyses. Workflows are described via a human readable, Python based language. They can be seamlessly scaled to server, cluster, grid and cloud environments, without the need to modify the workflow definition. Finally, Snakemake workflows can entail a description of required software, which will be automatically deployed to any execution environment.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>snpeff</code></td>
<td style="text-align: left;"><code>4.3</code><br><code>5.2</code><br></td>
<td style="text-align: left;"><a href="http://pcingola.github.io/SnpEff/se_introduction/">Website</a></td>
<td style="text-align: left;">SnpEff is a variant annotation and effect prediction tool.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>snphylo</code></td>
<td style="text-align: left;"><code>20141127</code><br></td>
<td style="text-align: left;"><a href="https://github.com/thlee/SNPhylo">Website</a></td>
<td style="text-align: left;">SNPhylo users can construct a phylogenetic tree from a file containing huge SNP data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>snptest</code></td>
<td style="text-align: left;"><code>2.5.6</code><br></td>
<td style="text-align: left;"><a href="https://www.well.ox.ac.uk/~gav/snptest/">Website</a></td>
<td style="text-align: left;">SNPTEST is a program for the analysis of single SNP association in genome-wide studies.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>spaceranger</code></td>
<td style="text-align: left;"><code>1.2.2</code><br><code>1.3.1</code><br><code>2.0.0</code><br><code>2.0.1</code><br><code>2.1.0</code><br><code>4.0.1</code><br></td>
<td style="text-align: left;"><a href="https://support.10xgenomics.com/spatial-gene-expression/software/overview/welcome">Website</a></td>
<td style="text-align: left;">Space Ranger is a set of analysis pipelines that process Visium Spatial Gene Expression data with brightfield and fluorescnce microscope images.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>spades</code></td>
<td style="text-align: left;"><code>3.15.0</code><br><code>3.15.2</code><br><code>3.15.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/ablab/spades">Website</a></td>
<td style="text-align: left;">SPAdes is an assembly toolkit containing various assembly pipelines.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>spliceai</code></td>
<td style="text-align: left;"><code>1.3.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/Illumina/SpliceAI">Website</a></td>
<td style="text-align: left;">A deep learning-based tool to identify splice variants.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>sratoolkit</code></td>
<td style="text-align: left;"><code>3.0.0</code><br><code>3.0.10</code><br><code>3.1.0</code><br><code>3.1.1</code><br><code>3.2.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/ncbi/sra-tools">Website</a></td>
<td style="text-align: left;">Sequence Read Archive Toolkit(SRA)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>staphb-tk</code></td>
<td style="text-align: left;"><code>2.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/StaPH-B/staphb_toolkit">Website</a></td>
<td style="text-align: left;">The StaPH-B ToolKit is a Python library of commonly used bioinformatics tools that help to inform public health action.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>star</code></td>
<td style="text-align: left;"><code>2.5.2</code><br><code>2.5.3a</code><br><code>2.6.1b</code><br><code>2.7.3a</code><br><code>2.7.9a</code><br><code>2.7.10b</code><br></td>
<td style="text-align: left;"><a href="https://github.com/alexdobin/STAR">Website</a></td>
<td style="text-align: left;">Spliced Transcripts Alignment to a Reference</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>star-fusion</code></td>
<td style="text-align: left;"><code>1.10.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/STAR-Fusion/STAR-Fusion/wiki">Website</a></td>
<td style="text-align: left;">STAR-Fusion further processes the output generated by the STAR aligner to map junction reads and spanning reads to a reference annotation set.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>stringtie</code></td>
<td style="text-align: left;"><code>2.1.4</code><br></td>
<td style="text-align: left;"><a href="https://ccb.jhu.edu/software/stringtie/index.shtml">Website</a></td>
<td style="text-align: left;">High-performance read alignment, quantification and mutation discovery, next-gen sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>subread</code></td>
<td style="text-align: left;"><code>2.0.1</code><br><code>2.0.6</code><br></td>
<td style="text-align: left;"><a href="https://sourceforge.net/projects/subread/">Website</a></td>
<td style="text-align: left;">High-performance read alignment, quantification and mutation discovery, next-gen sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>suite2p</code></td>
<td style="text-align: left;"><code>0.14.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/MouseLand/suite2p">Website</a></td>
<td style="text-align: left;">Pipeline for processing two-photon calcium imaging data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>svaba</code></td>
<td style="text-align: left;"><code>1.1.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/walaj/svaba">Website</a></td>
<td style="text-align: left;">SvABA - Structural variation and indel analysis by assembly.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>taxonkit</code></td>
<td style="text-align: left;"><code>0.14.2</code><br></td>
<td style="text-align: left;"><a href="https://bioinf.shenwei.me/taxonkit/">Website</a></td>
<td style="text-align: left;">A Practical and Efficient NCBI Taxonomy Toolkit also supports creating NCBI style taxdump files for custom taxonomies.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>tophat</code></td>
<td style="text-align: left;"><code>2.1.1</code><br></td>
<td style="text-align: left;"><a href="https://ccb.jhu.edu/software/tophat/index.shtml">Website</a></td>
<td style="text-align: left;">TopHat is a fast splice junction mapper for RNA-Seq reads.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>trf</code></td>
<td style="text-align: left;"><code>4.09</code><br></td>
<td style="text-align: left;"><a href="https://github.com/Benson-Genomics-Lab/TRF">Website</a></td>
<td style="text-align: left;">Tandem Repeats Finder is a program to locate and display tandem repeats in DNA sequences.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>trimgalore</code></td>
<td style="text-align: left;"><code>0.6.6</code><br></td>
<td style="text-align: left;"><a href="https://github.com/FelixKrueger/TrimGalore">Website</a></td>
<td style="text-align: left;">Trim Galore is a perl wrapper around Cutadapt and FastQC to consistently apply adapter and quality trimming to FastQ files, with extra functionality for RRBS data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>trimmomatic</code></td>
<td style="text-align: left;"><code>0.38</code><br></td>
<td style="text-align: left;"><a href="https://github.com/usadellab/Trimmomatic">Website</a></td>
<td style="text-align: left;">A flexible read trimming tool for Illumina NGS data</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>ucsctools</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="http://hgdownload.soe.ucsc.edu/admin/exe/">Website</a></td>
<td style="text-align: left;">Tools from the UCSC browser.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>umi_tools</code></td>
<td style="text-align: left;"><code>1.1.6</code><br></td>
<td style="text-align: left;"><a href="https://github.com/CGATOxford/UMI-tools">Website</a></td>
<td style="text-align: left;">Tools for dealing with Unique Molecular Identifiers</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>varscan</code></td>
<td style="text-align: left;"><code>2.4.0</code><br><code>2.4.1</code><br><code>2.4.2</code><br><code>2.4.3</code><br><code>2.4.4</code><br></td>
<td style="text-align: left;"><a href="http://dkoboldt.github.io/varscan/">Website</a></td>
<td style="text-align: left;">VarScan is a platform-independent mutation caller for targeted, exome, and whole-genome resequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>varseq</code></td>
<td style="text-align: left;"><code>2.6.1</code><br></td>
<td style="text-align: left;"><a href="https://www.goldenhelix.com/products/GenomeBrowse/">Website</a></td>
<td style="text-align: left;">The free Golden Helix GenomeBrowse tool delivers stunning visualizations of your genomic data that give you the power to see what is occurring at each base pair in your samples.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>vcf-split</code></td>
<td style="text-align: left;"><code>0.1.5</code><br></td>
<td style="text-align: left;"><a href="https://github.com/auerlab/biolibc">Website</a></td>
<td style="text-align: left;">vcf-split splits a combined-sample VCF stream into single-sample VCF files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>vcftools</code></td>
<td style="text-align: left;"><code>0.1.16</code><br></td>
<td style="text-align: left;"><a href="https://vcftools.github.io/">Website</a></td>
<td style="text-align: left;">A set of tools written in Perl and C++ for working with VCF files, such as those generated by the 1000 Genomes Project.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>velocyto</code></td>
<td style="text-align: left;"><code>0.17.17</code><br></td>
<td style="text-align: left;"><a href="http://velocyto.org/">Website</a></td>
<td style="text-align: left;">Velocyto is a library for the analysis of RNA velocity.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>vicaller</code></td>
<td style="text-align: left;"><code>1.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/xunchen85/VIcaller">Website</a></td>
<td style="text-align: left;">Viral Integration caller (VIcaller) is a bioinformatics tool designed for identifying viral integration events using high-throughput sequencing (HTS) data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>vifi</code></td>
<td style="text-align: left;"><code>072018</code><br></td>
<td style="text-align: left;"><a href="https://github.com/namphuon/ViFi">Website</a></td>
<td style="text-align: left;">ViFi is a tool for detecting viral integration and fusion mRNA sequences from Next Generation Sequencing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>virsorter</code></td>
<td style="text-align: left;"><code>2.2.4</code><br></td>
<td style="text-align: left;"><a href="https://github.com/simroux/VirSorter">Website</a></td>
<td style="text-align: left;">Mining viral signal from microbial genomic data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>genomics</strong></td>
<td style="text-align: left;"><code>virusseq</code></td>
<td style="text-align: left;"><code>072018</code><br></td>
<td style="text-align: left;"><a href="https://odin.mdacc.tmc.edu/~xsu1/VirusSeq.html">Website</a></td>
<td style="text-align: left;">A new algorithmic tool for detecting known viruses and their integration sites using next-generation sequencing of human cancer tissue.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>imaging</strong></td>
<td style="text-align: left;"><code>deeplabcut</code></td>
<td style="text-align: left;"><code>3.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/DeepLabCut">Website</a></td>
<td style="text-align: left;">Markerless pose estimation of user-defined features with deep learning for all animals including humans</td>
</tr>
<tr>
<td style="text-align: left;"><strong>imaging</strong></td>
<td style="text-align: left;"><code>mipav</code></td>
<td style="text-align: left;"><code>7.3.0</code><br><code>11.0.3</code><br></td>
<td style="text-align: left;"><a href="https://mipav.cit.nih.gov/">Website</a></td>
<td style="text-align: left;">The MIPAV (Medical Image Processing, Analysis, and Visualization) application enables quantitative analysis and visualization of medical images of numerous modalities such as PET, MRI, CT, or microscopy.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>imaging</strong></td>
<td style="text-align: left;"><code>mitograph</code></td>
<td style="text-align: left;"><code>3.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/vianamp/MitoGraph">Website</a></td>
<td style="text-align: left;">MitoGraph is a fully automated image processing method and software dedicated to calculating the three-dimensional morphology of mitochondria in live cells.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>imaging</strong></td>
<td style="text-align: left;"><code>monai</code></td>
<td style="text-align: left;"><code>1.3.2</code><br></td>
<td style="text-align: left;"><a href="https://monai.io/index.html">Website</a></td>
<td style="text-align: left;">MONAI is a set of open-source, freely availalbe collaborative frameworks built for accelerating research ad clinical collaboration in Medical Imaging.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>imaging</strong></td>
<td style="text-align: left;"><code>openslide</code></td>
<td style="text-align: left;"><code>3.4.1</code><br></td>
<td style="text-align: left;"><a href="https://openslide.org/">Website</a></td>
<td style="text-align: left;">OpenSlide is a C library that provides a simple interface to read whole-slide images (also known as virtual slides).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>imaging</strong></td>
<td style="text-align: left;"><code>seacr</code></td>
<td style="text-align: left;"><code>1.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/FredHutch/SEACR">Website</a></td>
<td style="text-align: left;">SEACR. Sparse Enrichment Analysis for CUT&amp;RUN</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>afni</code></td>
<td style="text-align: left;"><code>16.2.16</code><br><code>19.3.18</code><br><code>21.0.04</code><br><code>22.3.07</code><br><code>24.1.02</code><br><code>25.1.08</code><br></td>
<td style="text-align: left;"><a href="https://afni.nimh.nih.gov/">Website</a></td>
<td style="text-align: left;">AFNI (Analysis of Functional NeuroImages)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>ants</code></td>
<td style="text-align: left;"><code>2.2.0</code><br><code>2.4.0</code><br><code>2.5.2</code><br></td>
<td style="text-align: left;"><a href="http://stnava.github.io/ANTs/">Website</a></td>
<td style="text-align: left;">ANTs is a medical image registration and segmentation toolkit.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>biomedia</code></td>
<td style="text-align: left;"><code>1.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/BioMedIA/dhcp-structural-pipeline">Website</a></td>
<td style="text-align: left;">The dHCP structural pipeline performs structural analysis of neonatal brain MRI images (T1 and T2)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>camino</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="http://camino.cs.ucl.ac.uk/">Website</a></td>
<td style="text-align: left;">Camino is an open-source software toolkit for diffusion MRI processing.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>clara-train-sdk</code></td>
<td style="text-align: left;"><code>1.7.1</code><br></td>
<td style="text-align: left;"><a href="https://docs.nvidia.com/clara/clara-train-sdk/index.html">Website</a></td>
<td style="text-align: left;">NVIDIA Clara Train SDK provides a training framework to help accelerate deep learning training and inference for medical imaging use cases.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>dcm2nii</code></td>
<td style="text-align: left;"><code>122015</code><br></td>
<td style="text-align: left;"><a href="https://www.nitrc.org/projects/dcm2nii/">Website</a></td>
<td style="text-align: left;">dcm2nii is a popular tool for converting images from the complicated formats used by scanner manufacturers (DICOM, PAR/REC) to the simple NIfTI format used by many scientific tools.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>dcm2niix</code></td>
<td style="text-align: left;"><code>1.0.20210317</code><br><code>1.0.20240327</code><br></td>
<td style="text-align: left;"><a href="https://github.com/rordenlab/dcm2niix">Website</a></td>
<td style="text-align: left;">dcm2niix is designed to convert neuroimaging data from the DICOM format to the NIfTI format.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>dcmtk</code></td>
<td style="text-align: left;"><code>3.6.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/DCMTK/dcmtk">Website</a></td>
<td style="text-align: left;">This DICOM ToolKit (DCMTK) package consists of source code, documentation and installation instructions for a set of software libraries and applications implementing part of the DICOM/MEDICOM Standard.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>dke</code></td>
<td style="text-align: left;"><code>2.6</code><br></td>
<td style="text-align: left;"><a href="https://www.nitrc.org/projects/dke/">Website</a></td>
<td style="text-align: left;">Diffusional Kurtosis Estimator (DKE) is a software tool for post-processing diffusional kurtosis imaging (DKI) datasets that includes a suite of command-line programs along with a graphical user interface (GUI).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>dsistudio</code></td>
<td style="text-align: left;"><code>latest</code><br></td>
<td style="text-align: left;"><a href="https://dsi-studio.labsolver.org/">Website</a></td>
<td style="text-align: left;">DSI Studio is a tractography software tool that maps brain connections and correlates findings with neuropsychological disorders.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>fastsurfer</code></td>
<td style="text-align: left;"><code>2.4.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/Deep-MI/FastSurfer">Website</a></td>
<td style="text-align: left;">Segmentation and surface reconstruction of MRI images</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>fix</code></td>
<td style="text-align: left;"><code>1.06</code><br></td>
<td style="text-align: left;"><a href="https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FIX">Website</a></td>
<td style="text-align: left;">FIX attempts to auto-classify ICA components into good vs bad components, so that the bad components can be removed from the 4D FMRI data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>fmriprep</code></td>
<td style="text-align: left;"><code>20.2.3</code><br><code>21.0.2</code><br><code>22.0.1</code><br><code>22.0.2</code><br></td>
<td style="text-align: left;"><a href="https://fmriprep.org/en/stable/">Website</a></td>
<td style="text-align: left;">fMRIPrep is a functional magnetic resonance imaging (fMRI) data preprocessing pipeline that is designed to provide an easily accessible, state-of-the-art interface that is robust to variations in scan acquisition protocols and that requires minimal user input, while providing easily interpretable and comprehensive error and output reporting.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>freesurfer</code></td>
<td style="text-align: left;"><code>6.0.0</code><br><code>7.1.0</code><br><code>7.2.0</code><br><code>7.3.2</code><br><code>7.4.1</code><br><code>8.0.0-beta</code><br><code>8.0.0</code><br><code>8.1.0</code><br></td>
<td style="text-align: left;"><a href="https://surfer.nmr.mgh.harvard.edu/">Website</a></td>
<td style="text-align: left;">An open source neuroimaging toolkit for processing, analyzing, and visualizing human brain MR images.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>fsl</code></td>
<td style="text-align: left;"><code>5.0.11</code><br><code>6.0.4.singularity</code><br><code>6.0.4</code><br><code>6.0.7.11</code><br></td>
<td style="text-align: left;"><a href="https://fsl.fmrib.ox.ac.uk/fsl/fslwiki">Website</a></td>
<td style="text-align: left;">FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>fsleyes</code></td>
<td style="text-align: left;"><code>1.13.0</code><br></td>
<td style="text-align: left;"><a href="https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes">Website</a></td>
<td style="text-align: left;">FSLeyes (pronounced fossilise) is the new FSL image viewer for 3D and 4D data (replacing FSLView). It does not perform any processing or analysis of images - that is done by separate tools.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>gradunwarp</code></td>
<td style="text-align: left;"><code>1.2.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/Washington-University/gradunwarp">Website</a></td>
<td style="text-align: left;">gradunwarp is a Python/Numpy package used to unwarp the distorted volumes (due to the gradient field inhomogenities).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>hdbet</code></td>
<td style="text-align: left;"><code>2.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/MIC-DKFZ/HD-BET">Website</a></td>
<td style="text-align: left;">Brain Extraction Tool</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>infantfs</code></td>
<td style="text-align: left;"><code>20210109</code><br></td>
<td style="text-align: left;"><a href="https://surfer.nmr.mgh.harvard.edu/fswiki/infantFS">Website</a></td>
<td style="text-align: left;">An open source neuroimaging toolkit for processing, analyzing, and visualizing infant brain MR images.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>ismrmrd</code></td>
<td style="text-align: left;"><code>1.0.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/ismrmrd/siemens_to_ismrmrd">Website</a></td>
<td style="text-align: left;">The siemens_to_ismrmrd convertor is used to convert data from Siemens raw data format into ISMRMRD raw data format.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>mne</code></td>
<td style="text-align: left;"><code>0.22.0</code><br></td>
<td style="text-align: left;"><a href="https://mne.tools/stable/index.html">Website</a></td>
<td style="text-align: left;">Open-source Python package for exploring, visualizing, and analyzing human neurophysiological data; MEG, EEG, sEEG, ECoG, NIRS, and more.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>mricrogl</code></td>
<td style="text-align: left;"><code>1.0.20181111</code><br></td>
<td style="text-align: left;"><a href="https://github.com/rordenlab/MRIcroGL">Website</a></td>
<td style="text-align: left;">MRIcroGL is a cross-platform tool for viewing DICOM and NIfTI format images.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>mrtrix</code></td>
<td style="text-align: left;"><code>3.0.4</code><br></td>
<td style="text-align: left;"><a href="https://www.mrtrix.org/">Website</a></td>
<td style="text-align: left;">MRtrix3 provides a set of tools to perform various types of diffusion MRI analyses.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>mrtrix3tissue</code></td>
<td style="text-align: left;"><code>5.2.9</code><br></td>
<td style="text-align: left;"><a href="https://3tissue.github.io/">Website</a></td>
<td style="text-align: left;">MRtrix3Tissue is a fork of the MRtrix3 project. It aims to add capabilities for 3-Tissue CSD modelling and analysis to a complete version of the MRtrix3 software.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>pyalfe</code></td>
<td style="text-align: left;"><code>0.1.1</code><br></td>
<td style="text-align: left;"><a href="https://reghbali.github.io/pyalfe/intro.html">Website</a></td>
<td style="text-align: left;">PyALFE is an open-source python pipeline for Automated Lesion and Feature Extraction. It can process a clinical brain MRI study consisting of various sequences such as T1, T1 post contrast, FLAIR, T2, ADC, and CBF.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>pyradiomics</code></td>
<td style="text-align: left;"><code>3.1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/AIM-Harvard/pyradiomics">Website</a></td>
<td style="text-align: left;">Open-source python package for the extraction of Radiomics features from 2D and 3D images and binary masks.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>qsiprep</code></td>
<td style="text-align: left;"><code>1.0.2</code><br></td>
<td style="text-align: left;"><a href="https://qsiprep.readthedocs.io/en/1.0.1/">Website</a></td>
<td style="text-align: left;">QSIPrep configures pipelines for processing diffusion-weighted MRI (dMRI) data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>qsirecon</code></td>
<td style="text-align: left;"><code>1.1.1</code><br></td>
<td style="text-align: left;"><a href="https://qsirecon.readthedocs.io/en/1.1.0/">Website</a></td>
<td style="text-align: left;">QSIRecon builds post-processing workflows that produce many of the biologically-interesting dMRI derivatives used for hypothesis testing.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>simnibs</code></td>
<td style="text-align: left;"><code>4.1.0</code><br></td>
<td style="text-align: left;"><a href="https://simnibs.github.io/simnibs/build/html/index.html">Website</a></td>
<td style="text-align: left;">SimNIBS is a free and open source software package for the Simulation of Non-invasive Brain Stimulation.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>neuroimaging</strong></td>
<td style="text-align: left;"><code>workbench</code></td>
<td style="text-align: left;"><code>1.4.2</code><br><code>2.0.0</code><br></td>
<td style="text-align: left;"><a href="https://www.humanconnectome.org/software/connectome-workbench">Website</a></td>
<td style="text-align: left;">Connectome Workbench is an open source, freely available visualization and discovery tool used to map neuroimaging data</td>
</tr>
<tr>
<td style="text-align: left;"><strong>population genetics</strong></td>
<td style="text-align: left;"><code>eigensoft</code></td>
<td style="text-align: left;"><code>7.2.1</code><br><code>8.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/DReichLab/EIG">Website</a></td>
<td style="text-align: left;">The EIGENSOFT package combines functionality from our population genetics methods (Patterson et al. 2006) and our EIGENSTRAT stratification correction method (Price et al. 2006).</td>
</tr>
</tbody>
</table>
<h3 id="chemistry"><strong>chemistry</strong><a class="headerlink" href="#chemistry" title="Permanent link">#</a></h3>
<!-- markdownlint-disable MD045 MD056 MD058 -->
<table>
<thead>
<tr>
<th style="text-align: left;">Field</th>
<th style="text-align: left;">Module Name<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:110px"></a></th>
<th style="text-align: left;">Version(s)<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:90px"></a></th>
<th style="text-align: left;">URL</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><strong>computational chemistry</strong></td>
<td style="text-align: left;"><code>cyana</code></td>
<td style="text-align: left;"><code>3.98.15</code><br></td>
<td style="text-align: left;"><a href="https://cyana.org/wiki/Main_Page">Website</a></td>
<td style="text-align: left;">CYANA is a program for automated structure calculation of biological macromolecules on the basis of conformational constraints from NMR.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational chemistry</strong></td>
<td style="text-align: left;"><code>gamess</code></td>
<td style="text-align: left;"><code>2023R2</code><br></td>
<td style="text-align: left;"><a href="https://www.msg.chem.iastate.edu/index.html">Website</a></td>
<td style="text-align: left;">The General Atomic and Molecular Electronic Structure System is a general ab initio quantum chemistry package.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational chemistry</strong></td>
<td style="text-align: left;"><code>garant</code></td>
<td style="text-align: left;"><code>2.2</code><br></td>
<td style="text-align: left;"><a href="http://www.bpc.uni-frankfurt.de/guentert/wiki/index.php/GARANT">Website</a></td>
<td style="text-align: left;">GARANT is a free program that automatically assigns resonances in nuclear magnetic resonance spectra of proteins.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational chemistry</strong></td>
<td style="text-align: left;"><code>libcifpp</code></td>
<td style="text-align: left;"><code>7.0.9</code><br></td>
<td style="text-align: left;"><a href="https://github.com/PDB-REDO/libcifpp">Website</a></td>
<td style="text-align: left;">Library containing code to manipulate mmCIF and PDB files</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational chemistry</strong></td>
<td style="text-align: left;"><code>schrodinger</code></td>
<td style="text-align: left;"><code>2024-4</code><br><code>2025-1</code><br><code>2025-2</code><br><code>2025-3</code><br><code>2025-4</code><br></td>
<td style="text-align: left;"><a href="https://www.schrodinger.com/platform/drug-discovery">Website</a></td>
<td style="text-align: left;">Small Molecule Drug Discovery Suite</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational chemistry</strong></td>
<td style="text-align: left;"><code>xeasy</code></td>
<td style="text-align: left;"><code>1.2</code><br></td>
<td style="text-align: left;"><a href="http://www.bpc.uni-frankfurt.de/guentert/wiki/index.php/XEASY">Website</a></td>
<td style="text-align: left;">Interactive computer support of the analysis of NMR spectra for three-dimensional structure determination of biological macromolecules.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>electrostatics</strong></td>
<td style="text-align: left;"><code>apbs</code></td>
<td style="text-align: left;"><code>1.5</code><br></td>
<td style="text-align: left;"><a href="https://www.poissonboltzmann.org/">Website</a></td>
<td style="text-align: left;">APBS (Adaptive Poisson-Boltzmann Solver) solves the equations of continuum electrostatics for large biomolecular assemblages.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>electrostatics</strong></td>
<td style="text-align: left;"><code>pdb2pqr</code></td>
<td style="text-align: left;"><code>2.1.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/Electrostatics/pdb2pqr">Website</a></td>
<td style="text-align: left;">PDB2PQR - determining titration states, adding missing atoms, and assigning charges/radii to biomolecules.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>ambertools</code></td>
<td style="text-align: left;"><code>24</code><br></td>
<td style="text-align: left;"><a href="https://ambermd.org/AmberTools.php">Website</a></td>
<td style="text-align: left;">A suite that can be used to carry out complete molecular dynamics simulations.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>brer</code></td>
<td style="text-align: left;"><code>0.0.7</code><br></td>
<td style="text-align: left;"><a href="https://github.com/kassonlab/singularity-brer">Website</a></td>
<td style="text-align: left;">Python package for running bias-resampling ensemble refinement (BRER) simulations</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>catdcd</code></td>
<td style="text-align: left;"><code>4.0</code><br></td>
<td style="text-align: left;"><a href="https://www.ks.uiuc.edu/Development/MDTools/catdcd/">Website</a></td>
<td style="text-align: left;">Catdcd functions much like the Unix cat command; it concatenates DCD files into a single DCD file.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>gmx_mmpbsa</code></td>
<td style="text-align: left;"><code>1.6.4</code><br></td>
<td style="text-align: left;"><a href="https://valdes-tresanco-ms.github.io/gmx_MMPBSA/v1.6.3/">Website</a></td>
<td style="text-align: left;">gmx_MMPBSA is a new tool based on AMBER's MMPBSA.py aiming to perform end-state free energy calculations with GROMACS files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>gromacs</code></td>
<td style="text-align: left;"><code>2024.2</code><br><code>2019.6</code><br><code>2021.1</code><br><code>2021.2</code><br><code>2022.5-test</code><br><code>2022.5</code><br><code>2023.0</code><br><code>2025.2</code><br><code>2025.3</code><br></td>
<td style="text-align: left;"><a href="https://www.gromacs.org/">Website</a></td>
<td style="text-align: left;">A free and open-source software suite for high-performance molecular dynamics and output analysis.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>gromacs-ramd</code></td>
<td style="text-align: left;"><code>2020.5</code><br></td>
<td style="text-align: left;"><a href="https://github.com/HITS-MCM/gromacs-ramd">Website</a></td>
<td style="text-align: left;">Random Acceleration Molecular Dynamics (RAMD)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>mdanalysis</code></td>
<td style="text-align: left;"><code>1.0.0</code><br><code>2.8.0</code><br></td>
<td style="text-align: left;"><a href="https://www.mdanalysis.org/">Website</a></td>
<td style="text-align: left;">MDAnalysis is an object-oriented Python library to analyze trajectories from molecular dynamics (MD) simulations in many popular formats.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>mdtraj</code></td>
<td style="text-align: left;"><code>1.10.2</code><br></td>
<td style="text-align: left;"><a href="https://mdtraj.org/1.9.3/index.html">Website</a></td>
<td style="text-align: left;">MDTraj is a python library that allows users to manipulate molecular dynamics (MD) trajectories.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>namd2</code></td>
<td style="text-align: left;"><code>2.13</code><br><code>2.14-cuda-mpi</code><br><code>2.14-mpi</code><br><code>2.14</code><br></td>
<td style="text-align: left;"><a href="https://www.ks.uiuc.edu/Research/namd/">Website</a></td>
<td style="text-align: left;">NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>namd3</code></td>
<td style="text-align: left;"><code>3.0.1-cuda</code><br><code>3.0.1</code><br></td>
<td style="text-align: left;"><a href="https://www.ks.uiuc.edu/Research/namd/">Website</a></td>
<td style="text-align: left;">NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular dynamics</strong></td>
<td style="text-align: left;"><code>plumed</code></td>
<td style="text-align: left;"><code>2.7.1</code><br><code>2.8.2-test</code><br><code>2.8.2</code><br><code>2.9.0</code><br><code>2.9.1</code><br><code>2.10.0-OMPI-4.1.8</code><br><code>2.10.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/plumed/plumed2">Website</a></td>
<td style="text-align: left;">The community-developed PLUgin for MolEcular Dynamics</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular modelling</strong></td>
<td style="text-align: left;"><code>charm</code></td>
<td style="text-align: left;"><code>50</code><br></td>
<td style="text-align: left;"><a href="https://academiccharmm.org/">Website</a></td>
<td style="text-align: left;">A molecular simulation program with broad application to many-particle systems with a comprehensive set of energy functions, a variety of enhanced sampling methods, and support for multi-scale techniques including QM/MM, MM/CG, and a range of implicit solvent models.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular modelling</strong></td>
<td style="text-align: left;"><code>chimera</code></td>
<td style="text-align: left;"><code>1.15</code><br></td>
<td style="text-align: left;"><a href="https://www.cgl.ucsf.edu/chimera/">Website</a></td>
<td style="text-align: left;">UCSF Chimera is a program for the interactive visualization and analysis of molecular structures and related data, including density maps, trajectories, and sequence alignments.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular modelling</strong></td>
<td style="text-align: left;"><code>chimerax</code></td>
<td style="text-align: left;"><code>1.8</code><br></td>
<td style="text-align: left;"><a href="https://www.rbvi.ucsf.edu/chimerax/">Website</a></td>
<td style="text-align: left;">ChimeraX is a visualization program for molecular structures and related data, including density maps, trajectories, and sequence alignments.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular modelling</strong></td>
<td style="text-align: left;"><code>coot</code></td>
<td style="text-align: left;"><code>0.9.8.92</code><br></td>
<td style="text-align: left;"><a href="https://www2.mrc-lmb.cam.ac.uk/personal/pemsley/coot/">Website</a></td>
<td style="text-align: left;">Coot is for macromolecular model building, model completion and validation, particularly suitable for protein modelling using X-ray data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular modelling</strong></td>
<td style="text-align: left;"><code>openmm</code></td>
<td style="text-align: left;"><code>8.2.0</code><br></td>
<td style="text-align: left;"><a href="https://openmm.org">Website</a></td>
<td style="text-align: left;">A high-performance toolkit for molecular simulation. Use it as an application, a library, or a flexible programming environment.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular visualization</strong></td>
<td style="text-align: left;"><code>pymol</code></td>
<td style="text-align: left;"><code>3.0.0</code><br><code>3.1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/schrodinger/pymol-open-source">Website</a></td>
<td style="text-align: left;">PyMOL is a user-sponsored molecular visualization system on an open-source foundation, maintained and distributed by Schr√∂dinger.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>molecular visualization</strong></td>
<td style="text-align: left;"><code>vmd</code></td>
<td style="text-align: left;"><code>1.9.3</code><br><code>2.0.0a4</code><br></td>
<td style="text-align: left;"><a href="https://www.ks.uiuc.edu/Research/vmd/">Website</a></td>
<td style="text-align: left;">VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>tools</strong></td>
<td style="text-align: left;"><code>openbabel</code></td>
<td style="text-align: left;"><code>2.4.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/openbabel/openbabel">Website</a></td>
<td style="text-align: left;">Open Babel is a chemical toolbox designed to speak the many languages of chemical data.</td>
</tr>
</tbody>
</table>
<h3 id="devel"><strong>devel</strong><a class="headerlink" href="#devel" title="Permanent link">#</a></h3>
<!-- markdownlint-disable MD045 MD056 MD058 -->
<table>
<thead>
<tr>
<th style="text-align: left;">Field</th>
<th style="text-align: left;">Module Name<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:110px"></a></th>
<th style="text-align: left;">Version(s)<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:90px"></a></th>
<th style="text-align: left;">URL</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><strong>compiler</strong></td>
<td style="text-align: left;"><code>binutils</code></td>
<td style="text-align: left;"><code>2.37</code><br></td>
<td style="text-align: left;"><a href="https://www.gnu.org/software/binutils/">Website</a></td>
<td style="text-align: left;">The GNU Binary Utilities, or binutils, are a set of programming tools for creating and managing binary programs, object files, libraries, profile data, and assembly source code.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>compiler</strong></td>
<td style="text-align: left;"><code>gcc</code></td>
<td style="text-align: left;"><code>9.3.0</code><br><code>9.5.0</code><br><code>11.3.0</code><br><code>13.3.0</code><br></td>
<td style="text-align: left;"><a href="http://gcc.gnu.org">Website</a></td>
<td style="text-align: left;">The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Ada, Go, and D, as well as libraries for these languages.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>compiler</strong></td>
<td style="text-align: left;"><code>intel</code></td>
<td style="text-align: left;"><code>2021.1.1</code><br></td>
<td style="text-align: left;"><a href="http://software.intel.com/en-us/articles/intel-compilers">Website</a></td>
<td style="text-align: left;">Intel Compiler Family (C/C++/Fortran for x86_64)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>compiler</strong></td>
<td style="text-align: left;"><code>intel-oneapi</code></td>
<td style="text-align: left;"><code>2025.1.0</code><br></td>
<td style="text-align: left;"><a href="http://software.intel.com/en-us/articles/intel-compilers">Website</a></td>
<td style="text-align: left;">Intel Compiler Family (C/C++/Fortran for x86_64)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>data</strong></td>
<td style="text-align: left;"><code>netcdf/gcc/64</code></td>
<td style="text-align: left;"><code>4.7.4</code><br></td>
<td style="text-align: left;"><a href="https://www.unidata.ucar.edu/software/netcdf/">Website</a></td>
<td style="text-align: left;">NetCDF (Network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>data</strong></td>
<td style="text-align: left;"><code>netcdf/intel/64</code></td>
<td style="text-align: left;"><code>4.7.4</code><br></td>
<td style="text-align: left;"><a href="https://www.unidata.ucar.edu/software/netcdf/">Website</a></td>
<td style="text-align: left;">NetCDF (Network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>data</strong></td>
<td style="text-align: left;"><code>pnetcdf</code></td>
<td style="text-align: left;"><code>1.12.1</code><br></td>
<td style="text-align: left;"><a href="https://trac.mcs.anl.gov/projects/parallel-netcdf/">Website</a></td>
<td style="text-align: left;">PnetCDF is a high-performance parallel I/O library for accessing files in format compatibility</td>
</tr>
<tr>
<td style="text-align: left;"><strong>debugger</strong></td>
<td style="text-align: left;"><code>gdb</code></td>
<td style="text-align: left;"><code>gcc/64/12.1</code><br></td>
<td style="text-align: left;"><a href="https://www.sourceware.org/gdb/">Website</a></td>
<td style="text-align: left;">GDB, the GNU Project debugger, allows you to see what is going on inside another program while it executes -- or what another program was doing at the moment it crashed.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>debugger</strong></td>
<td style="text-align: left;"><code>valgrind</code></td>
<td style="text-align: left;"><code>3.16.1</code><br></td>
<td style="text-align: left;"><a href="http://valgrind.org">Website</a></td>
<td style="text-align: left;">Dynamic memory testing and debugging tools</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>cuda</code></td>
<td style="text-align: left;"><code>7.5</code><br><code>8.0</code><br><code>9.1</code><br><code>11.0.2</code><br><code>11.1.1</code><br><code>11.2.2</code><br><code>11.5.0</code><br><code>11.7.0</code><br><code>11.8.0</code><br><code>12.3</code><br><code>13.0.2</code><br></td>
<td style="text-align: left;"><a href="http://www.nvidia.com/cuda">Website</a></td>
<td style="text-align: left;">NVIDIA CUDA Toolkit for Linux</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>go</code></td>
<td style="text-align: left;"><code>1.15.6</code><br><code>1.22.2</code><br><code>1.24.5</code><br></td>
<td style="text-align: left;"><a href="https://golang.org/">Website</a></td>
<td style="text-align: left;">open source programming language</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>jdk</code></td>
<td style="text-align: left;"><code>11.0.11</code><br><code>17.0.8</code><br><code>22.0.2</code><br></td>
<td style="text-align: left;"><a href="https://www.oracle.com/java/technologies/">Website</a></td>
<td style="text-align: left;">Java is a high-level, class-based, object-oriented programming language that is designed to have as few implementation dependencies as possible.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>julia</code></td>
<td style="text-align: left;"><code>1.10.2</code><br><code>1.10.4</code><br></td>
<td style="text-align: left;"><a href="https://julialang.org/">Website</a></td>
<td style="text-align: left;">The Julia programming language is a flexible dynamic language, appropriate for scientific and numerical computing, with performance comparable to traditional statically-typed languages.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>lua</code></td>
<td style="text-align: left;"><code>5.1.4.9</code><br><code>5.4.3</code><br></td>
<td style="text-align: left;"><a href="https://www.lua.org/">Website</a></td>
<td style="text-align: left;">Lua is a powerful, efficient, lightweight, embeddable scripting language. It supports procedural programming, object-oriented programming, functional programming, data-driven programming, and data description.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>openjdk</code></td>
<td style="text-align: left;"><code>16.0.1</code><br><code>17.0.5.8</code><br><code>19.36.2238</code><br><code>21.0.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/AdoptOpenJDK">Website</a></td>
<td style="text-align: left;">Java‚Ñ¢ is the world's leading programming language and platform. AdoptOpenJDK uses infrastructure, build and test scripts to produce prebuilt binaries from OpenJDK‚Ñ¢ class libraries and a choice of either OpenJDK or the Eclipse OpenJ9 VM..</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>perl</code></td>
<td style="text-align: left;"><code>5.26.1</code><br><code>5.42.0</code><br></td>
<td style="text-align: left;"><a href="https://www.perl.org/">Website</a></td>
<td style="text-align: left;">Perl is a highly capable, feature-rich programming language with over 30 years of development.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>python</code></td>
<td style="text-align: left;"><code>3.9.1</code><br><code>intel-2021.1</code><br><code>2.7.18</code><br><code>3.10.16</code><br><code>3.12.10</code><br></td>
<td style="text-align: left;"><a href="http://www.python.org/">Website</a></td>
<td style="text-align: left;">scientific scripting package</td>
</tr>
<tr>
<td style="text-align: left;"><strong>language</strong></td>
<td style="text-align: left;"><code>ruby</code></td>
<td style="text-align: left;"><code>3.4.3</code><br></td>
<td style="text-align: left;"><a href="https://www.ruby-lang.org/en/">Website</a></td>
<td style="text-align: left;">A dynamic, open source programming language with a focus on simplicity and productivity.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>boost</code></td>
<td style="text-align: left;"><code>openmpi/gcc/1.75.0</code><br><code>impi/intel/1.75.0</code><br></td>
<td style="text-align: left;"><a href="http://www.boost.org">Website</a></td>
<td style="text-align: left;">Boost provides free peer-reviewed portable C++ source libraries.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>hdf5/impi/intel/64</code></td>
<td style="text-align: left;"><code>1.12.0</code><br></td>
<td style="text-align: left;"><a href="http://www.hdfgroup.org/HDF5/">Website</a></td>
<td style="text-align: left;">General purpose library and file format for storing scientific data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>hdf5/openmpi/gcc/64</code></td>
<td style="text-align: left;"><code>1.12.0</code><br></td>
<td style="text-align: left;"><a href="http://www.hdfgroup.org/HDF5/">Website</a></td>
<td style="text-align: left;">General purpose library and file format for storing scientific data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>nccl</code></td>
<td style="text-align: left;"><code>2.8.4-11.1.1</code><br></td>
<td style="text-align: left;"><a href="https://developer.nvidia.com/nccl">Website</a></td>
<td style="text-align: left;">The NVIDIA Collective Communication Library (NCCL) implements multi-GPU and multi-node communication primitives optimized for NVIDIA GPUs and Networking.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>numba</code></td>
<td style="text-align: left;"><code>0.60.0</code><br></td>
<td style="text-align: left;"><a href="https://numba.pydata.org/">Website</a></td>
<td style="text-align: left;">Numba is an open source JIT compiler that translates a subset of Python and NumPy code into fast machine code.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>opencv</code></td>
<td style="text-align: left;"><code>4.5.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/opencv/opencv">Website</a></td>
<td style="text-align: left;">Open Source Computer Vision Library</td>
</tr>
<tr>
<td style="text-align: left;"><strong>mpi</strong></td>
<td style="text-align: left;"><code>impi</code></td>
<td style="text-align: left;"><code>2021.1.1</code><br><code>2021.15</code><br></td>
<td style="text-align: left;"><a href="http://software.intel.com/en-us/articles/intel-mpi-library">Website</a></td>
<td style="text-align: left;">Intel MPI Library (C/C++/Fortran for x86_64)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>mpi</strong></td>
<td style="text-align: left;"><code>openmpi</code></td>
<td style="text-align: left;"><code>gcc/64/4.1.0</code><br><code>gcc/64/4.1.8</code><br></td>
<td style="text-align: left;"><a href="https://www.open-mpi.org">Website</a></td>
<td style="text-align: left;">OpenMPI Library (C/C++/Fortran for x86_64)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>networking</strong></td>
<td style="text-align: left;"><code>pmix</code></td>
<td style="text-align: left;"><code>3.1.6</code><br><code>3.1.6-OMPI-4.1.8</code><br><code>5.0.2</code><br></td>
<td style="text-align: left;"><a href="https://github.com/openpmix/openpmix">Website</a></td>
<td style="text-align: left;">Fabric communication services</td>
</tr>
<tr>
<td style="text-align: left;"><strong>networking</strong></td>
<td style="text-align: left;"><code>ucx</code></td>
<td style="text-align: left;"><code>1.9.0</code><br><code>1.14.1</code><br><code>1.19.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/openucx/ucx">Website</a></td>
<td style="text-align: left;">Fabric communication services</td>
</tr>
<tr>
<td style="text-align: left;"><strong>profiling</strong></td>
<td style="text-align: left;"><code>intel-advisor</code></td>
<td style="text-align: left;"><code>2021.1.1</code><br></td>
<td style="text-align: left;"><a href="https://software.intel.com/en-us/intel-advisor-xe">Website</a></td>
<td style="text-align: left;">Intel Advisor</td>
</tr>
<tr>
<td style="text-align: left;"><strong>text editor</strong></td>
<td style="text-align: left;"><code>emacs</code></td>
<td style="text-align: left;"><code>26.3</code><br><code>27.1</code><br></td>
<td style="text-align: left;"><a href="https://www.gnu.org/software/emacs/">Website</a></td>
<td style="text-align: left;">An extensible, customizable, free/libre text editor ‚Äî and more.</td>
</tr>
</tbody>
</table>
<h3 id="math"><strong>math</strong><a class="headerlink" href="#math" title="Permanent link">#</a></h3>
<!-- markdownlint-disable MD045 MD056 MD058 -->
<table>
<thead>
<tr>
<th style="text-align: left;">Field</th>
<th style="text-align: left;">Module Name<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:110px"></a></th>
<th style="text-align: left;">Version(s)<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:90px"></a></th>
<th style="text-align: left;">URL</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><strong>analytics</strong></td>
<td style="text-align: left;"><code>arrow</code></td>
<td style="text-align: left;"><code>4.0.0</code><br></td>
<td style="text-align: left;"><a href="https://arrow.apache.org/">Website</a></td>
<td style="text-align: left;">Apache Arrow defines a language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware like CPUs and GPUs.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>computational fluid dynamics</strong></td>
<td style="text-align: left;"><code>ansys</code></td>
<td style="text-align: left;"><code>2021R1</code><br><code>2022R1</code><br><code>2024R1</code><br><code>2024R2</code><br></td>
<td style="text-align: left;"><a href="https://www.ansys.com/">Website</a></td>
<td style="text-align: left;">CAE/multiphysics engineering simulation software for product design, testing and operation</td>
</tr>
<tr>
<td style="text-align: left;"><strong>data analytics</strong></td>
<td style="text-align: left;"><code>spark</code></td>
<td style="text-align: left;"><code>hadoop3.2/3.1.2</code><br></td>
<td style="text-align: left;"><a href="https://spark.apache.org/">Website</a></td>
<td style="text-align: left;">Apache Spark‚Ñ¢ is a unified analytics engine for large-scale data processing</td>
</tr>
<tr>
<td style="text-align: left;"><strong>deep learning</strong></td>
<td style="text-align: left;"><code>aif360</code></td>
<td style="text-align: left;"><code>0.6.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/Trusted-AI/AIF360">Website</a></td>
<td style="text-align: left;">The AI Fairness 360 toolkit is an extensible open-source library containing techniques developed by the research community to help detect and mitigate bias in machine learning models throughout the AI application lifecycle.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>deep learning</strong></td>
<td style="text-align: left;"><code>cudnn</code></td>
<td style="text-align: left;"><code>8.0.5-11.0.2</code><br><code>8.0.5-11.1.1</code><br><code>8.1.0-11.2.2</code><br><code>8.3.0-11.5.0</code><br><code>8.6.0-11.x</code><br><code>8.9.3-11.x</code><br><code>8.9.7-12.x</code><br><code>9.1.1-12.x</code><br><code>9.1.7-13.x</code><br></td>
<td style="text-align: left;"><a href="https://developer.nvidia.com/cudnn">Website</a></td>
<td style="text-align: left;">GPU-accelerated library of primitives for deep neural networks.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>deep learning</strong></td>
<td style="text-align: left;"><code>libtensorflowcc</code></td>
<td style="text-align: left;"><code>1.14.0</code><br></td>
<td style="text-align: left;"><a href="https://www.tensorflow.org/">Website</a></td>
<td style="text-align: left;">TensorFlow is an end-to-end open source platform for machine learning.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>deep learning</strong></td>
<td style="text-align: left;"><code>mxnet</code></td>
<td style="text-align: left;"><code>1.8.0</code><br></td>
<td style="text-align: left;"><a href="https://mxnet.apache.org/">Website</a></td>
<td style="text-align: left;">A truly open source deep learning framework suited for flexible research prototyping and production.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>deep learning</strong></td>
<td style="text-align: left;"><code>pytorch</code></td>
<td style="text-align: left;"><code>2.5.1</code><br><code>2.6.0</code><br></td>
<td style="text-align: left;"><a href="https://pytorch.org/">Website</a></td>
<td style="text-align: left;">PyTorch - An open source machine learning framework that accelerates the path from research prototyping to production deployment.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>deep learning</strong></td>
<td style="text-align: left;"><code>tensorflow</code></td>
<td style="text-align: left;"><code>2.16.1</code><br><code>2.18.0</code><br></td>
<td style="text-align: left;"><a href="https://www.tensorflow.org/">Website</a></td>
<td style="text-align: left;">TensorFlow is an end-to-end open source platform for machine learning.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>finite element analysis</strong></td>
<td style="text-align: left;"><code>febiostudio</code></td>
<td style="text-align: left;"><code>2.7</code><br></td>
<td style="text-align: left;"><a href="https://febio.org/">Website</a></td>
<td style="text-align: left;">FEBio is a software tool for nonlinear finite element analysis in biomechanics and biophysics and is specifically focused on solving nonlinear large deformation problems in biomechanics and biophysics.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>finite element analysis</strong></td>
<td style="text-align: left;"><code>lsdyna</code></td>
<td style="text-align: left;"><code>8.0.0</code><br><code>8.1.0</code><br><code>9.0.1</code><br><code>9.3.1</code><br><code>10.2.0</code><br><code>11.2.0</code><br><code>12.0.0</code><br><code>13.0.0</code><br><code>13.1.1</code><br><code>14.1.0</code><br><code>14.2.0</code><br></td>
<td style="text-align: left;"><a href="https://www.lstc.com/products/ls-dyna">Website</a></td>
<td style="text-align: left;">Advanced general-purpose multiphysics simulation software.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>finite element analysis</strong></td>
<td style="text-align: left;"><code>lsprepost</code></td>
<td style="text-align: left;"><code>4.8.17</code><br></td>
<td style="text-align: left;"><a href="https://www.lstc.com/products/ls-dyna">Website</a></td>
<td style="text-align: left;">finite element analysis</td>
</tr>
<tr>
<td style="text-align: left;"><strong>gis</strong></td>
<td style="text-align: left;"><code>gdal</code></td>
<td style="text-align: left;"><code>2.4.4</code><br><code>3.8.5</code><br></td>
<td style="text-align: left;"><a href="https://gdal.org/">Website</a></td>
<td style="text-align: left;">GDAL is a translator library for raster and vector geospatial data formats</td>
</tr>
<tr>
<td style="text-align: left;"><strong>gis</strong></td>
<td style="text-align: left;"><code>geos</code></td>
<td style="text-align: left;"><code>3.9.1</code><br><code>3.13.0</code><br></td>
<td style="text-align: left;"><a href="https://libgeos.org/">Website</a></td>
<td style="text-align: left;">GEOS is a C/C++ library for computational geometry with a focus on algorithms used in geographic information systems (GIS) software.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>gis</strong></td>
<td style="text-align: left;"><code>proj</code></td>
<td style="text-align: left;"><code>5.1.0</code><br><code>9.4.0</code><br></td>
<td style="text-align: left;"><a href="https://proj.org/">Website</a></td>
<td style="text-align: left;">PROJ is a generic coordinate transformation software that transforms geospatial coordinates from one coordinate reference system (CRS) to another.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>gis</strong></td>
<td style="text-align: left;"><code>udunits</code></td>
<td style="text-align: left;"><code>2.2.28</code><br></td>
<td style="text-align: left;"><a href="https://www.unidata.ucar.edu/software/udunits/">Website</a></td>
<td style="text-align: left;">The UDUNITS-2 package provides support for units of physical quantities.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>fftw/impi/intel/64</code></td>
<td style="text-align: left;"><code>2.1.5</code><br><code>3.3.9</code><br></td>
<td style="text-align: left;"><a href="http://www.fftw.org">Website</a></td>
<td style="text-align: left;">Numerical library, contains discrete Fourier transformation</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>fftw/openmpi/gcc/64</code></td>
<td style="text-align: left;"><code>3.3.9</code><br></td>
<td style="text-align: left;"><a href="http://www.fftw.org">Website</a></td>
<td style="text-align: left;">Numerical library, contains discrete Fourier transformation</td>
</tr>
<tr>
<td style="text-align: left;"><strong>machine learning</strong></td>
<td style="text-align: left;"><code>ollama</code></td>
<td style="text-align: left;"><code>0.3.14</code><br></td>
<td style="text-align: left;"><a href="https://ollama.com/">Website</a></td>
<td style="text-align: left;">Get up and running with large language models.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical analysis</strong></td>
<td style="text-align: left;"><code>mcr</code></td>
<td style="text-align: left;"><code>v81</code><br><code>v83</code><br><code>v84</code><br><code>v93</code><br><code>v96</code><br><code>v97</code><br><code>v99</code><br><code>v713</code><br><code>v717</code><br><code>v914</code><br></td>
<td style="text-align: left;"><a href="https://www.mathworks.com/products/compiler/matlab-runtime.html">Website</a></td>
<td style="text-align: left;">The MATLAB Runtime is a standalone set of shared libraries that enables the execution of compiled MATLAB, Simulink applications, or components.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical analysis</strong></td>
<td style="text-align: left;"><code>mosek</code></td>
<td style="text-align: left;"><code>8</code><br></td>
<td style="text-align: left;"><a href="https://www.mosek.com/">Website</a></td>
<td style="text-align: left;">MOSEK solves all your LPs, QPs, SOCPs, SDPs and MIPs. Includes interfaces to C, C++, Java, MATLAB, .NET, Python and R.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical analysis</strong></td>
<td style="text-align: left;"><code>mplus</code></td>
<td style="text-align: left;"><code>8.3</code><br></td>
<td style="text-align: left;"><a href="https://www.statmodel.com/">Website</a></td>
<td style="text-align: left;">Mplus is a latent variable modeling program</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical library</strong></td>
<td style="text-align: left;"><code>blas</code></td>
<td style="text-align: left;"><code>3.9.0</code><br></td>
<td style="text-align: left;"><a href="http://www.netlib.org/blas/">Website</a></td>
<td style="text-align: left;">The BLAS (Basic Linear Algebra Subprograms) are routines that provide standard building blocks for performing basic vector and matrix operations.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical library</strong></td>
<td style="text-align: left;"><code>glpk</code></td>
<td style="text-align: left;"><code>4.65</code><br></td>
<td style="text-align: left;"><a href="https://www.gnu.org/software/glpk/">Website</a></td>
<td style="text-align: left;">The GLPK (GNU Linear Programming Kit) package is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical library</strong></td>
<td style="text-align: left;"><code>gsl</code></td>
<td style="text-align: left;"><code>2.2</code><br><code>2.6</code><br><code>2.7</code><br></td>
<td style="text-align: left;"><a href="https://www.gnu.org/software/gsl/">Website</a></td>
<td style="text-align: left;">The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical library</strong></td>
<td style="text-align: left;"><code>lapack</code></td>
<td style="text-align: left;"><code>3.9.0</code><br><code>3.10.1</code><br></td>
<td style="text-align: left;"><a href="http://www.netlib.org/lapack/">Website</a></td>
<td style="text-align: left;">LAPACK (Linear Algebra Package) is a standard software library for numerical linear algebra.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical library</strong></td>
<td style="text-align: left;"><code>mkl</code></td>
<td style="text-align: left;"><code>2021.1.1</code><br></td>
<td style="text-align: left;"><a href="https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onemkl.html">Website</a></td>
<td style="text-align: left;">Intel oneAPI Math Kernel Library</td>
</tr>
<tr>
<td style="text-align: left;"><strong>numerical library</strong></td>
<td style="text-align: left;"><code>openblas</code></td>
<td style="text-align: left;"><code>0.3.13</code><br><code>0.3.27</code><br></td>
<td style="text-align: left;"><a href="https://github.com/xianyi/OpenBLAS">Website</a></td>
<td style="text-align: left;">OpenBLAS is an optimized BLAS (Basic Linear Algebra Subprograms) library based on GotoBLAS2 1.13 BSD version.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>optimization</strong></td>
<td style="text-align: left;"><code>nlopt</code></td>
<td style="text-align: left;"><code>2.7.1</code><br></td>
<td style="text-align: left;"><a href="https://nlopt.readthedocs.io/">Website</a></td>
<td style="text-align: left;">NLopt is a free/open-source library for nonlinear optimization.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>simulation</strong></td>
<td style="text-align: left;"><code>geant4</code></td>
<td style="text-align: left;"><code>10.4.2</code><br></td>
<td style="text-align: left;"><a href="https://geant4.web.cern.ch/">Website</a></td>
<td style="text-align: left;">Geant4 is a toolkit for the simulation of the passage of particles through matter.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>statistics</strong></td>
<td style="text-align: left;"><code>jags</code></td>
<td style="text-align: left;"><code>4.3.0</code><br><code>4.3.1</code><br><code>4.3.2</code><br></td>
<td style="text-align: left;"><a href="https://mcmc-jags.sourceforge.io/">Website</a></td>
<td style="text-align: left;">Just another Gibbs sampler (JAGS) is a program for simulation from Bayesian hierarchical models using Markov chain Monte Carlo (MCMC).</td>
</tr>
<tr>
<td style="text-align: left;"><strong>statistics</strong></td>
<td style="text-align: left;"><code>R</code></td>
<td style="text-align: left;"><code>4.0.4</code><br><code>4.1.1</code><br><code>4.2.1</code><br><code>4.3.1</code><br><code>4.3.3</code><br><code>4.4.0</code><br><code>4.4.2</code><br><code>4.5.0</code><br></td>
<td style="text-align: left;"><a href="https://www.r-project.org/">Website</a></td>
<td style="text-align: left;">statistical computing package</td>
</tr>
<tr>
<td style="text-align: left;"><strong>statistics</strong></td>
<td style="text-align: left;"><code>rjags</code></td>
<td style="text-align: left;"><code>4.6</code><br></td>
<td style="text-align: left;"><a href="https://mcmc-jags.sourceforge.io/">Website</a></td>
<td style="text-align: left;">JAGS is Just Another Gibbs Sampler.  It is a program for analysis of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  not wholly unlike BUGS.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>statistics</strong></td>
<td style="text-align: left;"><code>rstudio</code></td>
<td style="text-align: left;"><code>2022.02.1-461</code><br></td>
<td style="text-align: left;"><a href="https://www.rstudio.com/products/rstudio/">Website</a></td>
<td style="text-align: left;">RStudio is an integrated development environment (IDE) for R.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>statistics</strong></td>
<td style="text-align: left;"><code>rstudio-server</code></td>
<td style="text-align: left;"><code>2025.09.0-387</code><br><code>2025.09.2-418</code><br></td>
<td style="text-align: left;"><a href="https://posit.co/download/rstudio-server/">Website</a></td>
<td style="text-align: left;">RStudio Server enables you to provide a browser based interface to a version of R running on a remote Linux server, bringing the power and productivity of the RStudio IDE to server-based deployments of R.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>statistics</strong></td>
<td style="text-align: left;"><code>sas</code></td>
<td style="text-align: left;"><code>94-15.3</code><br></td>
<td style="text-align: left;"><a href="https://www.sas.com/en_us/home.html">Website</a></td>
<td style="text-align: left;">Providing a scalable, integrated software environment designed for data access, transformation, and reporting.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>statistics</strong></td>
<td style="text-align: left;"><code>spm</code></td>
<td style="text-align: left;"><code>12</code><br></td>
<td style="text-align: left;"><a href="https://www.fil.ion.ucl.ac.uk/spm/">Website</a></td>
<td style="text-align: left;">The SPM software package has been designed for the analysis of brain imaging data sequences.</td>
</tr>
</tbody>
</table>
<h3 id="system"><strong>system</strong><a class="headerlink" href="#system" title="Permanent link">#</a></h3>
<!-- markdownlint-disable MD045 MD056 MD058 -->
<table>
<thead>
<tr>
<th style="text-align: left;">Field</th>
<th style="text-align: left;">Module Name<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:110px"></a></th>
<th style="text-align: left;">Version(s)<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:90px"></a></th>
<th style="text-align: left;">URL</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><strong>benchmark</strong></td>
<td style="text-align: left;"><code>ior</code></td>
<td style="text-align: left;"><code>3.0.1</code><br></td>
<td style="text-align: left;"><a href="https://ior.readthedocs.io/en/latest/">Website</a></td>
<td style="text-align: left;">IOR is a parallel IO benchmark that can be used to test the performance of parallel storage systems using various interfaces and access patterns.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>build</strong></td>
<td style="text-align: left;"><code>cmake</code></td>
<td style="text-align: left;"><code>3.19.2</code><br><code>3.22.2</code><br><code>3.30.3</code><br></td>
<td style="text-align: left;"><a href="http://www.cmake.org">Website</a></td>
<td style="text-align: left;">tool for generation of files from source</td>
</tr>
<tr>
<td style="text-align: left;"><strong>build</strong></td>
<td style="text-align: left;"><code>patch</code></td>
<td style="text-align: left;"><code>2.7.6</code><br></td>
<td style="text-align: left;"><a href="http://savannah.gnu.org/projects/patch/">Website</a></td>
<td style="text-align: left;">Patch takes a patch file containing a difference listing produced by the diff program and applies those differences to one or more original files, producing patched versions.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>checkpointing</strong></td>
<td style="text-align: left;"><code>dmtcp</code></td>
<td style="text-align: left;"><code>2.6.0</code><br><code>3.1.2</code><br></td>
<td style="text-align: left;"><a href="https://dmtcp.sourceforge.io/">Website</a></td>
<td style="text-align: left;">DMTCP (Distributed MultiThreaded Checkpointing) transparently checkpoints a single-host or distributed computation in user-space -- with no modifications to user code or to the O/S.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>cloud interface</strong></td>
<td style="text-align: left;"><code>awscli</code></td>
<td style="text-align: left;"><code>2.1.37</code><br><code>2.7.32</code><br><code>2.22.35</code><br></td>
<td style="text-align: left;"><a href="https://aws.amazon.com/cli/">Website</a></td>
<td style="text-align: left;">AWS Command Line Interface (AWS CLI) is a unified tool to manage your AWS services.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>compression</strong></td>
<td style="text-align: left;"><code>pigz</code></td>
<td style="text-align: left;"><code>2.6</code><br></td>
<td style="text-align: left;"><a href="https://zlib.net/pigz/">Website</a></td>
<td style="text-align: left;">pigz, which stands for parallel implementation of gzip, is a fully functional replacement for gzip that exploits multiple processors and multiple cores to the hilt when compressing data.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>containers</strong></td>
<td style="text-align: left;"><code>apptainer</code></td>
<td style="text-align: left;"><code>1.3.0</code><br><code>1.4.1</code><br></td>
<td style="text-align: left;"><a href="https://apptainer.org/">Website</a></td>
<td style="text-align: left;">A tool for running containers on a hpc system.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>containers</strong></td>
<td style="text-align: left;"><code>singularity</code></td>
<td style="text-align: left;"><code>3.7.1</code><br></td>
<td style="text-align: left;"><a href="https://sylabs.io/singularity/">Website</a></td>
<td style="text-align: left;">A tool for running containers on a hpc system.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>data management</strong></td>
<td style="text-align: left;"><code>mpifileutils</code></td>
<td style="text-align: left;"><code>0.11</code><br></td>
<td style="text-align: left;"><a href="https://github.com/hpc/mpifileutils">Website</a></td>
<td style="text-align: left;">mpiFileUtils provides both a library called libmfu and a suite of MPI-based tools to manage large datasets, which may vary from large directory trees to large files.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>database</strong></td>
<td style="text-align: left;"><code>duckdb</code></td>
<td style="text-align: left;"><code>1.3.1</code><br></td>
<td style="text-align: left;"><a href="https://duckdb.org/">Website</a></td>
<td style="text-align: left;">DuckDB is a fast portable in-process database system.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>database</strong></td>
<td style="text-align: left;"><code>mysql-client</code></td>
<td style="text-align: left;"><code>8.0.43</code><br></td>
<td style="text-align: left;"><a href="https://dev.mysql.com/">Website</a></td>
<td style="text-align: left;">The MySQL client enables interaction with a MySQL database server.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>document processing</strong></td>
<td style="text-align: left;"><code>texinfo</code></td>
<td style="text-align: left;"><code>6.8</code><br><code>7.1</code><br></td>
<td style="text-align: left;"><a href="https://www.gnu.org/software/texinfo/">Website</a></td>
<td style="text-align: left;">Texinfo uses a single source file to produce output in a number of formats.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>document processing</strong></td>
<td style="text-align: left;"><code>texlive</code></td>
<td style="text-align: left;"><code>2020</code><br><code>2024</code><br></td>
<td style="text-align: left;"><a href="https://www.tug.org/texlive/">Website</a></td>
<td style="text-align: left;">TeX Live is intended to be a straightforward way to get up and running with the TeX document production system.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>file transfer</strong></td>
<td style="text-align: left;"><code>aria2c</code></td>
<td style="text-align: left;"><code>1.37.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/aria2/aria2">Website</a></td>
<td style="text-align: left;">aria2 is a utility for downloading files. The supported protocols are HTTP(S), FTP, SFTP, BitTorrent, and Metalink.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>file transfer</strong></td>
<td style="text-align: left;"><code>aspera-connect</code></td>
<td style="text-align: left;"><code>3.11.2.63</code><br></td>
<td style="text-align: left;"><a href="https://www.ibm.com/aspera/connect/">Website</a></td>
<td style="text-align: left;">High-performance transfer client</td>
</tr>
<tr>
<td style="text-align: left;"><strong>file transfer</strong></td>
<td style="text-align: left;"><code>globus-cli</code></td>
<td style="text-align: left;"><code>1.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/globus/globus-cli">Website</a></td>
<td style="text-align: left;">The CLI provides an interface to Globus services from the shell, and is suited to both interactive and simple scripting use cases. It is open source and available at https://github.com/globus/globus-cli</td>
</tr>
<tr>
<td style="text-align: left;"><strong>file transfer</strong></td>
<td style="text-align: left;"><code>globusconnect</code></td>
<td style="text-align: left;"><code>3.2.2</code><br><code>3.2.5</code><br></td>
<td style="text-align: left;"><a href="https://www.globus.org/globus-connect-personal">Website</a></td>
<td style="text-align: left;">Globus Connect Personal turns your laptop or other personal computer into a Globus endpoint and allows you to transfer and share files easily.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>file transfer</strong></td>
<td style="text-align: left;"><code>rclone</code></td>
<td style="text-align: left;"><code>1.57.0</code><br><code>1.67.0</code><br><code>1.69.2</code><br></td>
<td style="text-align: left;"><a href="https://rclone.org/">Website</a></td>
<td style="text-align: left;">Rclone is a command line program to manage files on cloud storage.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>ide</strong></td>
<td style="text-align: left;"><code>code-server</code></td>
<td style="text-align: left;"><code>4.12.0</code><br><code>4.20.0</code><br><code>4.92.2</code><br><code>4.100.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/coder/code-server">Website</a></td>
<td style="text-align: left;">Run VS Code on any machine anywhere and access it in the browser.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>ide</strong></td>
<td style="text-align: left;"><code>cursor</code></td>
<td style="text-align: left;"><code>0.45.5</code><br></td>
<td style="text-align: left;"><a href="https://www.cursor.com/">Website</a></td>
<td style="text-align: left;">Cursor is a code editor built for programming with AI.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>ide</strong></td>
<td style="text-align: left;"><code>positron</code></td>
<td style="text-align: left;"><code>2025.09.0-139</code><br></td>
<td style="text-align: left;"><a href="https://positron.posit.co/">Website</a></td>
<td style="text-align: left;">Positron unifies exploration and production work in one free, AI-powered environment, empowering the full spectrum of data science in Python and R.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>ide</strong></td>
<td style="text-align: left;"><code>spyder</code></td>
<td style="text-align: left;"><code>6.0.3</code><br></td>
<td style="text-align: left;"><a href="https://www.spyder-ide.org/">Website</a></td>
<td style="text-align: left;">Spyder is a free and open source scientific environment written in Python, for Python, and designed by and for scientists, engineers and data analysts.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>ide</strong></td>
<td style="text-align: left;"><code>vscode</code></td>
<td style="text-align: left;"><code>1.99.3</code><br></td>
<td style="text-align: left;"><a href="https://code.visualstudio.com/">Website</a></td>
<td style="text-align: left;">VS Code is a free code editor</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>giflib</code></td>
<td style="text-align: left;"><code>4.2.3</code><br></td>
<td style="text-align: left;"><a href="https://sourceforge.net/projects/giflib/">Website</a></td>
<td style="text-align: left;">A library and utilities for processing GIFs.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>libevent</code></td>
<td style="text-align: left;"><code>2.0.22</code><br><code>2.1.12</code><br></td>
<td style="text-align: left;"><a href="https://libevent.org/">Website</a></td>
<td style="text-align: left;">The libevent API provides a mechanism to execute a callback function when a specific event occurs on a file descriptor or after a timeout has been reached.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>libfabric</code></td>
<td style="text-align: left;"><code>1.11.2</code><br><code>1.22.0</code><br></td>
<td style="text-align: left;"><a href="https://ofiwg.github.io/libfabric/">Website</a></td>
<td style="text-align: left;">Libfabric OFI</td>
</tr>
<tr>
<td style="text-align: left;"><strong>lib</strong></td>
<td style="text-align: left;"><code>openjpeg</code></td>
<td style="text-align: left;"><code>1.5.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/uclouvain/openjpeg">Website</a></td>
<td style="text-align: left;">OpenJPEG is an open-source JPEG 2000 codec written in C language.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>library</strong></td>
<td style="text-align: left;"><code>hwloc</code></td>
<td style="text-align: left;"><code>2.3.0</code><br><code>2.9.3</code><br><code>2.11.2</code><br><code>2.12.2</code><br></td>
<td style="text-align: left;"><a href="https://www.open-mpi.org/projects/hwloc/">Website</a></td>
<td style="text-align: left;">Portable Hardware Locality (hwloc)</td>
</tr>
<tr>
<td style="text-align: left;"><strong>library</strong></td>
<td style="text-align: left;"><code>readline</code></td>
<td style="text-align: left;"><code>6.2.p5</code><br></td>
<td style="text-align: left;"><a href="https://ftp.gnu.org/gnu/readline/">Website</a></td>
<td style="text-align: left;">The GNU Readline library provides a set of functions for use by applications that allow users to edit command lines as they are typed in.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>media</strong></td>
<td style="text-align: left;"><code>ffmpeg</code></td>
<td style="text-align: left;"><code>4.4</code><br></td>
<td style="text-align: left;"><a href="https://ffmpeg.org/">Website</a></td>
<td style="text-align: left;">A complete, cross-platform solution to record, convert and stream audio and video.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>package manager</strong></td>
<td style="text-align: left;"><code>miniconda3</code></td>
<td style="text-align: left;"><code>4.9.2</code><br></td>
<td style="text-align: left;"><a href="https://docs.conda.io/en/latest/miniconda.html">Website</a></td>
<td style="text-align: left;">Miniconda is a small, bootstrap version of Anaconda.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>package manager</strong></td>
<td style="text-align: left;"><code>miniforge</code></td>
<td style="text-align: left;"><code>missing</code><br></td>
<td style="text-align: left;"><a href="https://github.com/conda-forge/miniforge">Website</a></td>
<td style="text-align: left;">Miniforge is a conda installer based off conda-forge.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>pipeline</strong></td>
<td style="text-align: left;"><code>nextflow</code></td>
<td style="text-align: left;"><code>24.10.4</code><br><code>25.04.4</code><br><code>25.10.2</code><br></td>
<td style="text-align: left;"><a href="https://www.nextflow.io/">Website</a></td>
<td style="text-align: left;">Software pipeline for automating omics-scale protein modeling and simulation setup.  http://ensembler.readthedocs.org/</td>
</tr>
<tr>
<td style="text-align: left;"><strong>profiling</strong></td>
<td style="text-align: left;"><code>monitor</code></td>
<td style="text-align: left;"><code>2.1.3</code><br></td>
<td style="text-align: left;"><a href="https://resource-monitor.readthedocs.io/en/latest/index.html">Website</a></td>
<td style="text-align: left;">A simple cross-platform system resource monitor.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>profiling</strong></td>
<td style="text-align: left;"><code>remora</code></td>
<td style="text-align: left;"><code>1.8.5-impi</code><br><code>1.8.5</code><br></td>
<td style="text-align: left;"><a href="https://github.com/TACC/remora">Website</a></td>
<td style="text-align: left;">REsource MOnitoring for Remote Applications</td>
</tr>
<tr>
<td style="text-align: left;"><strong>remote display</strong></td>
<td style="text-align: left;"><code>virtualgl</code></td>
<td style="text-align: left;"><code>2.6.5</code><br><code>3.0</code><br></td>
<td style="text-align: left;"><a href="https://virtualgl.org/">Website</a></td>
<td style="text-align: left;">VirtualGL is an open source toolkit that gives any Unix or Linux remote display software the ability to run OpenGL applications with full 3D hardware acceleration.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>scm</strong></td>
<td style="text-align: left;"><code>git</code></td>
<td style="text-align: left;"><code>2.30.0</code><br><code>2.48.1</code><br></td>
<td style="text-align: left;"><a href="http://git-scm.com">Website</a></td>
<td style="text-align: left;">Fast Version Control System</td>
</tr>
<tr>
<td style="text-align: left;"><strong>terminal</strong></td>
<td style="text-align: left;"><code>tmux</code></td>
<td style="text-align: left;"><code>3.1c</code><br></td>
<td style="text-align: left;"><a href="https://github.com/tmux/tmux">Website</a></td>
<td style="text-align: left;">tmux is a terminal multiplexer; it enables a number of terminals to be created, accessed, and controlled from a single screen.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>terminal</strong></td>
<td style="text-align: left;"><code>ttyd</code></td>
<td style="text-align: left;"><code>1.6.3</code><br></td>
<td style="text-align: left;"><a href="https://github.com/tsl0922/ttyd">Website</a></td>
<td style="text-align: left;">ttyd is a simple command-line tool for sharing terminal over the web.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>tools</strong></td>
<td style="text-align: left;"><code>imagemagick</code></td>
<td style="text-align: left;"><code>7.0.10</code><br><code>7.1.1</code><br></td>
<td style="text-align: left;"><a href="https://imagemagick.org/index.php">Website</a></td>
<td style="text-align: left;">Use ImageMagick to create, edit, compose, or convert digital images.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>tools</strong></td>
<td style="text-align: left;"><code>lmod</code></td>
<td style="text-align: left;"><code>9.0.4</code><br></td>
<td style="text-align: left;"><a href="http://www.tacc.utexas.edu/tacc-projects/lmod">Website</a></td>
<td style="text-align: left;">An environment module system</td>
</tr>
<tr>
<td style="text-align: left;"><strong>tools</strong></td>
<td style="text-align: left;"><code>parallel</code></td>
<td style="text-align: left;"><code>20210122</code><br></td>
<td style="text-align: left;"><a href="https://www.gnu.org/software/parallel/">Website</a></td>
<td style="text-align: left;">GNU parallel is a shell tool for executing jobs in parallel using one or more computers.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>tools</strong></td>
<td style="text-align: left;"><code>povray</code></td>
<td style="text-align: left;"><code>3.7.0.0</code><br></td>
<td style="text-align: left;"><a href="https://github.com/POV-Ray/povray">Website</a></td>
<td style="text-align: left;">The Persistence of Vision Raytracer is a high-quality, Free Software tool for creating stunning three-dimensional graphics.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>utilities</strong></td>
<td style="text-align: left;"><code>pandoc</code></td>
<td style="text-align: left;"><code>2.11.4</code><br></td>
<td style="text-align: left;"><a href="https://pandoc.org/">Website</a></td>
<td style="text-align: left;">Pandoc is a universal document converter.</td>
</tr>
</tbody>
</table>
<h3 id="viz"><strong>viz</strong><a class="headerlink" href="#viz" title="Permanent link">#</a></h3>
<!-- markdownlint-disable MD045 MD056 MD058 -->
<table>
<thead>
<tr>
<th style="text-align: left;">Field</th>
<th style="text-align: left;">Module Name<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:110px"></a></th>
<th style="text-align: left;">Version(s)<a class="glightbox" data-type="image" data-width="auto" data-height="auto" href="" data-desc-position="bottom"><img style="min-width:90px"></a></th>
<th style="text-align: left;">URL</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align: left;"><strong>graphics</strong></td>
<td style="text-align: left;"><code>blender</code></td>
<td style="text-align: left;"><code>2.79b</code><br><code>3.0.1</code><br></td>
<td style="text-align: left;"><a href="https://www.blender.org/">Website</a></td>
<td style="text-align: left;">Blender is the free and open source 3D creation suite.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>graphics</strong></td>
<td style="text-align: left;"><code>ilastik</code></td>
<td style="text-align: left;"><code>1.3.3</code><br><code>1.4.0rc6</code><br></td>
<td style="text-align: left;"><a href="https://www.ilastik.org/index.html">Website</a></td>
<td style="text-align: left;">the interactive learning and segmentation toolkit</td>
</tr>
<tr>
<td style="text-align: left;"><strong>graphics</strong></td>
<td style="text-align: left;"><code>qupath</code></td>
<td style="text-align: left;"><code>0.3.2</code><br><code>0.5.0</code><br><code>0.5.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/qupath/qupath">Website</a></td>
<td style="text-align: left;">QuPath is open source software for bioimage analysis.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>graphics</strong></td>
<td style="text-align: left;"><code>slicer</code></td>
<td style="text-align: left;"><code>5.10.0</code><br></td>
<td style="text-align: left;"><a href="https://www.slicer.org">Website</a></td>
<td style="text-align: left;">3D Slicer is a free, open source software for visualization, processing, segmentation, registration, and analysis of medical, biomedical, and other 3D images and meshes; and planning and navigating image-guided procedures.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>graphs</strong></td>
<td style="text-align: left;"><code>graphviz</code></td>
<td style="text-align: left;"><code>2.40.1</code><br></td>
<td style="text-align: left;"><a href="https://graphviz.org/">Website</a></td>
<td style="text-align: left;">Graph visualization is a way of representing structural information as diagrams of abstract graphs and networks.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>graphs</strong></td>
<td style="text-align: left;"><code>mcl</code></td>
<td style="text-align: left;"><code>14.137</code><br></td>
<td style="text-align: left;"><a href="http://micans.org/mcl/">Website</a></td>
<td style="text-align: left;">The MCL algorithm is short for the Markov Cluster Algorithm, a fast and scalable unsupervised cluster algorithm for graphs (also known as networks) based on simulation of (stochastic) flow in graphs.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>graphs</strong></td>
<td style="text-align: left;"><code>orca</code></td>
<td style="text-align: left;"><code>1.3.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/plotly/orca">Website</a></td>
<td style="text-align: left;">Command line application for generating static images of interactive plotly charts</td>
</tr>
<tr>
<td style="text-align: left;"><strong>imaging</strong></td>
<td style="text-align: left;"><code>kaleido</code></td>
<td style="text-align: left;"><code>0.2.1</code><br></td>
<td style="text-align: left;"><a href="https://github.com/plotly/Kaleido">Website</a></td>
<td style="text-align: left;">Kaleido is a cross-platform library for generating static images (e.g. png, svg, pdf, etc.) for web-based visualization libraries, with a particular focus on eliminating external dependencies.</td>
</tr>
<tr>
<td style="text-align: left;"><strong>imaging</strong></td>
<td style="text-align: left;"><code>netpbm</code></td>
<td style="text-align: left;"><code>10.73.30</code><br></td>
<td style="text-align: left;"><a href="https://sourceforge.net/projects/netpbm/">Website</a></td>
<td style="text-align: left;">A whole bunch of utilities for primitive manipulation of graphic images.</td>
</tr>
</tbody>
</table>







  
    
  
  


  <aside class="md-source-file">
    
      
  <span class="md-source-file__fact">
    <span class="md-icon" title="Last update">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21 13.1c-.1 0-.3.1-.4.2l-1 1 2.1 2.1 1-1c.2-.2.2-.6 0-.8l-1.3-1.3c-.1-.1-.2-.2-.4-.2m-1.9 1.8-6.1 6V23h2.1l6.1-6.1zM12.5 7v5.2l4 2.4-1 1L11 13V7zM11 21.9c-5.1-.5-9-4.8-9-9.9C2 6.5 6.5 2 12 2c5.3 0 9.6 4.1 10 9.3-.3-.1-.6-.2-1-.2s-.7.1-1 .2C19.6 7.2 16.2 4 12 4c-4.4 0-8 3.6-8 8 0 4.1 3.1 7.5 7.1 7.9l-.1.2z"></path></svg>
    </span>
    <span class="git-revision-date-localized-plugin git-revision-date-localized-plugin-date" title="February 17, 2026 18:58:19 UTC">February 17, 2026</span>
  </span>

    
    
    
    
  </aside>


  




                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"></path></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        
<div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://github.com/mcw-rcc" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M173.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6m-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3m44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9M252.8 8C114.1 8 8 113.3 8 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C436.2 457.8 504 362.9 504 252 504 113.3 391.5 8 252.8 8M105.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1m-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7m32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1m-11.4-14.7c-1.6 1-1.6 3.6 0 5.9s4.3 3.3 5.6 2.3c1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2"></path></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
      <div class="md-consent" data-md-component="consent" id="__consent" hidden>
        <div class="md-consent__overlay"></div>
        <aside class="md-consent__inner">
          <form class="md-consent__form md-grid md-typeset" name="consent">
            


  


<h4>Cookie consent</h4>
<p>We use cookies to recognize your repeated visits and preferences, as well as to measure the effectiveness of our documentation and whether users find what they're searching for. With your consent, you're helping us to make our documentation better.
</p>
<input class="md-toggle" type="checkbox" id="__settings">
<div class="md-consent__settings">
  <ul class="task-list">
    
    
      
        
  
  
    
    
  
  <li class="task-list-item">
    <label class="task-list-control">
      <input type="checkbox" name="analytics" checked>
      <span class="task-list-indicator"></span>
      Google Analytics
    </label>
  </li>

      
    
    
      
        
  
  
    
    
  
  <li class="task-list-item">
    <label class="task-list-control">
      <input type="checkbox" name="github" checked>
      <span class="task-list-indicator"></span>
      GitHub
    </label>
  </li>

      
    
    
  </ul>
</div>
<div class="md-consent__controls">
  
    
      <button class="md-button md-button--primary">Accept</button>
    
    
    
  
    
    
    
      <label class="md-button" for="__settings">Manage settings</label>
    
  
</div>
          </form>
        </aside>
      </div>
      <script>var consent=__md_get("__consent");if(consent)for(var input of document.forms.consent.elements)input.name&&(input.checked=consent[input.name]||!1);else"file:"!==location.protocol&&setTimeout((function(){document.querySelector("[data-md-component=consent]").hidden=!1}),250);var form=document.forms.consent;for(var action of["submit","reset"])form.addEventListener(action,(function(e){if(e.preventDefault(),"reset"===e.type)for(var n of document.forms.consent.elements)n.name&&(n.checked=!1);__md_set("__consent",Object.fromEntries(Array.from(new FormData(form).keys()).map((function(e){return[e,!0]})))),location.hash="",location.reload()}))</script>
    
    
      
      <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.indexes", "navigation.tabs", "navigation.tabs.sticky", "navigation.sections", "navigation.tracking", "navigation.top", "search.highlight", "search.share", "search.suggest", "content.code.copy"], "search": "../../assets/javascripts/workers/search.973d3a69.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.f55a23d4.min.js"></script>
      
        <script src="../../_static/js/mathjax.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
    
  
<script id="init-glightbox">const lightbox = GLightbox({"touchNavigation": true, "loop": false, "zoomable": true, "draggable": true, "openEffect": "zoom", "closeEffect": "zoom", "slideEffect": "slide"});
document$.subscribe(()=>{ lightbox.reload(); });
</script></body></html>